<!DOCTYPE html><html lang="en" ><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="theme-color" media="(prefers-color-scheme: light)" content="#f7f7f7"><meta name="theme-color" media="(prefers-color-scheme: dark)" content="#1b1b1e"><meta name="apple-mobile-web-app-capable" content="yes"><meta name="apple-mobile-web-app-status-bar-style" content="black-translucent"><meta name="viewport" content="width=device-width, user-scalable=no initial-scale=1, shrink-to-fit=no, viewport-fit=cover" ><meta name="generator" content="Jekyll v4.3.2" /><meta property="og:title" content="The Potential of Hyperpolarized13 C MRI in Assessing Signaling Pathways in Cancer" /><meta property="og:locale" content="en" /><meta name="description" content="Advances in genomics are enabling integration of various -omics to reveal the complexities underneath carcinogenesis. Multivariate signaling pathways are deregulated and evolve spatially and temporally depending on the tumor microenvironment. This finding shifts the focus of cancer research from “one disease–one target and drug” to “one disease–multiple pathway targets and combinational therapy” and imposes new challenges on the imaging community in terms of imaging targets, scales and information levels. In current clinical settings, most imaging modalities assess cancer risk through alternations in anatomy, function, metabolism, cellularity, or limited molecular events. Few clinical-translatable imaging modalities are capable of detecting aberrations in signaling pathways at the level of tissue biology. An exception to this is hyperpolarized 13 C magnetic resonance spectroscopic imaging (HP 13 C MRI), which is capable of imaging the molecular signatures of special metabolic enzymes using HP 13 C-labeled substrates. HP 13 C MRI can identify multiple metabolites including intermediates and products simultaneously to allow extraction of critical parameters such as flux alterations for multiple metabolic pathways. Meanwhile, recent progress in cancer metabolism research affirms that metabolic alterations are directly controlled by signaling pathways. Thus, in vivo assessment of aberrations occurring in signaling pathways becomes feasible through HP 13 C imaging. This report briefly reviews the connections between signaling pathways and cancer metabolic phenotypes, the current status of HP 13 C MRI in assessing signal pathways, and recent advances in HP 13 C MRI techniques. Integrated with cancer genomics and animal models, HP 13 C MRI may hold high promise in exploring important issues in cancer that are linked to functionality of signaling pathways. Examples include genomic-driven therapy, intratumoral heterogeneity, and drug resistances." /><meta property="og:description" content="Advances in genomics are enabling integration of various -omics to reveal the complexities underneath carcinogenesis. Multivariate signaling pathways are deregulated and evolve spatially and temporally depending on the tumor microenvironment. This finding shifts the focus of cancer research from “one disease–one target and drug” to “one disease–multiple pathway targets and combinational therapy” and imposes new challenges on the imaging community in terms of imaging targets, scales and information levels. In current clinical settings, most imaging modalities assess cancer risk through alternations in anatomy, function, metabolism, cellularity, or limited molecular events. Few clinical-translatable imaging modalities are capable of detecting aberrations in signaling pathways at the level of tissue biology. An exception to this is hyperpolarized 13 C magnetic resonance spectroscopic imaging (HP 13 C MRI), which is capable of imaging the molecular signatures of special metabolic enzymes using HP 13 C-labeled substrates. HP 13 C MRI can identify multiple metabolites including intermediates and products simultaneously to allow extraction of critical parameters such as flux alterations for multiple metabolic pathways. Meanwhile, recent progress in cancer metabolism research affirms that metabolic alterations are directly controlled by signaling pathways. Thus, in vivo assessment of aberrations occurring in signaling pathways becomes feasible through HP 13 C imaging. This report briefly reviews the connections between signaling pathways and cancer metabolic phenotypes, the current status of HP 13 C MRI in assessing signal pathways, and recent advances in HP 13 C MRI techniques. Integrated with cancer genomics and animal models, HP 13 C MRI may hold high promise in exploring important issues in cancer that are linked to functionality of signaling pathways. Examples include genomic-driven therapy, intratumoral heterogeneity, and drug resistances." /><link rel="canonical" href="https://clinicaltree.github.io/posts/the-potential-of-hyperpolarized13-c-mri-in-assessing-signaling-pathways-in-cancer/" /><meta property="og:url" content="https://clinicaltree.github.io/posts/the-potential-of-hyperpolarized13-c-mri-in-assessing-signaling-pathways-in-cancer/" /><meta property="og:site_name" content="Radiology Tree" /><meta property="og:type" content="article" /><meta property="article:published_time" content="2014-01-31T17:00:00+00:00" /><meta name="twitter:card" content="summary" /><meta property="twitter:title" content="The Potential of Hyperpolarized13 C MRI in Assessing Signaling Pathways in Cancer" /><meta name="twitter:site" content="@twitter_username" /><meta name="google-site-verification" content="RFHVRgQqK0eGjftEMCTDhsDrR8cJ_ZYcfCX52gXW8KM" /> <script type="application/ld+json"> {"@context":"https://schema.org","@type":"BlogPosting","dateModified":"2023-04-10T02:43:55+00:00","datePublished":"2014-01-31T17:00:00+00:00","description":"Advances in genomics are enabling integration of various -omics to reveal the complexities underneath carcinogenesis. Multivariate signaling pathways are deregulated and evolve spatially and temporally depending on the tumor microenvironment. This finding shifts the focus of cancer research from “one disease–one target and drug” to “one disease–multiple pathway targets and combinational therapy” and imposes new challenges on the imaging community in terms of imaging targets, scales and information levels. In current clinical settings, most imaging modalities assess cancer risk through alternations in anatomy, function, metabolism, cellularity, or limited molecular events. Few clinical-translatable imaging modalities are capable of detecting aberrations in signaling pathways at the level of tissue biology. An exception to this is hyperpolarized 13 C magnetic resonance spectroscopic imaging (HP 13 C MRI), which is capable of imaging the molecular signatures of special metabolic enzymes using HP 13 C-labeled substrates. HP 13 C MRI can identify multiple metabolites including intermediates and products simultaneously to allow extraction of critical parameters such as flux alterations for multiple metabolic pathways. Meanwhile, recent progress in cancer metabolism research affirms that metabolic alterations are directly controlled by signaling pathways. Thus, in vivo assessment of aberrations occurring in signaling pathways becomes feasible through HP 13 C imaging. This report briefly reviews the connections between signaling pathways and cancer metabolic phenotypes, the current status of HP 13 C MRI in assessing signal pathways, and recent advances in HP 13 C MRI techniques. Integrated with cancer genomics and animal models, HP 13 C MRI may hold high promise in exploring important issues in cancer that are linked to functionality of signaling pathways. Examples include genomic-driven therapy, intratumoral heterogeneity, and drug resistances.","headline":"The Potential of Hyperpolarized13 C MRI in Assessing Signaling Pathways in Cancer","mainEntityOfPage":{"@type":"WebPage","@id":"https://clinicaltree.github.io/posts/the-potential-of-hyperpolarized13-c-mri-in-assessing-signaling-pathways-in-cancer/"},"url":"https://clinicaltree.github.io/posts/the-potential-of-hyperpolarized13-c-mri-in-assessing-signaling-pathways-in-cancer/"}</script><title>The Potential of Hyperpolarized13 C MRI in Assessing Signaling Pathways in Cancer | Radiology Tree</title><link rel="apple-touch-icon" sizes="180x180" href="/assets/img/favicons/apple-touch-icon.png"><link rel="icon" type="image/png" sizes="32x32" href="/assets/img/favicons/favicon-32x32.png"><link rel="icon" type="image/png" sizes="16x16" href="/assets/img/favicons/favicon-16x16.png"><link rel="manifest" href="/assets/img/favicons/site.webmanifest"><link rel="shortcut icon" href="/assets/img/favicons/favicon.ico"><meta name="apple-mobile-web-app-title" content="Radiology Tree"><meta name="application-name" content="Radiology Tree"><meta name="msapplication-TileColor" content="#da532c"><meta name="msapplication-config" content="/assets/img/favicons/browserconfig.xml"><meta name="theme-color" content="#ffffff"><link rel="preconnect" href="https://fonts.googleapis.com" ><link rel="dns-prefetch" href="https://fonts.googleapis.com" ><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link rel="dns-prefetch" href="https://fonts.gstatic.com" crossorigin><link rel="preconnect" href="https://fonts.googleapis.com" ><link rel="dns-prefetch" href="https://fonts.googleapis.com" ><link rel="preconnect" href="https://cdn.jsdelivr.net" ><link rel="dns-prefetch" href="https://cdn.jsdelivr.net" ><link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Lato&family=Source+Sans+Pro:wght@400;600;700;900&display=swap"><link rel="preconnect" href="https://www.google-analytics.com" crossorigin="use-credentials"><link rel="dns-prefetch" href="https://www.google-analytics.com"><link rel="preconnect" href="https://www.googletagmanager.com" crossorigin="anonymous"><link rel="dns-prefetch" href="https://www.googletagmanager.com"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/css/bootstrap.min.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.2.1/css/all.min.css"><link rel="stylesheet" href="/assets/css/style.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/tocbot@4.20.1/dist/tocbot.min.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/magnific-popup@1.1.0/dist/magnific-popup.min.css"> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.1/dist/jquery.min.js"></script> <script type="text/javascript"> class ModeToggle { static get MODE_KEY() { return "mode"; } static get MODE_ATTR() { return "data-mode"; } static get DARK_MODE() { return "dark"; } static get LIGHT_MODE() { return "light"; } static get ID() { return "mode-toggle"; } constructor() { if (this.hasMode) { if (this.isDarkMode) { if (!this.isSysDarkPrefer) { this.setDark(); } } else { if (this.isSysDarkPrefer) { this.setLight(); } } } let self = this; /* always follow the system prefers */ this.sysDarkPrefers.addEventListener('change', () => { if (self.hasMode) { if (self.isDarkMode) { if (!self.isSysDarkPrefer) { self.setDark(); } } else { if (self.isSysDarkPrefer) { self.setLight(); } } self.clearMode(); } self.notify(); }); } /* constructor() */ get sysDarkPrefers() { return window.matchMedia("(prefers-color-scheme: dark)"); } get isSysDarkPrefer() { return this.sysDarkPrefers.matches; } get isDarkMode() { return this.mode === ModeToggle.DARK_MODE; } get isLightMode() { return this.mode === ModeToggle.LIGHT_MODE; } get hasMode() { return this.mode != null; } get mode() { return sessionStorage.getItem(ModeToggle.MODE_KEY); } /* get the current mode on screen */ get modeStatus() { if (this.isDarkMode || (!this.hasMode && this.isSysDarkPrefer)) { return ModeToggle.DARK_MODE; } else { return ModeToggle.LIGHT_MODE; } } setDark() { $('html').attr(ModeToggle.MODE_ATTR, ModeToggle.DARK_MODE); sessionStorage.setItem(ModeToggle.MODE_KEY, ModeToggle.DARK_MODE); } setLight() { $('html').attr(ModeToggle.MODE_ATTR, ModeToggle.LIGHT_MODE); sessionStorage.setItem(ModeToggle.MODE_KEY, ModeToggle.LIGHT_MODE); } clearMode() { $('html').removeAttr(ModeToggle.MODE_ATTR); sessionStorage.removeItem(ModeToggle.MODE_KEY); } /* Notify another plugins that the theme mode has changed */ notify() { window.postMessage({ direction: ModeToggle.ID, message: this.modeStatus }, "*"); } flipMode() { if (this.hasMode) { if (this.isSysDarkPrefer) { if (this.isLightMode) { this.clearMode(); } else { this.setLight(); } } else { if (this.isDarkMode) { this.clearMode(); } else { this.setDark(); } } } else { if (this.isSysDarkPrefer) { this.setLight(); } else { this.setDark(); } } this.notify(); } /* flipMode() */ } /* ModeToggle */ const modeToggle = new ModeToggle(); </script><body data-topbar-visible="true"><div id="sidebar" class="d-flex flex-column align-items-end"><div class="profile-wrapper text-center"><div id="avatar"> <a href="/" class="mx-auto"> <img src="https://storage.googleapis.com/clinicalpub.com/images/favicon.png" alt="avatar" onerror="this.style.display='none'"> </a></div><div class="site-title"> <a href="/">Radiology Tree</a></div><div class="site-subtitle font-italic">Update every day the best and the lastest articles, books, journals, clinical cases, videos, images... for radiologist</div></div><ul class="w-100"><li class="nav-item"> <a href="/" class="nav-link"> <i class="fa-fw fas fa-home ml-xl-3 mr-xl-3 unloaded"></i> <span>HOME</span> </a><li class="nav-item"> <a href="/categories/" class="nav-link"> <i class="fa-fw fas fa-stream ml-xl-3 mr-xl-3 unloaded"></i> <span>CATEGORIES</span> </a><li class="nav-item"> <a href="/tags/" class="nav-link"> <i class="fa-fw fas fa-tag ml-xl-3 mr-xl-3 unloaded"></i> <span>TAGS</span> </a><li class="nav-item"> <a href="/archives/" class="nav-link"> <i class="fa-fw fas fa-archive ml-xl-3 mr-xl-3 unloaded"></i> <span>ARCHIVES</span> </a><li class="nav-item"> <a href="/about/" class="nav-link"> <i class="fa-fw fas fa-info-circle ml-xl-3 mr-xl-3 unloaded"></i> <span>ABOUT</span> </a></ul><div class="sidebar-bottom mt-auto d-flex flex-wrap justify-content-center align-items-center"> <button class="mode-toggle btn" aria-label="Switch Mode"> <i class="fas fa-adjust"></i> </button> <span class="icon-border"></span> <a href="https://github.com/clinicaltree" aria-label="github" target="_blank" rel="noopener noreferrer"> <i class="fab fa-github"></i> </a> <a href="https://twitter.com/twitter_username" aria-label="twitter" target="_blank" rel="noopener noreferrer"> <i class="fab fa-twitter"></i> </a> <a href="javascript:location.href = 'mailto:' + ['clinicalpub.team','gmail.com'].join('@')" aria-label="email" > <i class="fas fa-envelope"></i> </a> <a href="/feed.xml" aria-label="rss" > <i class="fas fa-rss"></i> </a></div></div><div id="topbar-wrapper"><div id="topbar" class="container d-flex align-items-center justify-content-between h-100 pl-3 pr-3 pl-md-4 pr-md-4"> <span id="breadcrumb"> <span> <a href="/"> Home </a> </span> <span>The Potential of Hyperpolarized13 C MRI in Assessing Signaling Pathways in Cancer</span> </span> <i id="sidebar-trigger" class="fas fa-bars fa-fw"></i><div id="topbar-title"> Post</div><i id="search-trigger" class="fas fa-search fa-fw"></i> <span id="search-wrapper" class="align-items-center"> <i class="fas fa-search fa-fw"></i> <input class="form-control" id="search-input" type="search" aria-label="search" autocomplete="off" placeholder="Search..."> </span> <span id="search-cancel" >Cancel</span></div></div><div id="main-wrapper" class="d-flex justify-content-center"><div id="main" class="container pl-xl-4 pr-xl-4"><div class="row"><div id="core-wrapper" class="col-12 col-lg-11 col-xl-9 pr-xl-4"><div class="post pl-1 pr-1 pl-md-2 pr-md-2"><h1 data-toc-skip>The Potential of Hyperpolarized13 C MRI in Assessing Signaling Pathways in Cancer</h1><div class="post-meta text-muted"> <span> Posted <em class="" data-ts="1391187600" data-df="ll" data-toggle="tooltip" data-placement="bottom"> Jan 31, 2014 </em> </span> <span> Updated <em class="" data-ts="1681094635" data-df="ll" data-toggle="tooltip" data-placement="bottom"> Apr 10, 2023 </em> </span><div class="d-flex justify-content-between"> <span> By <em> <a href="">Huiming Zhang PhD</a> </em> </span><div> <span class="readtime" data-toggle="tooltip" data-placement="bottom" title="2773 words"> <em>15 min</em> read</span></div></div></div><div class="post-content"><p>Advances in genomics are enabling integration of various -omics to reveal the complexities underneath carcinogenesis. Multivariate signaling pathways are deregulated and evolve spatially and temporally depending on the tumor microenvironment. This finding shifts the focus of cancer research from “one disease–one target and drug” to “one disease–multiple pathway targets and combinational therapy” and imposes new challenges on the imaging community in terms of imaging targets, scales and information levels. In current clinical settings, most imaging modalities assess cancer risk through alternations in anatomy, function, metabolism, cellularity, or limited molecular events. Few clinical-translatable imaging modalities are capable of detecting aberrations in signaling pathways at the level of tissue biology. An exception to this is hyperpolarized 13 C magnetic resonance spectroscopic imaging (HP 13 C MRI), which is capable of imaging the molecular signatures of special metabolic enzymes using HP 13 C-labeled substrates. HP 13 C MRI can identify multiple metabolites including intermediates and products simultaneously to allow extraction of critical parameters such as flux alterations for multiple metabolic pathways. Meanwhile, recent progress in cancer metabolism research affirms that metabolic alterations are directly controlled by signaling pathways. Thus, in vivo assessment of aberrations occurring in signaling pathways becomes feasible through HP 13 C imaging. This report briefly reviews the connections between signaling pathways and cancer metabolic phenotypes, the current status of HP 13 C MRI in assessing signal pathways, and recent advances in HP 13 C MRI techniques. Integrated with cancer genomics and animal models, HP 13 C MRI may hold high promise in exploring important issues in cancer that are linked to functionality of signaling pathways. Examples include genomic-driven therapy, intratumoral heterogeneity, and drug resistances.</p><p>Tumor growth relies on various nutrient metabolic processes such as glycolysis, lipogenesis, and glutaminolysis. Many of the important metabolites are small molecules containing carbon and nitrogen. Molecules with isotopic variants of carbon ( 13 C) and nitrogen ( 15 N) have been useful in gaining insight into the complete metabolic process by magnetic resonance spectroscopy (MRS). But 13 C MRS has rarely been used in vivo in clinical studies until recently, with the emergence of a technique called dissolution dynamic nuclear polarization. This novel hyperpolarization technique can increase 13 C signal-to-noise ratio (SNR) &gt;10,000-fold in liquids . Such an enhancement is sufficient for collection of images with a few acquisitions or even a single scan.</p><p>Imaging modality based on this technique, the HP 13 C MRI, is expanding rapidly, primarily driven by interests of cancer metabolism. Using 13 C-labeled molecules of low molecular weight, this technique permits imaging of biochemical reactions in vivo for monitoring cancer and other diseases. Because many 13 C labeled probes have spin-lattice relaxation times (T 1 ) as short as minutes or less, an initial concern was whether this technique could have any impact on the clinical practice. To explore the clinical feasibility of HP 13 C MRI and its potential translational impact for cancer studies, the National Cancer Institute Cancer Imaging Program held a workshop at the 2010 ISMRM in Stockholm, Sweden. A consensus document (white paper) from the workshop was published in the early of 2011 . This white paper focused on the status, challenges and translational barriers to clinical applications for HP 13 C imaging and concluded that there were no significant barriers in translation to clinical utility. Recently a first-in-human study on prostate cancer was completed with HP [1- 13 C]pyruvate in 31 patients at University of California at San Francisco, demonstrating the clinical feasibility of this technology . Another clinic study with HP [1- 13 C]fumarate is under development at Cambridge University. These studies support the idea that HP 13 C MRI can become a new modality for clinical evaluation of metabolic phenotypes of cancer. Nevertheless, in the current era of cancer genomics and with the thought that all kinds of -omics are mutually attached in some as yet to be determined way, a specific question must be asked. Can HP 13 C MRI help us understand the fundamentals of cancer beyond the metabolism?</p><p>The progress in cancer genomics achieved in past decade is reshaping our strategies in evaluation and treatment of cancer. Genetic mutations, genetic instability, and microenvironment cause cancer to evolve in a complex way. Among the many suspected mutated genes found, a small group of genes have been identified that comprise a dozen essential signaling pathways to drive tumorigenesis . With these discoveries, present research paradigms are shifting from “one disease–one target and drug” to “one disease–multiple pathway targets and combinational therapies . This suggests a transition from the traditional genetic medicine, which is based on understanding the impact of single genes on disease, to a genomics-based medicine, which considers the impact of the entire genome along with environmental factors on diseases and health. This conceptual transition imposes new challenges on the imaging community where the interest is shifting to advance quantitative imaging technology to study signaling pathways in clinic setting. Recent advances in our understanding of cancer metabolism provide substantial evidence that cancer metabolism is directly linked to many signaling pathways . With these connections between cancer genomics and cancer metabolism, and with the promise and potential of HP 13 C MRI in studies of cancer metabolism, we are posed to explore the possibility of using HP 13 C MRI to assess signaling pathways in vivo, especially those critical pathways associated with tumor growth.</p><p>This report provides a brief summary of the current understanding of signaling pathways and their relationships to cancer metabolism. The unique features of HP 13 C MRI in differentiation of metabolic pathways and a summary of current applications in assessing signaling pathways and new utilities generated by the advanced imaging techniques are also presented. While HP 13 C MRI shows potential to explore various aspects of cancer metabolism, only those issues associated with manipulations and differentiations of signaling pathways will be discussed here. Because appropriate animal models offer unique systems for studying the signaling pathways, a summary of available animal models is presented. Methods of producing HP molecules with various mechanistic mechanisms are not included in this paper. Extensive reviews on this topic can be found elsewhere .</p><h2 id="signaling-pathways-where-are-the-targets"><span class="mr-2">Signaling Pathways: Where are the Targets?</span><a href="#signaling-pathways-where-are-the-targets" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p>Comprehensive genomic sequencing efforts in the past decade have led to the completion of mutated gene maps, known as genomics landscapes, for common human cancers . In general, these landscapes contain a few high percentage mutated genes appearing as “mountains” and a much larger number of infrequently mutated genes (&lt;5%) appearing as “hills.” About 140 of the frequently mutated genes are identified to function as “driver” of tumorigenesis, called “driver genes.” Among these driver genes, there are 64 tumor oncogenes, the genes that have high potential to cause cancer, and there are 74 tumor suppressor genes, the genes that can cause a loss of function and lead to cancer. These driver genes can be grouped into 12 signaling pathways in cell fate, cell survival, and genome maintenance, which represent three core cellular processes in tumorigenesis. Normally, oncogenes are directly targetable (i.e., it is known that 31 of the 64 oncogenes have enzymatic activities that can be targeted by drugs). Compared to the oncogenes, the tumor suppressor genes are not targetable. However, the inactivation caused by the mutation of tumor suppressor gene may activate some growth signaling at downstream that is targetable by small molecules. Therefore, multiple targeting of downstream nodes of the same signaling pathway becomes an important strategy to apply in the presence of mutated tumor suppressor genes. For typical solid tumors with 30 to 70 mutated genes, 2 to 8 of them are driver genes. Most solid tumors are dominated by the mutated tumor suppressor genes, and a few of them can contain more than one mutated oncogene. In these cases, pathway targeting or multisite simultaneous targeting become important approaches .</p><h2 id="metabolic-phenotypes-what-are-their-linkages-to-signaling-pathways"><span class="mr-2">Metabolic Phenotypes: What are Their Linkages to Signaling Pathways?</span><a href="#metabolic-phenotypes-what-are-their-linkages-to-signaling-pathways" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="metabolic-imaging-why-hp--13--c-mri"><span class="mr-2">Metabolic Imaging: Why HP 13 C MRI?</span><a href="#metabolic-imaging-why-hp--13--c-mri" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="hp--13--c-mri-in-assessing-signaling-pathways-what-have-been-done"><span class="mr-2">HP 13 C MRI in Assessing Signaling Pathways: What Have Been Done?</span><a href="#hp--13--c-mri-in-assessing-signaling-pathways-what-have-been-done" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="hp--13--c-mri-techniques-whats-new"><span class="mr-2">HP 13 C MRI Techniques: What’s New?</span><a href="#hp--13--c-mri-techniques-whats-new" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="assessing-signaling-pathways-whats-next"><span class="mr-2">Assessing Signaling Pathways: What’s Next?</span><a href="#assessing-signaling-pathways-whats-next" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><h2 id="potential-opportunities"><span class="mr-2">Potential opportunities</span><a href="#potential-opportunities" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="animal-models"><span class="mr-2">Animal models</span><a href="#animal-models" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="genomics-driven-approach"><span class="mr-2">Genomics-Driven Approach</span><a href="#genomics-driven-approach" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://storage.googleapis.com/dl.dentistrykey.com/clinical/ThePotentialofHyperpolarized13CMRIinAssessingSignalingPathwaysinCancer/0_1s20S1076633213005485.jpg" class="popup img-link "><img data-src="https://storage.googleapis.com/dl.dentistrykey.com/clinical/ThePotentialofHyperpolarized13CMRIinAssessingSignalingPathwaysinCancer/0_1s20S1076633213005485.jpg" alt="Figure 1, Application of hyperpolarized 13 C MRI in cancer: cancer genomics–driven approach. XG, xenograft mouse model; GEM, genetic engineered mouse model; PDX, patient-derived tumor xenograft model." class="lazyload" data-proofer-ignore></a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="conclusions"><span class="mr-2">Conclusions</span><a href="#conclusions" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="acknowledgment"><span class="mr-2">Acknowledgment</span><a href="#acknowledgment" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="references"><span class="mr-2">References</span><a href="#references" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><ul><li><p>1. Ardenkjaer-Larsen J.H., Fridlund B., Gram A., et. al.: Increase in signal-to-noise ratio of &gt; 10,000 times in liquid-state NMR. Proc Natl Acad Sci USA 2003; 100: pp. 10158-10163.</p><li><p>2. Kurhanewicz J., Vigneron D.B., Brindle K., et. al.: Analysis of cancer metabolism by imaging hyperpolarized nuclei: prospects for translation to clinical research. Neoplasia 2011; 13: pp. 81-97.</p><li><p>3. Nelson S.J., Kurhanewicz J., Vigneron D.B., et. al.: Metabolic imaging of patients with prostate cancer using hyperpolarized [1-(13)C]pyruvate. Sci Transl Med 2013; 5: pp. 198-208.</p><li><p>4. Vogelstein B., Papadopoulos N., Velculescu V.E., et. al.: Cancer genome landscapes. Science 2013; 339: pp. 1546-1558.</p><li><p>5. Yap T.A., Omlin A., de Bono J.S.: Development of therapeutic combinations targeting major cancer signaling pathways. J Clin Oncol 2013; 31: pp. 1592-1605.</p><li><p>6. Garraway L.A.: Genomics-driven oncology: framework for an emerging paradigm. J Clin Oncol 2013; 31: pp. 1806-1814.</p><li><p>7. Garay J.P., Gray J.W.: Omics and therapy: a basis for precision medicine. Mol Oncol 2012; 6: pp. 128-139.</p><li><p>8. Jones R.G., Thompson C.B.: Tumor suppressors and cell metabolism: a recipe for cancer growth. Genes Dev 2009; 23: pp. 537-548.</p><li><p>9. Ward P.S., Thompson C.B.: Metabolic reprogramming: a cancer hallmark even Warburg did not anticipate. Cancer Cell 2012; 21: pp. 297-308.</p><li><p>10. DeBerardinis R.J., Thompson C.B.: Cellular metabolism and disease: what do metabolic outliers teach us?. Cell 2012; 148: pp. 1132-1144.</p><li><p>11. Kroemer G., Pouyssegur J.: Tumor cell metabolism: cancer’s Achilles’ heel. Cancer Cell 2008; 13: pp. 472-482.</p><li><p>12. Cairns R.A., Harris I.S., Mak T.W.: Regulation of cancer cell metabolism. Nat Rev Cancer 2011; 11: pp. 85-95.</p><li><p>13. Brindle K.: Watching tumours gasp and die with MRI: the promise of hyperpolarised 13C MR spectroscopic imaging. Br J Radiol 2012; 85: pp. 697-708.</p><li><p>14. Nelson S.J., Ozhinsky E., Li Y., et. al.: Strategies for rapid in vivo 1H and hyperpolarized 13C MR spectroscopic imaging. J Magn Reson 2013; 229: pp. 187-197.</p><li><p>15. Dang C.V.: MYC on the path to cancer. Cell 2012; 149: pp. 22-35.</p><li><p>16. Linehan W.M., Srinivasan R., Schmidt L.S.: The genetic basis of kidney cancer: a metabolic disease. Nat Rev Urol 2010; 7: pp. 277-285.</p><li><p>17. Condeelis J., Weissleder R.: In vivo imaging in cancer. Cold Spring Harb Perspect Biol 2010; 2: pp. a003848.</p><li><p>18. James M.L., Gambhir S.S.: A molecular imaging primer: modalities, imaging agents, and applications. Physiol Rev 2012; 92: pp. 897-965.</p><li><p>19. Wistuba I.I., Gelovani J.G., Jacoby J.J., et. al.: Methodological and practical challenges for personalized cancer therapies. Nat Rev Clin Oncol 2011; 8: pp. 135-141.</p><li><p>20. Dafni H., Larson P.E., Hu S., et. al.: Hyperpolarized 13C spectroscopic imaging informs on hypoxia-inducible factor-1 and myc activity downstream of platelet-derived growth factor receptor. Cancer Res 2010; 70: pp. 7400-7410.</p><li><p>21. Chaumeil M.M., Ozawa T., Park I., et. al.: Hyperpolarized 13C MR spectroscopic imaging can be used to monitor everolimus treatment in vivo in an orthotopic rodent model of glioblastoma. NeuroImage 2012; 59: pp. 193-201.</p><li><p>22. Lodi A., Woods S.M., Ronen S.M.: Treatment with the MEK inhibitor U0126 induces decreased hyperpolarized pyruvate to lactate conversion in breast, but not prostate, cancer cells. NMR Biomed 2013; 26: pp. 299-306.</p><li><p>23. Hu S., Balakrishnan A., Bok R.A., et. al.: 13C-pyruvate imaging reveals alterations in glycolysis that precede c-Myc-induced tumor formation and regression. Cell Metab 2011; 14: pp. 131-142.</p><li><p>24. Dutta P., Le A., Vander Jagt D.L., et. al.: Evaluation of LDH-A and glutaminase inhibition in vivo by hyperpolarized 13C-pyruvate magnetic resonance spectroscopy of tumors. Cancer Res 2013; 73: pp. 4190-4195.</p><li><p>25. Bohndiek S.E., Kettunen M.I., Hu D.E., et. al.: Hyperpolarized (13)C spectroscopy detects early changes in tumor vasculature and metabolism after VEGF neutralization. Cancer Res 2012; 72: pp. 854-864.</p><li><p>26. Ardenkjaer-Larsen J.H., Leach A.M., Clarke N., et. al.: Dynamic nuclear polarization polarizer for sterile use intent. NMR Biomed 2011; 24: pp. 927-932.</p><li><p>27. Hu S., Larson P.E., Vancriekinge M., et. al.: Rapid sequential injections of hyperpolarized [1-(1)(3)C]pyruvate in vivo using a sub-kelvin, multi-sample DNP polarizer. Magn Reson Imaging 2013; 31: pp. 490-496.</p><li><p>28. Wilson D.M., Keshari K.R., Larson P.E., et. al.: Multi-compound polarization by DNP allows simultaneous assessment of multiple enzymatic activities in vivo. J Magn Reson 2010; 205: pp. 141-147.</p><li><p>29. Meier S., Karlsson M., Jensen P.R., et. al.: Metabolic pathway visualization in living yeast by DNP-NMR. Mol Biosyst 2011; 7: pp. 2834-2836.</p><li><p>30. Ohliger M.A., Larson P.E., Bok R.A., et. al.: Combined parallel and partial fourier MR reconstruction for accelerated 8-channel hyperpolarized carbon-13 in vivo magnetic resonance spectroscopic imaging (MRSI). J Magn Reson Imaging 2013; 38: pp. 701-713.</p><li><p>31. Keshari K.R., Sriram R., Van Criekinge M., et. al.: Metabolic reprogramming and validation of hyperpolarized (13) C lactate as a prostate cancer biomarker using a human prostate tissue slice culture bioreactor. Prostate 2013; 73: pp. 1171-1181.</p><li><p>32. Li L.Z., Kadlececk S., Xu H.N., et. al.: Ratiometric analysis in hyperpolarized NMR (I): test of the two-site exchange model and the quantification of reaction rate constants. NMR Biomed 2013; 26: pp. 1308-1320.</p><li><p>33. Swisher C.L., Larson P.E., Kruttwig K., et. al.: Quantitative measurement of cancer metabolism using stimulated echo hyperpolarized carbon-13 MRS. Magn Reson Med 2013, Feb 14; [Epub ahead of print]</p><li><p>34. Heiser L.M., Sadanandam A., Kuo W.L., et. al.: Subtype and pathway specific responses to anticancer compounds in breast cancer. Proc Natl Acad Sci U S A 2012; 109: pp. 2724-2729.</p><li><p>35. Woodcock J., Griffin J.P., Behrman R.E.: Development of novel combination therapies. N Engl J Med 2011; 364: pp. 985-987.</p><li><p>36. Ding L., Ley T.J., Larson D.E., et. al.: Clonal evolution in relapsed acute myeloid leukaemia revealed by whole-genome sequencing. Nature 2012; 481: pp. 506-510.</p><li><p>37. Gerlinger M., Rowan A.J., Horswell S., et. al.: Intratumor heterogeneity and branched evolution revealed by multiregion sequencing. N Engl J Med 2012; 366: pp. 883-892.</p><li><p>38. Marusyk A., Polyak K.: Tumor heterogeneity: causes and consequences. Biochim Biophys Acta 2010; 1805: pp. 105-117.</p><li><p>39. Engelman J.A.: Targeting PI3K signalling in cancer: opportunities, challenges and limitations. Nat Rev Cancer 2009; 9: pp. 550-562.</p><li><p>40. Grinde M.T., Moestue S.A., Borgan E., et. al.: 13C high-resolution-magic angle spinning MRS reveals differences in glucose metabolism between two breast cancer xenograft models with different gene expression patterns. NMR Biomed 2011; 24: pp. 1243-1252.</p><li><p>41. Perou C.M., Borresen-Dale A.L.: Systems biology and genomics of breast cancer. Cold Spring Harb Perspect Biol 2011; 3:</p><li><p>42. Van Dyke T., Jacks T.: Cancer modeling in the modern era: progress and challenges. Cell 2002; 108: pp. 135-144.</p><li><p>43. Cheon D.J., Orsulic S.: Mouse models of cancer. Annu Rev Pathol 2011; 6: pp. 95-119.</p><li><p>44. Tentler J.J., Tan A.C., Weekes C.D., et. al.: Patient-derived tumour xenografts as models for oncology drug development. Nat Rev Clin Oncol 2012; 9: pp. 338-350.</p><li><p>45. Sharpless N.E., Depinho R.A.: The mighty mouse: genetically engineered mouse models in cancer drug development. Nat Rev Drug Discov 2006; 5: pp. 741-754.</p><li><p>46. Singh M., Murriel C.L., Johnson L.: Genetically engineered mouse models: closing the gap between preclinical data and trial outcomes. Cancer Res 2012; 72: pp. 2695-2700.</p><li><p>47. Nardella C., Lunardi A., Patnaik A., et. al.: The APL paradigm and the “co-clinical trial” project. Cancer Discov 2011; 1: pp. 108-116.</p><li><p>48. Chen Z., Cheng K., Walton Z., et. al.: A murine lung cancer co-clinical trial identifies genetic modifiers of therapeutic response. Nature 2012; 483: pp. 613-617.</p></ul></div><div class="post-tail-wrapper text-muted"><div class="post-meta mb-3"> <i class="far fa-folder-open fa-fw mr-1"></i> <a href='/categories/academic-radiology/'>Academic Radiology</a>, <a href='/categories/volume-21/'>Volume 21</a>, <a href='/categories/issue-2/'>Issue 2</a></div><div class="post-tags"> <i class="fa fa-tags fa-fw mr-1"></i> <a href="/tags/journals/" class="post-tag no-text-decoration" >Journals</a> <a href="/tags/general-radiology/" class="post-tag no-text-decoration" >General Radiology</a></div><div class="post-tail-bottom d-flex justify-content-between align-items-center mt-3 pt-5 pb-2"><div class="license-wrapper"> This post is licensed under <a href="https://creativecommons.org/licenses/by/4.0/"> CC BY 4.0 </a> by the author.</div><div class="share-wrapper"> <span class="share-label text-muted mr-1">Share</span> <span class="share-icons"> <a href="https://twitter.com/intent/tweet?text=The%20Potential%20of%20Hyperpolarized13%20C%20MRI%20in%20Assessing%20Signaling%20Pathways%20in%20Cancer%20-%20Radiology%20Tree&url=https%3A%2F%2Fclinicaltree.github.io%2Fposts%2Fthe-potential-of-hyperpolarized13-c-mri-in-assessing-signaling-pathways-in-cancer%2F" data-toggle="tooltip" data-placement="top" title="Twitter" target="_blank" rel="noopener" aria-label="Twitter"> <i class="fa-fw fab fa-twitter"></i> </a> <a href="https://www.facebook.com/sharer/sharer.php?title=The%20Potential%20of%20Hyperpolarized13%20C%20MRI%20in%20Assessing%20Signaling%20Pathways%20in%20Cancer%20-%20Radiology%20Tree&u=https%3A%2F%2Fclinicaltree.github.io%2Fposts%2Fthe-potential-of-hyperpolarized13-c-mri-in-assessing-signaling-pathways-in-cancer%2F" data-toggle="tooltip" data-placement="top" title="Facebook" target="_blank" rel="noopener" aria-label="Facebook"> <i class="fa-fw fab fa-facebook-square"></i> </a> <a href="https://t.me/share/url?url=https%3A%2F%2Fclinicaltree.github.io%2Fposts%2Fthe-potential-of-hyperpolarized13-c-mri-in-assessing-signaling-pathways-in-cancer%2F&text=The%20Potential%20of%20Hyperpolarized13%20C%20MRI%20in%20Assessing%20Signaling%20Pathways%20in%20Cancer%20-%20Radiology%20Tree" data-toggle="tooltip" data-placement="top" title="Telegram" target="_blank" rel="noopener" aria-label="Telegram"> <i class="fa-fw fab fa-telegram"></i> </a> <i id="copy-link" class="fa-fw fas fa-link small" data-toggle="tooltip" data-placement="top" title="Copy link" data-title-succeed="Link copied successfully!"> </i> </span></div></div></div></div></div><div id="panel-wrapper" class="col-xl-3 pl-2 text-muted"><div class="access"><div id="access-lastmod" class="post"><div class="panel-heading">Recently Updated</div><ul class="post-content pl-0 pb-1 ml-1 mt-2"><li><a href="/posts/neurometabolites-alteration-in-the-acute-phase-of-mild-traumatic-brain-injury-mtbi/">Neurometabolites Alteration in the Acute Phase of Mild Traumatic Brain Injury (mTBI)</a><li><a href="/posts/reinforcing-the-importance-and-feasibility-of-implementing-a-low-dose-protocol-for-ct-guided-biopsie/">Reinforcing the Importance and Feasibility of Implementing a Low-dose Protocol for CT-guided Biopsies</a><li><a href="/posts/rethinking-the-pgy-1-basic-clinical-year/">Rethinking the PGY-1 Basic Clinical Year</a><li><a href="/posts/single-injection-dual-phase-cone-beam-ct-dp-cbct-vascular-anatomy-assessment-and-occult-nodule-det/">Single Injection Dual-Phase Cone Beam CT (DP-CBCT) Vascular Anatomy Assessment and Occult Nodule Detection; Have We Reached the Focus?</a><li><a href="/posts/the-yellow-scale-is-superior-to-the-gray-scale-for-detecting-acute-ischemic-stroke-on-a-monitor-disp/">The Yellow Scale Is Superior to the Gray Scale for Detecting Acute Ischemic Stroke on a Monitor Display in Computed Tomography</a></ul></div><div id="access-tags"><div class="panel-heading">Trending Tags</div><div class="d-flex flex-wrap mt-3 mb-1 mr-3"> <a class="post-tag" href="/tags/general-radiology/">General Radiology</a> <a class="post-tag" href="/tags/journals/">Journals</a></div></div></div><div id="toc-wrapper" class="pl-0 pr-4 mb-5"><div class="panel-heading pl-3 pt-2 mb-2">Contents</div><nav id="toc"></nav></div><script src="https://cdn.jsdelivr.net/npm/tocbot@4.20.1/dist/tocbot.min.js"></script></div></div><div class="row"><div id="tail-wrapper" class="col-12 col-lg-11 col-xl-9 pl-3 pr-3 pr-xl-4 mt-5"><div id="related-posts" class="mb-2 mb-sm-4"><h3 class="pt-2 mb-4 ml-1" data-toc-skip>Further Reading</h3><div class="card-deck mb-4"><div class="card"> <a href="/posts/adaptable-near-infrared-spectroscopy-fiber-array-for-improved-coupling-to-different-breast-sizes-dur/"><div class="card-body"> <em class="small" data-ts="1391187600" data-df="ll" > Jan 31, 2014 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>Adaptable Near-Infrared Spectroscopy Fiber Array for Improved Coupling to Different Breast Sizes During Clinical MRI</h3><div class="text-muted small"><p> Rationale and Objectives Near-infrared spectroscopy (NIRS) of breast can provide functional information on the vascular and structural compartments of tissues in regions identified during simultan...</p></div></div></a></div><div class="card"> <a href="/posts/advances-in-noninvasive-functional-imaging-of-bone/"><div class="card-body"> <em class="small" data-ts="1391187600" data-df="ll" > Jan 31, 2014 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>Advances in Noninvasive Functional Imaging of Bone</h3><div class="text-muted small"><p> The demand for functional imaging in clinical medicine is comprehensive. Although the gold standard for the functional imaging of human bones in clinical settings is still radionuclide-based imagin...</p></div></div></a></div><div class="card"> <a href="/posts/blood-flow-reduction-in-breast-tissue-due-to-mammographic-compression/"><div class="card-body"> <em class="small" data-ts="1391187600" data-df="ll" > Jan 31, 2014 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>Blood Flow Reduction in Breast Tissue due to Mammographic Compression</h3><div class="text-muted small"><p> Rationale and objectives This study measures hemodynamic properties such as blood flow and hemoglobin concentration and oxygenation in the healthy human breast under a wide range of compressive lo...</p></div></div></a></div></div></div><div class="post-navigation d-flex justify-content-between"> <a href="/posts/the-pivotal-role-of-fdg-pet-ct-in-modern-medicine/" class="btn btn-outline-primary" prompt="Older"><p>The Pivotal Role of FDG-PET/CT in Modern Medicine</p></a> <a href="/posts/time-resolved-mri-oximetry-for-quantifying-cmro2-and-vascular-reactivity/" class="btn btn-outline-primary" prompt="Newer"><p>Time-Resolved MRI Oximetry for Quantifying CMRO2 and Vascular Reactivity</p></a></div></div></div></div><div id="search-result-wrapper" class="d-flex justify-content-center unloaded"><div class="col-12 col-sm-11 post-content"><div id="search-hints"><div id="access-tags"><div class="panel-heading">Trending Tags</div><div class="d-flex flex-wrap mt-3 mb-1 mr-3"> <a class="post-tag" href="/tags/general-radiology/">General Radiology</a> <a class="post-tag" href="/tags/journals/">Journals</a></div></div></div><div id="search-results" class="d-flex flex-wrap justify-content-center text-muted mt-3"></div></div></div></div><footer><div class="container pl-lg-4 pr-lg-4"><div class="d-flex justify-content-between align-items-center text-muted ml-md-3 mr-md-3"><div class="footer-left"><p class="mb-0"> © 2023 <a href="https://twitter.com/username">Clinical Team</a>. <span data-toggle="tooltip" data-placement="top" title="Except where otherwise noted, the blog posts on this site are licensed under the Creative Commons Attribution 4.0 International (CC BY 4.0) License by the author.">Some rights reserved.</span></p></div><div class="footer-right"><p class="mb-0">Using the <a href="https://jekyllrb.com" target="_blank" rel="noopener">Jekyll</a> theme <a href="https://github.com/cotes2020/jekyll-theme-chirpy" target="_blank" rel="noopener">Chirpy</a>.</p></div></div></div></footer><div id="mask"></div><a id="back-to-top" href="#" aria-label="back-to-top" class="btn btn-lg btn-box-shadow" role="button"> <i class="fas fa-angle-up"></i> </a><div id="notification" class="toast" role="alert" aria-live="assertive" aria-atomic="true" data-animation="true" data-autohide="false"><div class="toast-header"> <button type="button" class="ml-2 ml-auto close" data-dismiss="toast" aria-label="Close"> <span aria-hidden="true">&times;</span> </button></div><div class="toast-body text-center pt-0"><p class="pl-2 pr-2 mb-3">A new version of content is available.</p><button type="button" class="btn btn-primary" aria-label="Update"> Update </button></div></div><script src="https://cdn.jsdelivr.net/npm/simple-jekyll-search@1.10.0/dest/simple-jekyll-search.min.js"></script> <script> SimpleJekyllSearch({ searchInput: document.getElementById('search-input'), resultsContainer: document.getElementById('search-results'), json: '/assets/js/data/search.json', searchResultTemplate: '<div class="pl-1 pr-1 pl-sm-2 pr-sm-2 pl-lg-4 pr-lg-4 pl-xl-0 pr-xl-0"> <a href="{url}">{title}</a><div class="post-meta d-flex flex-column flex-sm-row text-muted mt-1 mb-1"> {categories} {tags}</div><p>{snippet}</p></div>', noResultsText: '<p class="mt-5">Oops! No results found.</p>', templateMiddleware: function(prop, value, template) { if (prop === 'categories') { if (value === '') { return `${value}`; } else { return `<div class="mr-sm-4"><i class="far fa-folder fa-fw"></i>${value}</div>`; } } if (prop === 'tags') { if (value === '') { return `${value}`; } else { return `<div><i class="fa fa-tag fa-fw"></i>${value}</div>`; } } } }); </script> <script src="https://cdn.jsdelivr.net/combine/npm/magnific-popup@1.1.0/dist/jquery.magnific-popup.min.js,npm/lazysizes@5.3.2/lazysizes.min.js,npm/clipboard@2.0.11/dist/clipboard.min.js"></script> <script src="https://cdn.jsdelivr.net/combine/npm/dayjs@1.11.6/dayjs.min.js,npm/dayjs@1.11.6/locale/en.min.js,npm/dayjs@1.11.6/plugin/relativeTime.min.js,npm/dayjs@1.11.6/plugin/localizedFormat.min.js"></script> <script defer src="/assets/js/dist/post.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/js/bootstrap.bundle.min.js"></script> <script defer src="/app.js"></script> <script defer src="https://www.googletagmanager.com/gtag/js?id=G-L66SLQK23K"></script> <script> document.addEventListener("DOMContentLoaded", function(event) { window.dataLayer = window.dataLayer || []; function gtag(){dataLayer.push(arguments);} gtag('js', new Date()); gtag('config', 'G-L66SLQK23K'); }); </script>
