<!DOCTYPE html><html lang="en" ><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="theme-color" media="(prefers-color-scheme: light)" content="#f7f7f7"><meta name="theme-color" media="(prefers-color-scheme: dark)" content="#1b1b1e"><meta name="apple-mobile-web-app-capable" content="yes"><meta name="apple-mobile-web-app-status-bar-style" content="black-translucent"><meta name="viewport" content="width=device-width, user-scalable=no initial-scale=1, shrink-to-fit=no, viewport-fit=cover" ><meta name="generator" content="Jekyll v4.3.2" /><meta property="og:title" content="An MRI-based Radiomics Classifier for Preoperative Prediction of Ki-67 Status in Breast Cancer" /><meta property="og:locale" content="en" /><meta name="description" content="Rationale and Objectives" /><meta property="og:description" content="Rationale and Objectives" /><link rel="canonical" href="https://clinicaltree.github.io/posts/an-mri-based-radiomics-classifier-for-preoperative-prediction-of-ki-67-status-in-breast-cancer/" /><meta property="og:url" content="https://clinicaltree.github.io/posts/an-mri-based-radiomics-classifier-for-preoperative-prediction-of-ki-67-status-in-breast-cancer/" /><meta property="og:site_name" content="Radiology Tree" /><meta property="og:type" content="article" /><meta property="article:published_time" content="2018-08-31T17:00:00+00:00" /><meta name="twitter:card" content="summary" /><meta property="twitter:title" content="An MRI-based Radiomics Classifier for Preoperative Prediction of Ki-67 Status in Breast Cancer" /><meta name="twitter:site" content="@twitter_username" /><meta name="google-site-verification" content="RFHVRgQqK0eGjftEMCTDhsDrR8cJ_ZYcfCX52gXW8KM" /> <script type="application/ld+json"> {"@context":"https://schema.org","@type":"BlogPosting","dateModified":"2023-04-10T03:59:08+00:00","datePublished":"2018-08-31T17:00:00+00:00","description":"Rationale and Objectives","headline":"An MRI-based Radiomics Classifier for Preoperative Prediction of Ki-67 Status in Breast Cancer","mainEntityOfPage":{"@type":"WebPage","@id":"https://clinicaltree.github.io/posts/an-mri-based-radiomics-classifier-for-preoperative-prediction-of-ki-67-status-in-breast-cancer/"},"url":"https://clinicaltree.github.io/posts/an-mri-based-radiomics-classifier-for-preoperative-prediction-of-ki-67-status-in-breast-cancer/"}</script><title>An MRI-based Radiomics Classifier for Preoperative Prediction of Ki-67 Status in Breast Cancer | Radiology Tree</title><link rel="apple-touch-icon" sizes="180x180" href="/assets/img/favicons/apple-touch-icon.png"><link rel="icon" type="image/png" sizes="32x32" href="/assets/img/favicons/favicon-32x32.png"><link rel="icon" type="image/png" sizes="16x16" href="/assets/img/favicons/favicon-16x16.png"><link rel="manifest" href="/assets/img/favicons/site.webmanifest"><link rel="shortcut icon" href="/assets/img/favicons/favicon.ico"><meta name="apple-mobile-web-app-title" content="Radiology Tree"><meta name="application-name" content="Radiology Tree"><meta name="msapplication-TileColor" content="#da532c"><meta name="msapplication-config" content="/assets/img/favicons/browserconfig.xml"><meta name="theme-color" content="#ffffff"><link rel="preconnect" href="https://fonts.googleapis.com" ><link rel="dns-prefetch" href="https://fonts.googleapis.com" ><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link rel="dns-prefetch" href="https://fonts.gstatic.com" crossorigin><link rel="preconnect" href="https://fonts.googleapis.com" ><link rel="dns-prefetch" href="https://fonts.googleapis.com" ><link rel="preconnect" href="https://cdn.jsdelivr.net" ><link rel="dns-prefetch" href="https://cdn.jsdelivr.net" ><link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Lato&family=Source+Sans+Pro:wght@400;600;700;900&display=swap"><link rel="preconnect" href="https://www.google-analytics.com" crossorigin="use-credentials"><link rel="dns-prefetch" href="https://www.google-analytics.com"><link rel="preconnect" href="https://www.googletagmanager.com" crossorigin="anonymous"><link rel="dns-prefetch" href="https://www.googletagmanager.com"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/css/bootstrap.min.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.2.1/css/all.min.css"><link rel="stylesheet" href="/assets/css/style.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/tocbot@4.20.1/dist/tocbot.min.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/magnific-popup@1.1.0/dist/magnific-popup.min.css"> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.1/dist/jquery.min.js"></script> <script type="text/javascript"> class ModeToggle { static get MODE_KEY() { return "mode"; } static get MODE_ATTR() { return "data-mode"; } static get DARK_MODE() { return "dark"; } static get LIGHT_MODE() { return "light"; } static get ID() { return "mode-toggle"; } constructor() { if (this.hasMode) { if (this.isDarkMode) { if (!this.isSysDarkPrefer) { this.setDark(); } } else { if (this.isSysDarkPrefer) { this.setLight(); } } } let self = this; /* always follow the system prefers */ this.sysDarkPrefers.addEventListener('change', () => { if (self.hasMode) { if (self.isDarkMode) { if (!self.isSysDarkPrefer) { self.setDark(); } } else { if (self.isSysDarkPrefer) { self.setLight(); } } self.clearMode(); } self.notify(); }); } /* constructor() */ get sysDarkPrefers() { return window.matchMedia("(prefers-color-scheme: dark)"); } get isSysDarkPrefer() { return this.sysDarkPrefers.matches; } get isDarkMode() { return this.mode === ModeToggle.DARK_MODE; } get isLightMode() { return this.mode === ModeToggle.LIGHT_MODE; } get hasMode() { return this.mode != null; } get mode() { return sessionStorage.getItem(ModeToggle.MODE_KEY); } /* get the current mode on screen */ get modeStatus() { if (this.isDarkMode || (!this.hasMode && this.isSysDarkPrefer)) { return ModeToggle.DARK_MODE; } else { return ModeToggle.LIGHT_MODE; } } setDark() { $('html').attr(ModeToggle.MODE_ATTR, ModeToggle.DARK_MODE); sessionStorage.setItem(ModeToggle.MODE_KEY, ModeToggle.DARK_MODE); } setLight() { $('html').attr(ModeToggle.MODE_ATTR, ModeToggle.LIGHT_MODE); sessionStorage.setItem(ModeToggle.MODE_KEY, ModeToggle.LIGHT_MODE); } clearMode() { $('html').removeAttr(ModeToggle.MODE_ATTR); sessionStorage.removeItem(ModeToggle.MODE_KEY); } /* Notify another plugins that the theme mode has changed */ notify() { window.postMessage({ direction: ModeToggle.ID, message: this.modeStatus }, "*"); } flipMode() { if (this.hasMode) { if (this.isSysDarkPrefer) { if (this.isLightMode) { this.clearMode(); } else { this.setLight(); } } else { if (this.isDarkMode) { this.clearMode(); } else { this.setDark(); } } } else { if (this.isSysDarkPrefer) { this.setLight(); } else { this.setDark(); } } this.notify(); } /* flipMode() */ } /* ModeToggle */ const modeToggle = new ModeToggle(); </script><body data-topbar-visible="true"><div id="sidebar" class="d-flex flex-column align-items-end"><div class="profile-wrapper text-center"><div id="avatar"> <a href="/" class="mx-auto"> <img src="https://storage.googleapis.com/clinicalpub.com/images/favicon.png" alt="avatar" onerror="this.style.display='none'"> </a></div><div class="site-title"> <a href="/">Radiology Tree</a></div><div class="site-subtitle font-italic">Update every day the best and the lastest articles, books, journals, clinical cases, videos, images... for radiologist</div></div><ul class="w-100"><li class="nav-item"> <a href="/" class="nav-link"> <i class="fa-fw fas fa-home ml-xl-3 mr-xl-3 unloaded"></i> <span>HOME</span> </a><li class="nav-item"> <a href="/categories/" class="nav-link"> <i class="fa-fw fas fa-stream ml-xl-3 mr-xl-3 unloaded"></i> <span>CATEGORIES</span> </a><li class="nav-item"> <a href="/tags/" class="nav-link"> <i class="fa-fw fas fa-tag ml-xl-3 mr-xl-3 unloaded"></i> <span>TAGS</span> </a><li class="nav-item"> <a href="/archives/" class="nav-link"> <i class="fa-fw fas fa-archive ml-xl-3 mr-xl-3 unloaded"></i> <span>ARCHIVES</span> </a><li class="nav-item"> <a href="/about/" class="nav-link"> <i class="fa-fw fas fa-info-circle ml-xl-3 mr-xl-3 unloaded"></i> <span>ABOUT</span> </a></ul><div class="sidebar-bottom mt-auto d-flex flex-wrap justify-content-center align-items-center"> <button class="mode-toggle btn" aria-label="Switch Mode"> <i class="fas fa-adjust"></i> </button> <span class="icon-border"></span> <a href="https://github.com/clinicaltree" aria-label="github" target="_blank" rel="noopener noreferrer"> <i class="fab fa-github"></i> </a> <a href="https://twitter.com/twitter_username" aria-label="twitter" target="_blank" rel="noopener noreferrer"> <i class="fab fa-twitter"></i> </a> <a href="javascript:location.href = 'mailto:' + ['clinicalpub.team','gmail.com'].join('@')" aria-label="email" > <i class="fas fa-envelope"></i> </a> <a href="/feed.xml" aria-label="rss" > <i class="fas fa-rss"></i> </a></div></div><div id="topbar-wrapper"><div id="topbar" class="container d-flex align-items-center justify-content-between h-100 pl-3 pr-3 pl-md-4 pr-md-4"> <span id="breadcrumb"> <span> <a href="/"> Home </a> </span> <span>An MRI-based Radiomics Classifier for Preoperative Prediction of Ki-67 Status in Breast Cancer</span> </span> <i id="sidebar-trigger" class="fas fa-bars fa-fw"></i><div id="topbar-title"> Post</div><i id="search-trigger" class="fas fa-search fa-fw"></i> <span id="search-wrapper" class="align-items-center"> <i class="fas fa-search fa-fw"></i> <input class="form-control" id="search-input" type="search" aria-label="search" autocomplete="off" placeholder="Search..."> </span> <span id="search-cancel" >Cancel</span></div></div><div id="main-wrapper" class="d-flex justify-content-center"><div id="main" class="container pl-xl-4 pr-xl-4"><div class="row"><div id="core-wrapper" class="col-12 col-lg-11 col-xl-9 pr-xl-4"><div class="post pl-1 pr-1 pl-md-2 pr-md-2"><h1 data-toc-skip>An MRI-based Radiomics Classifier for Preoperative Prediction of Ki-67 Status in Breast Cancer</h1><div class="post-meta text-muted"> <span> Posted <em class="" data-ts="1535734800" data-df="ll" data-toggle="tooltip" data-placement="bottom"> Aug 31, 2018 </em> </span> <span> Updated <em class="" data-ts="1681099148" data-df="ll" data-toggle="tooltip" data-placement="bottom"> Apr 10, 2023 </em> </span><div class="d-flex justify-content-between"> <span> By <em> <a href="">Cuishan Liang MD</a> </em>, <em> <a href="">Zixuan Cheng MPhil</a> </em>, <em> <a href="">Yanqi Huang MD</a> </em>, <em> <a href="">Lan He MPhil</a> </em>, <em> <a href="">Xin Chen MD</a> </em>, <em> <a href="">Zelan Ma MD</a> </em>, <em> <a href="">Xiaomei Huang MD</a> </em>, <em> <a href="">Chang-Hong Liang MD</a> </em>, <em> <a href="">Zaiyi Liu MD</a> </em> </span><div> <span class="readtime" data-toggle="tooltip" data-placement="bottom" title="2559 words"> <em>14 min</em> read</span></div></div></div><div class="post-content"><h2 id="rationale-and-objectives"><span class="mr-2">Rationale and Objectives</span><a href="#rationale-and-objectives" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p>This study aims to investigate the value of a magnetic resonance imaging–based radiomics classifier for preoperatively predicting the Ki-67 status in patients with breast cancer.</p><h2 id="materials-and-methods"><span class="mr-2">Materials and Methods</span><a href="#materials-and-methods" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p>We chronologically divided 318 patients with clinicopathologically confirmed breast cancer into a training dataset ( <em>n</em> = 200) and a validation dataset ( <em>n</em> = 118). Radiomics features were extracted from T2-weighted (T2W) and contrast-enhanced T1-weighted (T1+C) images of breast cancer. Radiomics feature selection and radiomics classifiers were generated using the least absolute shrinkage and selection operator regression analysis method. The correlation between the radiomics classifiers and the Ki-67 status in patients with breast cancer was explored. The predictive performances of the radiomics classifiers for the Ki-67 status were evaluated with receiver operating characteristic curves in the training dataset and validated in the validation dataset.</p><h2 id="results"><span class="mr-2">Results</span><a href="#results" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p>Through the radiomics feature selection, 16 and 14 features based on T2W and T1+C images, respectively, were selected to constitute the radiomics classifiers. The radiomics classifier based on T2W images was significantly correlated with the Ki-67 status in both the training and the validation datasets (both <em>P</em> &lt; .0001). The radiomics classifier based on T1+C images was significantly correlated with the Ki-67 status in the training dataset ( <em>P</em> &lt; .0001) but not in the validation dataset ( <em>P</em> = .083). The T2W image–based radiomics classifier exhibited good discrimination for Ki-67 status, with areas under the receiver operating characteristic curves of 0.762 (95% confidence interval: 0.685, 0.838) and 0.740 (95% confidence interval: 0.645, 0.836) in the training and validation datasets, respectively.</p><h2 id="conclusions"><span class="mr-2">Conclusions</span><a href="#conclusions" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p>The T2W image–based radiomics classifier was a significant predictor of Ki-67 status in patients with breast cancer. Thus, it may serve as a noninvasive approach to facilitate the preoperative prediction of Ki-67 status in clinical practice.</p><h2 id="introduction"><span class="mr-2">Introduction</span><a href="#introduction" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p>Breast cancer is the most common cancer in women, and the incidence rates are increasing worldwide . As personalized, precision medicine is becoming more prevalent, tumor biomarker assays are playing a more important role in guiding clinical care . Biomarker assay results can indicate the diagnosis and prognosis, as well as the need for monitoring for recurrence or progression in patients with breast cancer according to the American Society of Clinical Oncology guidelines . In particular, the Ki-67 labeling index is known to be a valuable prognostic marker in breast cancer . Moreover, the Ki-67 value is a significant indicator of potential triage to chemotherapy, whereas the Ki-67-based Preoperative Endocrine Prognostic Index is a feasible predictor of the risk of relapse . According to a study by Dowsett et al. , the predictive performance of the recurrence-free survival can be improved by measuring the tumor Ki-67 labeling index in patients undergoing short-term endocrine therapy. However, the accuracy of traditional invasive detection methods, that is, biopsy samples, is influenced by sampling errors, as they ignore intratumoral heterogeneity expression . In addition, research has shown that the Ki-67 status dynamically changes during treatment , and thus it is difficult to evaluate the Ki-67 status using only one biopsy. Therefore, an accurate, noninvasive predictor of the Ki-67 status in patients with breast cancer is clinically desirable.</p><p>Radiomics, which differs from the traditional practice of using medical images solely for visual interpretation, is the transition of digital medical images into mineable data through the high-throughput extraction of abundant quantitative features based on shape, intensity, size, or volume, among others . Radiomics data can be applied to build descriptive or predictive models that correlate quantitative image features with phenotypes or gene-protein markers, potentially assisting in the following: cancer detection, diagnosis, and staging; predicting the response to treatment; monitoring the disease status; and assessing the prognosis . Radiomics features have already been shown to aid in the diagnosis , molecular subtyping , prognosis, and prediction of the response to treatment in patients with breast cancer . Furthermore, in a study by Ha et al., 18 F-fluorodeoxyglucose positron emission tomography–based radiomics patterns of locally advanced breast cancer were associated with Ki-67 expression, the pathologic complete response to neoadjuvant chemotherapy, and the risk of recurrence, supporting the potential of such patterns to assist in the development of personalized disease management strategies .</p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="materials-and-methods-1"><span class="mr-2">Materials and Methods</span><a href="#materials-and-methods-1" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><h2 id="patients"><span class="mr-2">Patients</span><a href="#patients" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="ki-67-measurements"><span class="mr-2">Ki-67 Measurements</span><a href="#ki-67-measurements" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="mr-image-acquisition"><span class="mr-2">MR Image Acquisition</span><a href="#mr-image-acquisition" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="region-of-interest-delineation"><span class="mr-2">Region of Interest Delineation</span><a href="#region-of-interest-delineation" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="radiomics-feature-extraction"><span class="mr-2">Radiomics Feature Extraction</span><a href="#radiomics-feature-extraction" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="radiomics-feature-selection-and-classifier-building"><span class="mr-2">Radiomics Feature Selection and Classifier Building</span><a href="#radiomics-feature-selection-and-classifier-building" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p>X′=(X−X¯¯¯)/SD. X</p><p>′</p><p>=</p><p>(</p><p>X</p><p>−</p><p>X</p><p>¯</p><p>)</p><p>/</p><p>S</p><p>D</p><p>.</p><p>The least absolute shrinkage and selection operator (LASSO) logistic regression analysis method, which is suited to the regression of high-dimensional data, was used to perform radiomics feature selection in the training dataset. This method minimizes the sum of squares of residues, with the sum of the absolute values of the selected features coefficients being no more than a tuning parameter (λ) . As λ becomes smaller, the coefficients of more features are set to 0. Features that contributed well to the model were selected and used to construct a radiomics classifier when the <strong>area under the receiver operating characteristic (ROC) curve (AUC)</strong> of the binary logistic regression model was the largest. Finally, a radiomics score (Rad-score) was calculated for each patient through the linear combination of the selected features weighted by their respective coefficients.</p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="statistical-analyses"><span class="mr-2">Statistical Analyses</span><a href="#statistical-analyses" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><h2 id="patients-clinicopathologic-characteristics"><span class="mr-2">Patients’ Clinicopathologic Characteristics</span><a href="#patients-clinicopathologic-characteristics" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="performance-of-the-radiomics-classifiers"><span class="mr-2">Performance of the Radiomics Classifiers</span><a href="#performance-of-the-radiomics-classifiers" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="results-1"><span class="mr-2">Results</span><a href="#results-1" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><h2 id="patients-clinicopathologic-characteristics-1"><span class="mr-2">Patients’ Clinicopathologic Characteristics</span><a href="#patients-clinicopathologic-characteristics-1" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="radiomics-feature-selection-and-classifier-building-1"><span class="mr-2">Radiomics Feature Selection and Classifier Building</span><a href="#radiomics-feature-selection-and-classifier-building-1" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="performance-of-the-radiomics-classifiers-1"><span class="mr-2">Performance of the Radiomics Classifiers</span><a href="#performance-of-the-radiomics-classifiers-1" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p>TABLE 1</p><p>Radiomics Score (Rad-score) for Ki-67 Status in the Training and Validation Dataset</p><p>Rad-score Ki-67 (−) Median (IQR) Ki-67(+) Median (IQR)<em>P</em> T2 Training 0.841 (0.446, 1.155) 1.325 (0.963, 1.616) &lt;.0001 Validation 0.956 (0.711, 1.268) 1.390 (1.064, 1.691) &lt;.0001 T1+C Training 0.682 (0.301, 1.229) 1.397 (1.061, 1.654) &lt;.0001 Validation 0.956 (0.625, 1.448) 1.206 (0.836, 1.554) .083</p><p>IQR, interquartile range; Rad-score, radiomics score; T1+C, contrast-enhanced T1-weighted images; T2, T2-weighted images.</p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p>TABLE 2</p><p>The Predictive Performance of Radiomics Classifier Based on T2WI</p><p>Dataset Cutoff AUC (95% CI) SEN SPE Accuracy PPV NPV Training 1.044 0.762 (0.685,0.838) 0.720 0.700 0.715 0.878 0.455 Validation 1.044 0.740 (0.645,0.836) 0.759 0.645 0.729 0.857 0.488</p><p>95% CI, 95% confidence interval; AUC, area under curve; NPV, negative predictive value; PPV, positive predictive value; SEN, sensitivity; SPE, specificity.</p><p><a href="https://storage.googleapis.com/dl.dentistrykey.com/clinical/AnMRIbasedRadiomicsClassifierforPreoperativePredictionofKi67StatusinBreastCancer/0_1s20S1076633218300187.jpg" class="popup img-link "><img data-src="https://storage.googleapis.com/dl.dentistrykey.com/clinical/AnMRIbasedRadiomicsClassifierforPreoperativePredictionofKi67StatusinBreastCancer/0_1s20S1076633218300187.jpg" alt="Figure 1, The radiomics score (Rad-score) for each patient with regard to the classification of Ki-67 status in the training and validation datasets. The graphs in (a) and (b) represent the Rad-score distributions based on T2W images in the training and validation datasets, respectively. The graphs in (c) and (d) represent the Rad-score distributions based on T1+C images in the training and validation datasets, respectively. The blue triangles indicate Ki-67-positive patients, whereas the red dots indicate Ki-67-negative patients. The solid line suggests the best cutoff of radiomics classifier for the discrimination of Ki-67-positive and Ki-67-negative patients, below which patients are classified to be Ki-67-negative patients, whereas above which patients are classified to be Ki-67-positive patients. The cutoff value is 1.044 on T2WI and 1.081 on T1+C images. (Color version of figure is available online.)" class="lazyload" data-proofer-ignore></a></p><p><a href="https://storage.googleapis.com/dl.dentistrykey.com/clinical/AnMRIbasedRadiomicsClassifierforPreoperativePredictionofKi67StatusinBreastCancer/1_1s20S1076633218300187.jpg" class="popup img-link "><img data-src="https://storage.googleapis.com/dl.dentistrykey.com/clinical/AnMRIbasedRadiomicsClassifierforPreoperativePredictionofKi67StatusinBreastCancer/1_1s20S1076633218300187.jpg" alt="Figure 2, Receiver operating characteristic curves of T2W images in the training and validation datasets. AUC, area under curve." class="lazyload" data-proofer-ignore></a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="discussion"><span class="mr-2">Discussion</span><a href="#discussion" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="acknowledgments"><span class="mr-2">Acknowledgments</span><a href="#acknowledgments" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="supplementary-data"><span class="mr-2">Supplementary Data</span><a href="#supplementary-data" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p>Appendix S1</p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="references"><span class="mr-2">References</span><a href="#references" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><ul><li><p>1. Bray F., Jemal A., Grey N., et. al.: Global cancer transitions according to the Human Development Index (2008–2030): a population-based study. Lancet Oncol 2012; 13: pp. 790-801.</p><li><p>2. Torre L.A., Bray F., Siegel R.L., et. al.: Global cancer statistics, 2012. CA Cancer J Clin 2015; 65: pp. 87-108.</p><li><p>3. Harris L.N., Ismaila N., McShane L.M., et. al.: Use of biomarkers to guide decisions on adjuvant systemic therapy for women with early-stage invasive breast cancer: American Society of Clinical Oncology Clinical Practice Guideline. J Clin Oncol 2016; 34: pp. 1134-1150.</p><li><p>4. Harris L., Fritsche H., Mennel R., et. al.: American Society of Clinical Oncology 2007 update of recommendations for the use of tumor markers in breast cancer. J Clin Oncol 2007; 25: pp. 5287-5312.</p><li><p>5. Viale G., Giobbie-Hurder A., Regan M.M., et. al.: Prognostic and predictive value of centrally reviewed Ki-67 labeling index in postmenopausal women with endocrine-responsive breast cancer: results from Breast International Group Trial 1–98 comparing adjuvant tamoxifen with letrozole. J Clin Oncol 2008; 26: pp. 5569-5575.</p><li><p>6. Ellis M.J., Suman V.J., Hoog J., et. al.: Ki67 proliferation index as a tool for chemotherapy decisions during and after neoadjuvant aromatase inhibitor treatment of breast cancer: results from the American College of Surgeons Oncology Group Z1031 Trial (Alliance). J Clin Oncol 2017; 35: pp. 1061-1069.</p><li><p>7. Dowsett M., Smith I.E., Ebbs S.R., et. al.: Prognostic value of Ki67 expression after short-term presurgical endocrine therapy for primary breast cancer. J Natl Cancer Inst 2007; 99: pp. 167-170.</p><li><p>8. Diwakar N., Sperandio M., Sherriff M., et. al.: Heterogeneity, histological features and DNA ploidy in oral carcinoma by image-based analysis. Oral Oncol 2005; 41: pp. 416-422.</p><li><p>9. Gillies R.J., Kinahan P.E., Hricak H.: Radiomics: images are more than pictures, they are data. Radiology 2016; 278: pp. 563-577.</p><li><p>10. Lambin P., Rios-Velazquez E., Leijenaar R., et. al.: Radiomics: extracting more information from medical images using advanced feature analysis. Eur J Cancer 2012; 48: pp. 441-446.</p><li><p>11. Huang Y., Liu Z., He L., et. al.: Radiomics signature: a potential biomarker for the prediction of disease-free survival in early-stage (I or II) non-small cell lung cancer. Radiology 2016; 281: pp. 947-957.</p><li><p>12. Huang Y.Q., Liang C.H., He L., et. al.: Development and validation of a radiomics nomogram for preoperative prediction of lymph node metastasis in colorectal cancer. J Clin Oncol 2016; 34: pp. 2157-2164.</p><li><p>13. Liang C., Huang Y., He L., et. al.: The development and validation of a CT-based radiomics signature for the preoperative discrimination of stage I-II and stage III-IV colorectal cancer. Oncotarget 2016; 7: pp. 31401-31412.</p><li><p>14. Bickelhaupt S., Paech D., Kickingereder P., et. al.: Prediction of malignancy by a radiomic signature from contrast agent-free diffusion MRI in suspicious breast lesions found on screening mammography. J Magn Reson Imaging 2017; 46: pp. 604-616.</p><li><p>15. Li H., Zhu Y., Burnside E.S., et. al.: Quantitative MRI radiomics in the prediction of molecular classifications of breast cancer subtypes in the TCGA/TCIA data set. NPJ Breast Cancer 2016; 2:</p><li><p>16. Li H., Zhu Y., Burnside E.S., et. al.: MR imaging radiomics signatures for predicting the risk of breast cancer recurrence as given by research versions of MammaPrint, Oncotype DX, and PAM50 gene assays. Radiology 2016; 281: pp. 382-391.</p><li><p>17. Ha S., Park S., Bang J.I., et. al.: Metabolic radiomics for pretreatment 18F-FDG PET/CT to characterize locally advanced breast cancer: histopathologic characteristics, response to neoadjuvant chemotherapy, and prognosis. Sci Rep 2017; 7: pp. 1556.</p><li><p>18. Cuzick J., Dowsett M., Pineda S., et. al.: Prognostic value of a combined estrogen receptor, progesterone receptor, Ki-67, and human epidermal growth factor receptor 2 immunohistochemical score and comparison with the Genomic Health recurrence score in early breast cancer. J Clin Oncol 2011; 29: pp. 4273-4278.</p><li><p>19. Goldhirsch A., Wood W.C., Coates A.S., et. al.: Strategies for subtypes—dealing with the diversity of breast cancer: highlights of the St. Gallen International Expert Consensus on the Primary Therapy of Early Breast Cancer 2011. Ann Oncol 2011; 22: pp. 1736-1747.</p><li><p>20. Sasikala V., LakshmiPrabha V.: A comparative study on the swarm intelligence based feature selection approaches for fake and real fingerprint classification. 2015 International Conference on Soft-Computing and Networks Security (ICSNS)2015.pp. 1-8.</p><li><p>21. Aerts H.J., Velazquez E.R., Leijenaar R.T., et. al.: Decoding tumour phenotype by noninvasive imaging using a quantitative radiomics approach. Nat Commun 2014; 5: pp. 4006.</p><li><p>22. Osborne M.R., Presnell B., Turlach B.A.: On the LASSO and its dual. J Comput Graph Stat 2000; 9: pp. 319-337.</p><li><p>23. Stathopoulos G.P., Malamos N.A., Markopoulos C., et. al.: The role of Ki-67 in the proliferation and prognosis of breast cancer molecular classification subtypes. Anticancer Drugs 2014; 25: pp. 950-957.</p><li><p>24. Porschen R., Lohe B., Hengels K.J., et. al.: Assessment of cell proliferation in colorectal carcinomas using the monoclonal antibody Ki-67. Correlation with pathohistologic criteria and influence of irradiation. Cancer 1989; 64: pp. 2501-2505.</p><li><p>25. Besusparis J., Plancoulaine B., Rasmusson A., et. al.: Impact of tissue sampling on accuracy of Ki67 immunohistochemistry evaluation in breast cancer. Diagn Pathol 2016; 11: pp. 82.</p><li><p>26. Boros M., Moncea D., Moldovan C., et. al.: Intratumoral heterogeneity for Ki-67 index in invasive breast carcinoma: a study on 131 consecutive cases. Appl Immunohistochem Mol Morphol 2017; 25: pp. 338-340.</p><li><p>27. Ng F., Ganeshan B., Kozarski R., et. al.: Assessment of primary colorectal cancer heterogeneity by using whole-tumor texture analysis: contrast-enhanced CT texture as a biomarker of 5-year survival. Radiology 2013; 266: pp. 177-184.</p><li><p>28. Yun B.L., Cho N., Li M., et. al.: Intratumoral heterogeneity of breast cancer xenograft models: texture analysis of diffusion-weighted MR imaging. Korean J Radiol 2014; 15: pp. 591-604.</p><li><p>29. Sala E., Mema E., Himoto Y., et. al.: Unravelling tumour heterogeneity using next-generation imaging: radiomics, radiogenomics, and habitat imaging. Clin Radiol 2017; 72: pp. 3-10.</p><li><p>30. Santamaria G., Velasco M., Bargallo X., et. al.: Radiologic and pathologic findings in breast tumors with high signal intensity on T2-weighted MR images. Radiographics 2010; 30: pp. 533-548.</p><li><p>31. Velasco M., Santamaria G., Ganau S., et. al.: MRI of metaplastic carcinoma of the breast. AJR Am J Roentgenol 2005; 184: pp. 1274-1278.</p><li><p>32. Kuhl C.: The current status of breast MR imaging. Part I. Choice of technique, image interpretation, diagnostic accuracy, and transfer to clinical practice. Radiology 2007; 244: pp. 356-378.</p><li><p>33. Schieda N., Coffey N., Gulavita P., et. al.: Prostatic ductal adenocarcinoma: an aggressive tumour variant unrecognized on T2 weighted magnetic resonance imaging (MRI). Eur Radiol 2014; 24: pp. 1349-1356.</p><li><p>34. Jha R.C., Zanello P.A., Nguyen X.M., et. al.: Small hepatocellular carcinoma: MRI findings for predicting tumor growth rates. Acad Radiol 2014; 21: pp. 1455-1464.</p><li><p>35. Iglesias A., Arias M., Santiago P., et. al.: Benign breast lesions that simulate malignancy: magnetic resonance imaging with radiologic-pathologic correlation. Curr Probl Diagn Radiol 2007; 36: pp. 66-82.</p><li><p>36. Huang J., Hahn T., Hoisington L., et. al.: Improving suspicious breast lesion characterization using semi-automatic lesion fractional volume washout kinetic analysis. Med Phys 2011; 38: pp. 5998-6009.</p><li><p>37. Mann R.M., Mus R.D., van Zelst J., et. al.: A novel approach to contrast-enhanced breast magnetic resonance imaging for screening: high-resolution ultrafast dynamic imaging. Invest Radiol 2014; 49: pp. 579-585.</p><li><p>38. Kuhl C.K.: Abbreviated breast MRI for screening women with dense breast: the EA1141 trial. Br J Radiol 2017; pp. 20170441.</p><li><p>39. Petrillo A., Fusco R., Sansone M., et. al.: Abbreviated breast dynamic contrast-enhanced MR imaging for lesion detection and characterization: the experience of an Italian oncologic center. Breast Cancer Res Treat 2017; 164: pp. 401-410.</p><li><p>40. Lubner M.G., Stabo N., Lubner S.J., et. al.: CT textural analysis of hepatic metastatic colorectal cancer: pre-treatment tumor heterogeneity correlates with pathology and clinical outcomes. Abdom Imaging 2015; 40: pp. 2331-2337.</p></ul></div><div class="post-tail-wrapper text-muted"><div class="post-meta mb-3"> <i class="far fa-folder-open fa-fw mr-1"></i> <a href='/categories/academic-radiology/'>Academic Radiology</a>, <a href='/categories/volume-25/'>Volume 25</a>, <a href='/categories/issue-9/'>Issue 9</a></div><div class="post-tags"> <i class="fa fa-tags fa-fw mr-1"></i> <a href="/tags/journals/" class="post-tag no-text-decoration" >Journals</a> <a href="/tags/general-radiology/" class="post-tag no-text-decoration" >General Radiology</a></div><div class="post-tail-bottom d-flex justify-content-between align-items-center mt-3 pt-5 pb-2"><div class="license-wrapper"> This post is licensed under <a href="https://creativecommons.org/licenses/by/4.0/"> CC BY 4.0 </a> by the author.</div><div class="share-wrapper"> <span class="share-label text-muted mr-1">Share</span> <span class="share-icons"> <a href="https://twitter.com/intent/tweet?text=An%20MRI-based%20Radiomics%20Classifier%20for%20Preoperative%20Prediction%20of%20Ki-67%20Status%20in%20Breast%20Cancer%20-%20Radiology%20Tree&url=https%3A%2F%2Fclinicaltree.github.io%2Fposts%2Fan-mri-based-radiomics-classifier-for-preoperative-prediction-of-ki-67-status-in-breast-cancer%2F" data-toggle="tooltip" data-placement="top" title="Twitter" target="_blank" rel="noopener" aria-label="Twitter"> <i class="fa-fw fab fa-twitter"></i> </a> <a href="https://www.facebook.com/sharer/sharer.php?title=An%20MRI-based%20Radiomics%20Classifier%20for%20Preoperative%20Prediction%20of%20Ki-67%20Status%20in%20Breast%20Cancer%20-%20Radiology%20Tree&u=https%3A%2F%2Fclinicaltree.github.io%2Fposts%2Fan-mri-based-radiomics-classifier-for-preoperative-prediction-of-ki-67-status-in-breast-cancer%2F" data-toggle="tooltip" data-placement="top" title="Facebook" target="_blank" rel="noopener" aria-label="Facebook"> <i class="fa-fw fab fa-facebook-square"></i> </a> <a href="https://t.me/share/url?url=https%3A%2F%2Fclinicaltree.github.io%2Fposts%2Fan-mri-based-radiomics-classifier-for-preoperative-prediction-of-ki-67-status-in-breast-cancer%2F&text=An%20MRI-based%20Radiomics%20Classifier%20for%20Preoperative%20Prediction%20of%20Ki-67%20Status%20in%20Breast%20Cancer%20-%20Radiology%20Tree" data-toggle="tooltip" data-placement="top" title="Telegram" target="_blank" rel="noopener" aria-label="Telegram"> <i class="fa-fw fab fa-telegram"></i> </a> <i id="copy-link" class="fa-fw fas fa-link small" data-toggle="tooltip" data-placement="top" title="Copy link" data-title-succeed="Link copied successfully!"> </i> </span></div></div></div></div></div><div id="panel-wrapper" class="col-xl-3 pl-2 text-muted"><div class="access"><div id="access-lastmod" class="post"><div class="panel-heading">Recently Updated</div><ul class="post-content pl-0 pb-1 ml-1 mt-2"><li><a href="/posts/neurometabolites-alteration-in-the-acute-phase-of-mild-traumatic-brain-injury-mtbi/">Neurometabolites Alteration in the Acute Phase of Mild Traumatic Brain Injury (mTBI)</a><li><a href="/posts/reinforcing-the-importance-and-feasibility-of-implementing-a-low-dose-protocol-for-ct-guided-biopsie/">Reinforcing the Importance and Feasibility of Implementing a Low-dose Protocol for CT-guided Biopsies</a><li><a href="/posts/rethinking-the-pgy-1-basic-clinical-year/">Rethinking the PGY-1 Basic Clinical Year</a><li><a href="/posts/single-injection-dual-phase-cone-beam-ct-dp-cbct-vascular-anatomy-assessment-and-occult-nodule-det/">Single Injection Dual-Phase Cone Beam CT (DP-CBCT) Vascular Anatomy Assessment and Occult Nodule Detection; Have We Reached the Focus?</a><li><a href="/posts/the-yellow-scale-is-superior-to-the-gray-scale-for-detecting-acute-ischemic-stroke-on-a-monitor-disp/">The Yellow Scale Is Superior to the Gray Scale for Detecting Acute Ischemic Stroke on a Monitor Display in Computed Tomography</a></ul></div><div id="access-tags"><div class="panel-heading">Trending Tags</div><div class="d-flex flex-wrap mt-3 mb-1 mr-3"> <a class="post-tag" href="/tags/general-radiology/">General Radiology</a> <a class="post-tag" href="/tags/journals/">Journals</a></div></div></div><div id="toc-wrapper" class="pl-0 pr-4 mb-5"><div class="panel-heading pl-3 pt-2 mb-2">Contents</div><nav id="toc"></nav></div><script src="https://cdn.jsdelivr.net/npm/tocbot@4.20.1/dist/tocbot.min.js"></script></div></div><div class="row"><div id="tail-wrapper" class="col-12 col-lg-11 col-xl-9 pl-3 pr-3 pr-xl-4 mt-5"><div id="related-posts" class="mb-2 mb-sm-4"><h3 class="pt-2 mb-4 ml-1" data-toc-skip>Further Reading</h3><div class="card-deck mb-4"><div class="card"> <a href="/posts/neurometabolites-alteration-in-the-acute-phase-of-mild-traumatic-brain-injury-mtbi/"><div class="card-body"> <em class="small" data-ts="1535734800" data-df="ll" > Aug 31, 2018 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>Neurometabolites Alteration in the Acute Phase of Mild Traumatic Brain Injury (mTBI)</h3><div class="text-muted small"><p> Rationale and Objectives Magnetic resonance spectroscopy is a noninvasive imaging technique that allows for reliable assessment of microscopic changes in brain cytoarchitecture, neuronal injuries,...</p></div></div></a></div><div class="card"> <a href="/posts/reinforcing-the-importance-and-feasibility-of-implementing-a-low-dose-protocol-for-ct-guided-biopsie/"><div class="card-body"> <em class="small" data-ts="1535734800" data-df="ll" > Aug 31, 2018 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>Reinforcing the Importance and Feasibility of Implementing a Low-dose Protocol for CT-guided Biopsies</h3><div class="text-muted small"><p> Rationale and Objectives This study sought to more definitely illustrate the impact and feasibility of implementing a low-dose protocol for computed tomography (CT)-guided biopsies using size-spec...</p></div></div></a></div><div class="card"> <a href="/posts/rethinking-the-pgy-1-basic-clinical-year/"><div class="card-body"> <em class="small" data-ts="1535734800" data-df="ll" > Aug 31, 2018 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>Rethinking the PGY-1 Basic Clinical Year</h3><div class="text-muted small"><p> Rationale and Objectives Recently, the relevance of the postgraduate year 1 (PGY-1) Basic Clinical Year for radiology residents has been questioned. The purpose of this study was to determine the ...</p></div></div></a></div></div></div><div class="post-navigation d-flex justify-content-between"> <a href="/posts/a-dynamic-graph-cuts-method-with-integrated-multiple-feature-maps-for-segmenting-kidneys-in-2d-ultra/" class="btn btn-outline-primary" prompt="Older"><p>A Dynamic Graph Cuts Method with Integrated Multiple Feature Maps for Segmenting Kidneys in 2D Ultrasound Images</p></a> <a href="/posts/association-of-focal-radiation-dose-adjusted-on-cross-sections-with-subsolid-nodule-visibility-and-q/" class="btn btn-outline-primary" prompt="Newer"><p>Association of Focal Radiation Dose Adjusted on Cross Sections with Subsolid Nodule Visibility and Quantification on Computed Tomography Images Using AIDR 3D</p></a></div></div></div></div><div id="search-result-wrapper" class="d-flex justify-content-center unloaded"><div class="col-12 col-sm-11 post-content"><div id="search-hints"><div id="access-tags"><div class="panel-heading">Trending Tags</div><div class="d-flex flex-wrap mt-3 mb-1 mr-3"> <a class="post-tag" href="/tags/general-radiology/">General Radiology</a> <a class="post-tag" href="/tags/journals/">Journals</a></div></div></div><div id="search-results" class="d-flex flex-wrap justify-content-center text-muted mt-3"></div></div></div></div><footer><div class="container pl-lg-4 pr-lg-4"><div class="d-flex justify-content-between align-items-center text-muted ml-md-3 mr-md-3"><div class="footer-left"><p class="mb-0"> © 2023 <a href="https://twitter.com/username">Clinical Team</a>. <span data-toggle="tooltip" data-placement="top" title="Except where otherwise noted, the blog posts on this site are licensed under the Creative Commons Attribution 4.0 International (CC BY 4.0) License by the author.">Some rights reserved.</span></p></div><div class="footer-right"><p class="mb-0">Using the <a href="https://jekyllrb.com" target="_blank" rel="noopener">Jekyll</a> theme <a href="https://github.com/cotes2020/jekyll-theme-chirpy" target="_blank" rel="noopener">Chirpy</a>.</p></div></div></div></footer><div id="mask"></div><a id="back-to-top" href="#" aria-label="back-to-top" class="btn btn-lg btn-box-shadow" role="button"> <i class="fas fa-angle-up"></i> </a><div id="notification" class="toast" role="alert" aria-live="assertive" aria-atomic="true" data-animation="true" data-autohide="false"><div class="toast-header"> <button type="button" class="ml-2 ml-auto close" data-dismiss="toast" aria-label="Close"> <span aria-hidden="true">&times;</span> </button></div><div class="toast-body text-center pt-0"><p class="pl-2 pr-2 mb-3">A new version of content is available.</p><button type="button" class="btn btn-primary" aria-label="Update"> Update </button></div></div><script src="https://cdn.jsdelivr.net/npm/simple-jekyll-search@1.10.0/dest/simple-jekyll-search.min.js"></script> <script> SimpleJekyllSearch({ searchInput: document.getElementById('search-input'), resultsContainer: document.getElementById('search-results'), json: '/assets/js/data/search.json', searchResultTemplate: '<div class="pl-1 pr-1 pl-sm-2 pr-sm-2 pl-lg-4 pr-lg-4 pl-xl-0 pr-xl-0"> <a href="{url}">{title}</a><div class="post-meta d-flex flex-column flex-sm-row text-muted mt-1 mb-1"> {categories} {tags}</div><p>{snippet}</p></div>', noResultsText: '<p class="mt-5">Oops! No results found.</p>', templateMiddleware: function(prop, value, template) { if (prop === 'categories') { if (value === '') { return `${value}`; } else { return `<div class="mr-sm-4"><i class="far fa-folder fa-fw"></i>${value}</div>`; } } if (prop === 'tags') { if (value === '') { return `${value}`; } else { return `<div><i class="fa fa-tag fa-fw"></i>${value}</div>`; } } } }); </script> <script src="https://cdn.jsdelivr.net/combine/npm/magnific-popup@1.1.0/dist/jquery.magnific-popup.min.js,npm/lazysizes@5.3.2/lazysizes.min.js,npm/clipboard@2.0.11/dist/clipboard.min.js"></script> <script src="https://cdn.jsdelivr.net/combine/npm/dayjs@1.11.6/dayjs.min.js,npm/dayjs@1.11.6/locale/en.min.js,npm/dayjs@1.11.6/plugin/relativeTime.min.js,npm/dayjs@1.11.6/plugin/localizedFormat.min.js"></script> <script defer src="/assets/js/dist/post.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/js/bootstrap.bundle.min.js"></script> <script defer src="/app.js"></script> <script defer src="https://www.googletagmanager.com/gtag/js?id=G-L66SLQK23K"></script> <script> document.addEventListener("DOMContentLoaded", function(event) { window.dataLayer = window.dataLayer || []; function gtag(){dataLayer.push(arguments);} gtag('js', new Date()); gtag('config', 'G-L66SLQK23K'); }); </script>
