<!DOCTYPE html><html lang="en" ><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="theme-color" media="(prefers-color-scheme: light)" content="#f7f7f7"><meta name="theme-color" media="(prefers-color-scheme: dark)" content="#1b1b1e"><meta name="apple-mobile-web-app-capable" content="yes"><meta name="apple-mobile-web-app-status-bar-style" content="black-translucent"><meta name="viewport" content="width=device-width, user-scalable=no initial-scale=1, shrink-to-fit=no, viewport-fit=cover" ><meta name="generator" content="Jekyll v4.3.2" /><meta property="og:title" content="Comparing Mammography Abnormality Features to Genetic Variants in the Prediction of Breast Cancer in Women Recommended for Breast Biopsy" /><meta property="og:locale" content="en" /><meta name="description" content="Rationale and Objectives" /><meta property="og:description" content="Rationale and Objectives" /><link rel="canonical" href="https://clinicaltree.github.io/posts/comparing-mammography-abnormality-features-to-genetic-variants-in-the-prediction-of-breast-cancer-in/" /><meta property="og:url" content="https://clinicaltree.github.io/posts/comparing-mammography-abnormality-features-to-genetic-variants-in-the-prediction-of-breast-cancer-in/" /><meta property="og:site_name" content="Radiology Tree" /><meta property="og:type" content="article" /><meta property="article:published_time" content="2015-12-31T17:00:00+00:00" /><meta name="twitter:card" content="summary" /><meta property="twitter:title" content="Comparing Mammography Abnormality Features to Genetic Variants in the Prediction of Breast Cancer in Women Recommended for Breast Biopsy" /><meta name="twitter:site" content="@twitter_username" /><meta name="google-site-verification" content="RFHVRgQqK0eGjftEMCTDhsDrR8cJ_ZYcfCX52gXW8KM" /> <script type="application/ld+json"> {"@context":"https://schema.org","@type":"BlogPosting","dateModified":"2023-04-10T03:59:08+00:00","datePublished":"2015-12-31T17:00:00+00:00","description":"Rationale and Objectives","headline":"Comparing Mammography Abnormality Features to Genetic Variants in the Prediction of Breast Cancer in Women Recommended for Breast Biopsy","mainEntityOfPage":{"@type":"WebPage","@id":"https://clinicaltree.github.io/posts/comparing-mammography-abnormality-features-to-genetic-variants-in-the-prediction-of-breast-cancer-in/"},"url":"https://clinicaltree.github.io/posts/comparing-mammography-abnormality-features-to-genetic-variants-in-the-prediction-of-breast-cancer-in/"}</script><title>Comparing Mammography Abnormality Features to Genetic Variants in the Prediction of Breast Cancer in Women Recommended for Breast Biopsy | Radiology Tree</title><link rel="apple-touch-icon" sizes="180x180" href="/assets/img/favicons/apple-touch-icon.png"><link rel="icon" type="image/png" sizes="32x32" href="/assets/img/favicons/favicon-32x32.png"><link rel="icon" type="image/png" sizes="16x16" href="/assets/img/favicons/favicon-16x16.png"><link rel="manifest" href="/assets/img/favicons/site.webmanifest"><link rel="shortcut icon" href="/assets/img/favicons/favicon.ico"><meta name="apple-mobile-web-app-title" content="Radiology Tree"><meta name="application-name" content="Radiology Tree"><meta name="msapplication-TileColor" content="#da532c"><meta name="msapplication-config" content="/assets/img/favicons/browserconfig.xml"><meta name="theme-color" content="#ffffff"><link rel="preconnect" href="https://fonts.googleapis.com" ><link rel="dns-prefetch" href="https://fonts.googleapis.com" ><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link rel="dns-prefetch" href="https://fonts.gstatic.com" crossorigin><link rel="preconnect" href="https://fonts.googleapis.com" ><link rel="dns-prefetch" href="https://fonts.googleapis.com" ><link rel="preconnect" href="https://cdn.jsdelivr.net" ><link rel="dns-prefetch" href="https://cdn.jsdelivr.net" ><link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Lato&family=Source+Sans+Pro:wght@400;600;700;900&display=swap"><link rel="preconnect" href="https://www.google-analytics.com" crossorigin="use-credentials"><link rel="dns-prefetch" href="https://www.google-analytics.com"><link rel="preconnect" href="https://www.googletagmanager.com" crossorigin="anonymous"><link rel="dns-prefetch" href="https://www.googletagmanager.com"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/css/bootstrap.min.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.2.1/css/all.min.css"><link rel="stylesheet" href="/assets/css/style.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/tocbot@4.20.1/dist/tocbot.min.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/magnific-popup@1.1.0/dist/magnific-popup.min.css"> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.1/dist/jquery.min.js"></script> <script type="text/javascript"> class ModeToggle { static get MODE_KEY() { return "mode"; } static get MODE_ATTR() { return "data-mode"; } static get DARK_MODE() { return "dark"; } static get LIGHT_MODE() { return "light"; } static get ID() { return "mode-toggle"; } constructor() { if (this.hasMode) { if (this.isDarkMode) { if (!this.isSysDarkPrefer) { this.setDark(); } } else { if (this.isSysDarkPrefer) { this.setLight(); } } } let self = this; /* always follow the system prefers */ this.sysDarkPrefers.addEventListener('change', () => { if (self.hasMode) { if (self.isDarkMode) { if (!self.isSysDarkPrefer) { self.setDark(); } } else { if (self.isSysDarkPrefer) { self.setLight(); } } self.clearMode(); } self.notify(); }); } /* constructor() */ get sysDarkPrefers() { return window.matchMedia("(prefers-color-scheme: dark)"); } get isSysDarkPrefer() { return this.sysDarkPrefers.matches; } get isDarkMode() { return this.mode === ModeToggle.DARK_MODE; } get isLightMode() { return this.mode === ModeToggle.LIGHT_MODE; } get hasMode() { return this.mode != null; } get mode() { return sessionStorage.getItem(ModeToggle.MODE_KEY); } /* get the current mode on screen */ get modeStatus() { if (this.isDarkMode || (!this.hasMode && this.isSysDarkPrefer)) { return ModeToggle.DARK_MODE; } else { return ModeToggle.LIGHT_MODE; } } setDark() { $('html').attr(ModeToggle.MODE_ATTR, ModeToggle.DARK_MODE); sessionStorage.setItem(ModeToggle.MODE_KEY, ModeToggle.DARK_MODE); } setLight() { $('html').attr(ModeToggle.MODE_ATTR, ModeToggle.LIGHT_MODE); sessionStorage.setItem(ModeToggle.MODE_KEY, ModeToggle.LIGHT_MODE); } clearMode() { $('html').removeAttr(ModeToggle.MODE_ATTR); sessionStorage.removeItem(ModeToggle.MODE_KEY); } /* Notify another plugins that the theme mode has changed */ notify() { window.postMessage({ direction: ModeToggle.ID, message: this.modeStatus }, "*"); } flipMode() { if (this.hasMode) { if (this.isSysDarkPrefer) { if (this.isLightMode) { this.clearMode(); } else { this.setLight(); } } else { if (this.isDarkMode) { this.clearMode(); } else { this.setDark(); } } } else { if (this.isSysDarkPrefer) { this.setLight(); } else { this.setDark(); } } this.notify(); } /* flipMode() */ } /* ModeToggle */ const modeToggle = new ModeToggle(); </script><body data-topbar-visible="true"><div id="sidebar" class="d-flex flex-column align-items-end"><div class="profile-wrapper text-center"><div id="avatar"> <a href="/" class="mx-auto"> <img src="https://storage.googleapis.com/clinicalpub.com/images/favicon.png" alt="avatar" onerror="this.style.display='none'"> </a></div><div class="site-title"> <a href="/">Radiology Tree</a></div><div class="site-subtitle font-italic">Update every day the best and the lastest articles, books, journals, clinical cases, videos, images... for radiologist</div></div><ul class="w-100"><li class="nav-item"> <a href="/" class="nav-link"> <i class="fa-fw fas fa-home ml-xl-3 mr-xl-3 unloaded"></i> <span>HOME</span> </a><li class="nav-item"> <a href="/categories/" class="nav-link"> <i class="fa-fw fas fa-stream ml-xl-3 mr-xl-3 unloaded"></i> <span>CATEGORIES</span> </a><li class="nav-item"> <a href="/tags/" class="nav-link"> <i class="fa-fw fas fa-tag ml-xl-3 mr-xl-3 unloaded"></i> <span>TAGS</span> </a><li class="nav-item"> <a href="/archives/" class="nav-link"> <i class="fa-fw fas fa-archive ml-xl-3 mr-xl-3 unloaded"></i> <span>ARCHIVES</span> </a><li class="nav-item"> <a href="/about/" class="nav-link"> <i class="fa-fw fas fa-info-circle ml-xl-3 mr-xl-3 unloaded"></i> <span>ABOUT</span> </a></ul><div class="sidebar-bottom mt-auto d-flex flex-wrap justify-content-center align-items-center"> <button class="mode-toggle btn" aria-label="Switch Mode"> <i class="fas fa-adjust"></i> </button> <span class="icon-border"></span> <a href="https://github.com/clinicaltree" aria-label="github" target="_blank" rel="noopener noreferrer"> <i class="fab fa-github"></i> </a> <a href="https://twitter.com/twitter_username" aria-label="twitter" target="_blank" rel="noopener noreferrer"> <i class="fab fa-twitter"></i> </a> <a href="javascript:location.href = 'mailto:' + ['clinicalpub.team','gmail.com'].join('@')" aria-label="email" > <i class="fas fa-envelope"></i> </a> <a href="/feed.xml" aria-label="rss" > <i class="fas fa-rss"></i> </a></div></div><div id="topbar-wrapper"><div id="topbar" class="container d-flex align-items-center justify-content-between h-100 pl-3 pr-3 pl-md-4 pr-md-4"> <span id="breadcrumb"> <span> <a href="/"> Home </a> </span> <span>Comparing Mammography Abnormality Features to Genetic Variants in the Prediction of Breast Cancer in Women Recommended for Breast Biopsy</span> </span> <i id="sidebar-trigger" class="fas fa-bars fa-fw"></i><div id="topbar-title"> Post</div><i id="search-trigger" class="fas fa-search fa-fw"></i> <span id="search-wrapper" class="align-items-center"> <i class="fas fa-search fa-fw"></i> <input class="form-control" id="search-input" type="search" aria-label="search" autocomplete="off" placeholder="Search..."> </span> <span id="search-cancel" >Cancel</span></div></div><div id="main-wrapper" class="d-flex justify-content-center"><div id="main" class="container pl-xl-4 pr-xl-4"><div class="row"><div id="core-wrapper" class="col-12 col-lg-11 col-xl-9 pr-xl-4"><div class="post pl-1 pr-1 pl-md-2 pr-md-2"><h1 data-toc-skip>Comparing Mammography Abnormality Features to Genetic Variants in the Prediction of Breast Cancer in Women Recommended for Breast Biopsy</h1><div class="post-meta text-muted"> <span> Posted <em class="" data-ts="1451581200" data-df="ll" data-toggle="tooltip" data-placement="bottom"> Dec 31, 2015 </em> </span> <span> Updated <em class="" data-ts="1681099148" data-df="ll" data-toggle="tooltip" data-placement="bottom"> Apr 10, 2023 </em> </span><div class="d-flex justify-content-between"> <span> By <em> <a href="">Elizabeth S. Burnside MD MPH MS</a> </em>, <em> <a href="">Jie Liu PhD</a> </em>, <em> <a href="">Yirong Wu PhD</a> </em>, <em> <a href="">Adedayo A. Onitilo MD MSCR FACP</a> </em>, <em> <a href="">Catherine A. McCarty PhD</a> </em>, <em> <a href="">C. David Page PhD</a> </em>, <em> <a href="">Peggy L. Peissig PhD</a> </em>, <em> <a href="">Amy Trentham-Dietz PhD</a> </em>, <em> <a href="">Terrie Kitchner</a> </em>, <em> <a href="">Jun Fan PhD</a> </em>, <em> <a href="">Ming Yuan PhD</a> </em> </span><div> <span class="readtime" data-toggle="tooltip" data-placement="bottom" title="2377 words"> <em>13 min</em> read</span></div></div></div><div class="post-content"><h2 id="rationale-and-objectives"><span class="mr-2">Rationale and Objectives</span><a href="#rationale-and-objectives" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p>The discovery of germline genetic variants associated with breast cancer has engendered interest in risk stratification for improved, targeted detection and diagnosis. However, there has yet to be a comparison of the predictive ability of these genetic variants with mammography abnormality descriptors.</p><h2 id="materials-and-methods"><span class="mr-2">Materials and Methods</span><a href="#materials-and-methods" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p>Our institutional review board-approved, Health Insurance Portability and Accountability Act-compliant study utilized a personalized medicine registry in which participants consented to provide a DNA sample and to participate in longitudinal follow-up. In our retrospective, age-matched, case-controlled study of 373 cases and 395 controls who underwent breast biopsy, we collected risk factors selected a priori based on the literature, including demographic variables based on the Gail model, common germline genetic variants, and diagnostic mammography findings according to Breast Imaging Reporting and Data System (BI-RADS). We developed predictive models using logistic regression to determine the predictive ability of (1) demographic variables, (2) 10 selected genetic variants, or (3) mammography BI-RADS features. We evaluated each model in turn by calculating a risk score for each patient using 10-fold cross-validation, used this risk estimate to construct Receiver Operator Characteristic Curve (ROC) curves, and compared the area under the ROC curve (AUC) of each using the DeLong method.</p><h2 id="results"><span class="mr-2">Results</span><a href="#results" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p>The performance of the regression model using demographic risk factors was not statistically different from the model using genetic variants ( <em>P</em> = 0.9). The model using mammography features (AUC = 0.689) was superior to both the demographic model (AUC = .598; <em>P</em> &lt; 0.001) and the genetic model (AUC = .601; <em>P</em> &lt; 0.001).</p><h2 id="conclusions"><span class="mr-2">Conclusions</span><a href="#conclusions" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p>BI-RADS features exceeded the ability of demographic and 10 selected germline genetic variants to predict breast cancer in women recommended for biopsy.</p><h2 id="introduction"><span class="mr-2">Introduction</span><a href="#introduction" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p>Over the last several decades, predictive variables have been discovered and incorporated into risk prediction models with the goal of personalizing breast cancer screening and diagnosis. One highly predictive source of information is abnormality level feature descriptors observed on mammography as described in the Breast Imaging Reporting and Data System (BI-RADS) . Other emerging sources are the ever-growing genome-wide association studies (GWAS) that identify genetic variants (single nucleotide polymorphisms [SNPs]). The SNPs discovered via recent GWAS are distinct from mutations in the BRCA1 and BRCA2 tumor suppressor genes . Although both are germline genetic risk factors (inherited from parental lineage), SNPs discovered in recent GWAS are single base pair DNA sequence variations conferring modest risk (low penetrance) but occurring commonly (high frequency) within the human population. Expansion of genetic risk prediction may depend on polygenic risk stratification, that is, weighing many high-frequency, low-penetrance SNPs at once . Early attempts to use such SNPs to predict breast cancer risk have demonstrated only modest improvements over conventional demographic risk factors, like those in the Gail model .</p><p>Breast cancer risk is determined by a combination of genetic and environmental factors. Intermediate phenotypes like imaging can capture and convey these interactions of these risk factors and provide biomarkers that can augment comprehensive risk prediction. Because demographic risk factors, genetic variants, and imaging features will all likely have some level of predictive value, determining which variables provide the best predictive power in any given setting becomes extremely important. Investing limited resources in collection of the best predictive variables will provide the most benefit. Prior literature evaluated risk prediction with genetics and breast density and one paper added BI-RADS assessment category . Despite the proven predictive ability of abnormality-level features described in the BI-RADS lexicon (e.g. mass and calcification descriptors as well as associated findings like architectural distortion), comparison to demographic or genetic risk has been limited. To estimate breast cancer risk in women recommended for breast biopsy, we compare the performance of predictive models using distinct data elements: demographic risk factors, germline genetic variants, or mammography abnormality features.</p><h2 id="materials-and-methods-1"><span class="mr-2">Materials and Methods</span><a href="#materials-and-methods-1" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><h2 id="subjects"><span class="mr-2">Subjects</span><a href="#subjects" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="epidemiological-risk-factors"><span class="mr-2">Epidemiological Risk Factors</span><a href="#epidemiological-risk-factors" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="genetic-variants"><span class="mr-2">Genetic Variants</span><a href="#genetic-variants" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p>Table 1</p><p>Common Genetic Variants Associated with Breast Cancer</p><p>SNPs Chromosome Gene High-risk Allele Low-risk Allele RS1045485 2q CASP8 G C RS13281615 8q Unknown G A RS13387042 2q Unknown A G RS2981582 10q FGFR2 T C RS3803662 16q TOX3 T C RS3817198 11p LSP1 C T RS889312 5q MAP3K1 C A RS10941679 5p Unknown G A RS999737 14q RAD51L1 C T RS11249433 1p Unknown C T</p><p>SNP, single nucleotide polymorphism.</p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="mammography-features"><span class="mr-2">Mammography Features</span><a href="#mammography-features" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="statistical-analysis"><span class="mr-2">Statistical Analysis</span><a href="#statistical-analysis" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p>Table 2</p><p>Demographic Variables, Genetic Factors, and Mammographic Features in the Predictive Models \*</p><p>Variable Cases</p><p>( <em>N</em> = 373) Controls</p><p>( <em>N</em> = 395) OR 95% CI ¥<em>P</em> Value ¥ Age at menarche ≥14 105 (28.2%) 84 (21.2%) Referent 12–13 189 (50.7%) 224 (56.6%) 0.63 (0.33,1.21) 0.15 7–11 79 (21.1%) 88 (22.2%) 0.73 (0.29,1.83) 0.46 Number of biopsies 0 303 (81.2%) 337 (85.6%) Referent 1 60 (16.0%) 52 (13.1%) 1.35 (0.84,2.16) 0.21 ≥2 10 (2.6%) 6 (1.5%) 2.89 (0.95,8.75) 0.061 Number of pregnancies ≥6 52 (14.0%) 47 (12.0%) Referent 3–5 164 (43.9%) 175 (44.4%) 1.09 (0.63,1.89) 0.76 1–2 126 (33.8%) 129 (32.7%) 1.18 (0.66,2.13) 0.57 0 31 (8.3%) 43 (10.9%) 0.72 (0.34,1.51) 0.38 Number of first-degree relatives with breast cancer 0 268 (71.8%) 325 (82.2%) Referent 1 91 (24.4%) 57 (14.4%)<strong>2.05</strong><strong>(1.34,3.15)</strong><strong>0.0010</strong> ≥2 14 (3.7%) 13 (3.2%) 1.68 (0.70,4.04) 0.24 Number of risk-conferring variant alleles 0–6 26 (6.9%) 58 (14.6%) Referent 7 or 8 97 (26.0%) 126 (31.9%)<strong>1.92</strong><strong>(1.04,3.55)</strong><strong>0.037</strong> 9 or 10 135 (36.1%) 126 (31.9%)<strong>2.69</strong><strong>(1.46,4.95)</strong><strong>0.0016</strong> 11 or 12 90 (24.1%) 73 (18.4%)<strong>2.74</strong><strong>(1.41,5.30)</strong><strong>0.0029</strong> ≥13 25 (6.7%) 12 (3.0%)<strong>4.95</strong><strong>(1.89,12.96)</strong><strong>0.0012</strong> Breast density Fatty 24 (6.5%) 26 (6.5%) Referent Scattered 30 (8.1%) 58 (14.6%) 0.46 (0.12,1.80) 0.24 Heterogeneous 293 (78.6%) 298 (75.4%) 0.99 (0.25,3.92) 0.98 Extremely dense 25 (6.8%) 14 (3.4%) 2.47 (0.27,22.27) 0.37 Mass margin Circumscribed 16 (4.2%) 36 (9.1%)<strong>0.42</strong><strong>(0.21,0.85)</strong><strong>0.016</strong> Obscured 8 (2.1%) 14 (3.5%) 0.59 (0.22,1.58) 0.29 Ill defined 47 (12.6%) 48 (12.1%) 1.42 (0.87,2.31) 0.16 Spiculated 82 (21.9%) 4 (1.0%)<strong>27.56</strong><strong>(9.64,78.78)</strong><strong>&lt;0.001</strong> Suspicious microcalcification shape 63 (16.8%) 79 (20.0%) 1.49 (0.82,2.72) 0.19 Suspicious microcalcification distribution 79 (21.1%) 117 (29.6%)<strong>0.56</strong><strong>(0.33,0.94)</strong><strong>0.029</strong> Architectural distortion 50 (13.4%) 21 (5.3%)<strong>2.32</strong><strong>(1.25,4.28)</strong><strong>0.0073</strong></p><p>¥ Bold signifies statistically significant variables.</p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="assessment-of-model-performance"><span class="mr-2">Assessment of Model Performance</span><a href="#assessment-of-model-performance" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="institutional-review-board-irb-approvals"><span class="mr-2">Institutional Review Board (IRB) Approvals</span><a href="#institutional-review-board-irb-approvals" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="results-1"><span class="mr-2">Results</span><a href="#results-1" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://storage.googleapis.com/dl.dentistrykey.com/clinical/ComparingMammographyAbnormalityFeaturestoGeneticVariantsinthePredictionofBreastCancerinWomenRecommendedforBreastBiopsy/0_1s20S107663321500416X.jpg" class="popup img-link "><img data-src="https://storage.googleapis.com/dl.dentistrykey.com/clinical/ComparingMammographyAbnormalityFeaturestoGeneticVariantsinthePredictionofBreastCancerinWomenRecommendedforBreastBiopsy/0_1s20S107663321500416X.jpg" alt="Figure 1, Plot shows the year that included biopsies (cases and controls) were performed, from which corresponding diagnostic mammography examinations were identified. Date ranges of the sequential editions of the Breast Imaging Reporting and Data System (BI-RADS) lexicon are demarcated to illustrate the evolution of the lexicon noting that utilization invariably lags dissemination. Prior to the publication of the BI-RADS lexicon, standardized descriptors were also available in the scientific literature. (28)" class="lazyload" data-proofer-ignore></a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p>Table 3</p><p>Frequency of Microcalcification Descriptors in Cases and Controls</p><p>Control Case Total Microcalcification shape Benign descriptor 33 (53%) 29 (47%) 62 Amorphous 10 (83%) 2 (17%) 12 Pleomorphic 70 (54%) 59 (46%) 129 Fine linear 0 (0%) 5 (100%) 5 Total 113 95 208 Microcalcification distribution Benign descriptor 11 (52%) 10 (48%) 21 Grouped 115 (62%) 71 (38%) 186 Segmental 0 (0%) 6 (100%) 6 Linear 18 (46%) 21 (54%) 39 Total 144 108 252</p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="model-performance"><span class="mr-2">Model Performance</span><a href="#model-performance" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://storage.googleapis.com/dl.dentistrykey.com/clinical/ComparingMammographyAbnormalityFeaturestoGeneticVariantsinthePredictionofBreastCancerinWomenRecommendedforBreastBiopsy/1_1s20S107663321500416X.jpg" class="popup img-link "><img data-src="https://storage.googleapis.com/dl.dentistrykey.com/clinical/ComparingMammographyAbnormalityFeaturestoGeneticVariantsinthePredictionofBreastCancerinWomenRecommendedforBreastBiopsy/1_1s20S107663321500416X.jpg" alt="Figure 2, Receiver operating characteristic curves for the three models that were compared." class="lazyload" data-proofer-ignore></a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="discussion"><span class="mr-2">Discussion</span><a href="#discussion" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="funding"><span class="mr-2">Funding</span><a href="#funding" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="acknowledgments"><span class="mr-2">Acknowledgments</span><a href="#acknowledgments" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="appendix"><span class="mr-2">Appendix</span><a href="#appendix" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://storage.googleapis.com/dl.dentistrykey.com/clinical/ComparingMammographyAbnormalityFeaturestoGeneticVariantsinthePredictionofBreastCancerinWomenRecommendedforBreastBiopsy/2_1s20S107663321500416X.jpg" class="popup img-link "><img data-src="https://storage.googleapis.com/dl.dentistrykey.com/clinical/ComparingMammographyAbnormalityFeaturestoGeneticVariantsinthePredictionofBreastCancerinWomenRecommendedforBreastBiopsy/2_1s20S107663321500416X.jpg" alt="Figure A1, Utilization of BI-RADS descriptors for cases and controls over the time of study. (Color version of figure available online)." class="lazyload" data-proofer-ignore></a></p><p><a href="https://storage.googleapis.com/dl.dentistrykey.com/clinical/ComparingMammographyAbnormalityFeaturestoGeneticVariantsinthePredictionofBreastCancerinWomenRecommendedforBreastBiopsy/3_1s20S107663321500416X.jpg" class="popup img-link "><img data-src="https://storage.googleapis.com/dl.dentistrykey.com/clinical/ComparingMammographyAbnormalityFeaturestoGeneticVariantsinthePredictionofBreastCancerinWomenRecommendedforBreastBiopsy/3_1s20S107663321500416X.jpg" alt="Figure A2, Utilization of BI-RADS descriptors for cases over the time of study. (Color version of figure available online)." class="lazyload" data-proofer-ignore></a></p><p><a href="https://storage.googleapis.com/dl.dentistrykey.com/clinical/ComparingMammographyAbnormalityFeaturestoGeneticVariantsinthePredictionofBreastCancerinWomenRecommendedforBreastBiopsy/4_1s20S107663321500416X.jpg" class="popup img-link "><img data-src="https://storage.googleapis.com/dl.dentistrykey.com/clinical/ComparingMammographyAbnormalityFeaturestoGeneticVariantsinthePredictionofBreastCancerinWomenRecommendedforBreastBiopsy/4_1s20S107663321500416X.jpg" alt="Figure A3, Utilization of BI-RADS descriptors for controls over the time of study. (Color version of figure available online)." class="lazyload" data-proofer-ignore></a></p><p>Table A1</p><p>Performance of MAMMOGRAPHIC Models Using Progressively More Recent Subsets of the Data</p><p>Time Frame_n_ AUC 1989–2010 768 .693 1993–2010 743 .701 1998–2010 662 .711</p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="references"><span class="mr-2">References</span><a href="#references" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><ul><li><p>1. Gail M.H., Brinton L.A., Byar D.P., et. al.: Projecting individualized probabilities of developing breast cancer for white females who are being examined annually. J Natl Cancer Inst 1989; 81: pp. 1879-1886.</p><li><p>2. Barlow W.E., White E., Ballard-Barbash R., et. al.: Prospective breast cancer risk prediction model for women undergoing screening mammography. J Natl Cancer Inst 2006; 98: pp. 1204-1214.</p><li><p>3. Pashayan N., Duffy S.W., Chowdhury S., et. al.: Polygenic susceptibility to prostate and breast cancer: implications for personalised screening. Br J Cancer 2011; 104: pp. 1656-1663.</p><li><p>4. Baker J.A., Kornguth P.J., Lo J.Y., et. al.: Breast cancer: prediction with artificial neural network based on BI-RADS® standardized lexicon. Radiology 1995; 196: pp. 817-822.</p><li><p>5. Liberman L., Abramson A.F., Squires F.B., et. al.: The breast imaging reporting and data system: positive predictive value of mammographic features and final assessment categories. AJR Am J Roentgenol 1998; 171: pp. 35-40.</p><li><p>6. Burnside E.S., Rubin D.L., Fine J.P., et. al.: Bayesian network to predict breast cancer risk of mammographic microcalcifications and reduce number of benign biopsy results: initial experience. Radiology 2006; 240: pp. 666-673.</p><li><p>7. Chhatwal J., Alagoz O., Lindstrom M.J., et. al.: A logistic regression model based on the national mammography database format to aid breast cancer diagnosis. AJR Am J Roentgenol 2009; 192: pp. 1117-1127.</p><li><p>8. Timmers J.M., Verbeek A.L., IntHout J., et. al.: Breast cancer risk prediction model: a nomogram based on common mammographic screening findings. Eur Radiol 2013; 23: pp. 2413-2419.</p><li><p>9. Linger R.J., Kruk P.A.: BRCA1 16 years later: risk-associated BRCA1 mutations and their functional implications. FEBS J 2010; 277: pp. 3086-3096.</p><li><p>10. Pharoah P.D., Antoniou A.C., Easton D.F., et. al.: Polygenes, risk prediction, and targeted prevention of breast cancer. N Engl J Med 2008; 358: pp. 2796-2803.</p><li><p>11. Wacholder S., Hartge P., Prentice R., et. al.: Performance of common genetic variants in breast-cancer risk models. N Engl J Med 2010; 362: pp. 986-993.</p><li><p>12. Gail M.H.: Discriminatory accuracy from single-nucleotide polymorphisms in models to predict breast cancer risk. J Natl Cancer Inst 2008; 100: pp. 1037-1041.</p><li><p>13. Gail M.H.: Value of adding single-nucleotide polymorphism genotypes to a breast cancer risk model. J Natl Cancer Inst 2009; 101: pp. 959-963.</p><li><p>14. Devilee P., Rookus M.A.: A tiny step closer to personalized risk prediction for breast cancer. N Engl J Med 2010; 362: pp. 1043-1045.</p><li><p>15. Tamimi R.M., Cox D., Kraft P., et. al.: Breast cancer susceptibility loci and mammographic density. Breast Cancer Res 2008; 10: pp. R66.</p><li><p>16. Darabi H., Czene K., Zhao W., et. al.: Breast cancer risk prediction and individualised screening based on common genetic variation and breast density measurement. Breast Cancer Res 2012; 14: pp. R25.</p><li><p>17. Armstrong K., Handorf E.A., Chen J., et. al.: Breast cancer risk prediction and mammography biopsy decisions: a model-based study. Am J Prev Med 2013; 44: pp. 15-22.</p><li><p>18. McCarty C.A., Wilke R.A., Giampietro P.F., et. al.: Marshfield Clinic Personalized Medicine Research Project (PMRP): design, methods and recruitment for a large population-based biobank. Personalized Med 2005; 2: pp. 49-79.</p><li><p>19. Wu Y., Alagoz O., Ayvaci M.U., et. al.: A comprehensive methodology for determining the most informative mammographic features. J Digit Imaging 2013; 26: pp. 941-947.</p><li><p>20. Kobayashi S., Sugiura H., Ando Y., et. al.: Reproductive history and breast cancer risk. Breast Cancer 2012; 19: pp. 302-308.</p><li><p>21. McCarty C.A., Chisholm R.L., Chute C.G., et. al.: The eMERGE Network: a consortium of biorepositories linked to electronic medical records data for conducting genomic studies. BMC Med Genomics 2011; 4: pp. 13.</p><li><p>22. Easton D.F., Pooley K.A., Dunning A.M., et. al.: Genome-wide association study identifies novel breast cancer susceptibility loci. Nature 2007; 447: pp. 1087-1093.</p><li><p>23. Hunter D.J., Kraft P., Jacobs K.B., et. al.: A genome-wide association study identifies alleles in FGFR2 associated with risk of sporadic postmenopausal breast cancer. Nat Genet 2007; 39: pp. 870-874.</p><li><p>24. American College of Radiology : 4th ed.2003.American College of RadiologyReston VA</p><li><p>25. American College of Radiology : 3rd ed.1998.American College of RadiologyReston VA</p><li><p>26. Houssam N., Ryan W., Elizabeth B., et. al.: Information Extraction for Clinical Data Mining: A Mammography Case Study. In: Proceedings of the 2009 IEEE International Conference on Data Mining Workshops: IEEE Computer Society2009.</p><li><p>27. Percha B., Nassif H., Lipson J., et. al.: Automatic classification of mammography reports by BI-RADS breast tissue composition class. J Am Med Inform Assoc 2012; 19: pp. 913-916.</p><li><p>28. Swets J.A.: Measuring the accuracy of diagnostic systems. Science 1988; 240: pp. 1285-1293.</p><li><p>29. Burnside E.S., Sickles E.A., Bassett L.W., et. al.: The ACR BI-RADS experience: learning from history. J Am Coll Radiol 2009; 6: pp. 851-860.</p><li><p>30. Rubin D.B.: Multiple imputation for nonresponse in surveys.1987.John Wiley &amp; SonsNew York</p><li><p>31. van Buuren S.: Flexible imputation of missing data.CRC/Chapman &amp; Hall Interdisciplinary Statistics Series2012.Chapman and Hall/CRCBoca Raton, FL</p><li><p>32. van Buuren S., Oudshoorn C.G.M.: Multivariate Imputation by Chained Equations MICE V1.0 User’s manual. Leiden, The Netherlands: TNO Prevention and Health; June2000. Report No.: 90-6743-677-1</p><li><p>33. van Buuren S., Groothuis-Oudshoorn K.: mice: Multivariate Imputation by Chained Equations in R. J Stat Softw 2011; 45: pp. 1-67.</p><li><p>34. Van Buuren S., Oudshoorn C.G.M.: Multivariate imputation by chained equations: MICE V1.0 user’s manual.2000.TNO Prevention and HealthLeiden</p><li><p>35. DeLong E.R., DeLong D.M., Clarke-Pearson D.L.: Comparing the areas under two or more correlated receiver operating characteristic curves: a nonparametric approach. Biometrics 1988; 44: pp. 837-845.</p><li><p>36. R Core Team : 2013.R Foundation for Statistical ComputingVienna, Austria [3.0.1]</p><li><p>37. Tice J.A., Cummings S.R., Smith-Bindman R., et. al.: Using clinical factors and mammographic breast density to estimate breast cancer risk: development and validation of a new predictive model. Ann Intern Med 2008; 148: pp. 337-347.</p><li><p>38. Berg W.A., Arnoldus C.L., Teferra E., et. al.: Biopsy of amorphous breast calcifications: pathologic outcome and yield at stereotactic biopsy. Radiology 2001; 221: pp. 495-503.</p><li><p>39. Burnside E.S., Ochsner J.E., Fowler K.J., et. al.: Use of microcalcification descriptors in BI-RADS 4th edition to stratify risk of malignancy. Radiology 2007; 242: pp. 388-395.</p><li><p>40. Michailidou K., Hall P., Gonzalez-Neira A., et. al.: Large-scale genotyping identifies 41 new loci associated with breast cancer risk. Nat Genet 2013; 45: pp. 353-361. 361e1-2</p></ul></div><div class="post-tail-wrapper text-muted"><div class="post-meta mb-3"> <i class="far fa-folder-open fa-fw mr-1"></i> <a href='/categories/academic-radiology/'>Academic Radiology</a>, <a href='/categories/volume-23/'>Volume 23</a>, <a href='/categories/issue-1/'>Issue 1</a></div><div class="post-tags"> <i class="fa fa-tags fa-fw mr-1"></i> <a href="/tags/journals/" class="post-tag no-text-decoration" >Journals</a> <a href="/tags/general-radiology/" class="post-tag no-text-decoration" >General Radiology</a></div><div class="post-tail-bottom d-flex justify-content-between align-items-center mt-3 pt-5 pb-2"><div class="license-wrapper"> This post is licensed under <a href="https://creativecommons.org/licenses/by/4.0/"> CC BY 4.0 </a> by the author.</div><div class="share-wrapper"> <span class="share-label text-muted mr-1">Share</span> <span class="share-icons"> <a href="https://twitter.com/intent/tweet?text=Comparing%20Mammography%20Abnormality%20Features%20to%20Genetic%20Variants%20in%20the%20Prediction%20of%20Breast%20Cancer%20in%20Women%20Recommended%20for%20Breast%20Biopsy%20-%20Radiology%20Tree&url=https%3A%2F%2Fclinicaltree.github.io%2Fposts%2Fcomparing-mammography-abnormality-features-to-genetic-variants-in-the-prediction-of-breast-cancer-in%2F" data-toggle="tooltip" data-placement="top" title="Twitter" target="_blank" rel="noopener" aria-label="Twitter"> <i class="fa-fw fab fa-twitter"></i> </a> <a href="https://www.facebook.com/sharer/sharer.php?title=Comparing%20Mammography%20Abnormality%20Features%20to%20Genetic%20Variants%20in%20the%20Prediction%20of%20Breast%20Cancer%20in%20Women%20Recommended%20for%20Breast%20Biopsy%20-%20Radiology%20Tree&u=https%3A%2F%2Fclinicaltree.github.io%2Fposts%2Fcomparing-mammography-abnormality-features-to-genetic-variants-in-the-prediction-of-breast-cancer-in%2F" data-toggle="tooltip" data-placement="top" title="Facebook" target="_blank" rel="noopener" aria-label="Facebook"> <i class="fa-fw fab fa-facebook-square"></i> </a> <a href="https://t.me/share/url?url=https%3A%2F%2Fclinicaltree.github.io%2Fposts%2Fcomparing-mammography-abnormality-features-to-genetic-variants-in-the-prediction-of-breast-cancer-in%2F&text=Comparing%20Mammography%20Abnormality%20Features%20to%20Genetic%20Variants%20in%20the%20Prediction%20of%20Breast%20Cancer%20in%20Women%20Recommended%20for%20Breast%20Biopsy%20-%20Radiology%20Tree" data-toggle="tooltip" data-placement="top" title="Telegram" target="_blank" rel="noopener" aria-label="Telegram"> <i class="fa-fw fab fa-telegram"></i> </a> <i id="copy-link" class="fa-fw fas fa-link small" data-toggle="tooltip" data-placement="top" title="Copy link" data-title-succeed="Link copied successfully!"> </i> </span></div></div></div></div></div><div id="panel-wrapper" class="col-xl-3 pl-2 text-muted"><div class="access"><div id="access-lastmod" class="post"><div class="panel-heading">Recently Updated</div><ul class="post-content pl-0 pb-1 ml-1 mt-2"><li><a href="/posts/neurometabolites-alteration-in-the-acute-phase-of-mild-traumatic-brain-injury-mtbi/">Neurometabolites Alteration in the Acute Phase of Mild Traumatic Brain Injury (mTBI)</a><li><a href="/posts/reinforcing-the-importance-and-feasibility-of-implementing-a-low-dose-protocol-for-ct-guided-biopsie/">Reinforcing the Importance and Feasibility of Implementing a Low-dose Protocol for CT-guided Biopsies</a><li><a href="/posts/rethinking-the-pgy-1-basic-clinical-year/">Rethinking the PGY-1 Basic Clinical Year</a><li><a href="/posts/single-injection-dual-phase-cone-beam-ct-dp-cbct-vascular-anatomy-assessment-and-occult-nodule-det/">Single Injection Dual-Phase Cone Beam CT (DP-CBCT) Vascular Anatomy Assessment and Occult Nodule Detection; Have We Reached the Focus?</a><li><a href="/posts/the-yellow-scale-is-superior-to-the-gray-scale-for-detecting-acute-ischemic-stroke-on-a-monitor-disp/">The Yellow Scale Is Superior to the Gray Scale for Detecting Acute Ischemic Stroke on a Monitor Display in Computed Tomography</a></ul></div><div id="access-tags"><div class="panel-heading">Trending Tags</div><div class="d-flex flex-wrap mt-3 mb-1 mr-3"> <a class="post-tag" href="/tags/general-radiology/">General Radiology</a> <a class="post-tag" href="/tags/journals/">Journals</a></div></div></div><div id="toc-wrapper" class="pl-0 pr-4 mb-5"><div class="panel-heading pl-3 pt-2 mb-2">Contents</div><nav id="toc"></nav></div><script src="https://cdn.jsdelivr.net/npm/tocbot@4.20.1/dist/tocbot.min.js"></script></div></div><div class="row"><div id="tail-wrapper" class="col-12 col-lg-11 col-xl-9 pl-3 pr-3 pr-xl-4 mt-5"><div id="related-posts" class="mb-2 mb-sm-4"><h3 class="pt-2 mb-4 ml-1" data-toc-skip>Further Reading</h3><div class="card-deck mb-4"><div class="card"> <a href="/posts/an-aneurysm-by-any-other-name/"><div class="card-body"> <em class="small" data-ts="1451581200" data-df="ll" > Dec 31, 2015 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>An Aneurysm by Any Other Name</h3><div class="text-muted small"><p> I applaud Amber et al for their piece in the August issue of Academic Radiology in which they explore the impact of diagnosing small intracranial aneurysms . The topic is truly deserving of attenti...</p></div></div></a></div><div class="card"> <a href="/posts/anisotropy-of-solid-breast-lesions-in-2d-shear-wave-elastography-is-an-indicator-of-malignancy/"><div class="card-body"> <em class="small" data-ts="1451581200" data-df="ll" > Dec 31, 2015 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>Anisotropy of Solid Breast Lesions in 2D Shear Wave Elastography is an Indicator of Malignancy</h3><div class="text-muted small"><p> Rationale and Objectives To investigate if anisotropy at two-dimensional shear wave elastography (SWE) suggests malignancy and whether it correlates with prognostic and predictive factors in breas...</p></div></div></a></div><div class="card"> <a href="/posts/big-data-and-the-future-of-radiology-informatics/"><div class="card-body"> <em class="small" data-ts="1451581200" data-df="ll" > Dec 31, 2015 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>Big Data and the Future of Radiology Informatics</h3><div class="text-muted small"><p> Rapid growth in the amount of data that is electronically recorded as part of routine clinical operations has generated great interest in the use of Big Data methodologies to address clinical and r...</p></div></div></a></div></div></div><div class="post-navigation d-flex justify-content-between"> <a href="/posts/clinical-radiology-and-radiology-research-in-a-sea-of-change/" class="btn btn-outline-primary" prompt="Older"><p>Clinical Radiology and Radiology Research in a Sea of Change</p></a> <a href="/posts/gender-bias-in-diagnostic-radiology-resident-selection-does-it-exist/" class="btn btn-outline-primary" prompt="Newer"><p>Gender Bias in Diagnostic Radiology Resident Selection, Does it Exist?</p></a></div></div></div></div><div id="search-result-wrapper" class="d-flex justify-content-center unloaded"><div class="col-12 col-sm-11 post-content"><div id="search-hints"><div id="access-tags"><div class="panel-heading">Trending Tags</div><div class="d-flex flex-wrap mt-3 mb-1 mr-3"> <a class="post-tag" href="/tags/general-radiology/">General Radiology</a> <a class="post-tag" href="/tags/journals/">Journals</a></div></div></div><div id="search-results" class="d-flex flex-wrap justify-content-center text-muted mt-3"></div></div></div></div><footer><div class="container pl-lg-4 pr-lg-4"><div class="d-flex justify-content-between align-items-center text-muted ml-md-3 mr-md-3"><div class="footer-left"><p class="mb-0"> © 2023 <a href="https://twitter.com/username">Clinical Team</a>. <span data-toggle="tooltip" data-placement="top" title="Except where otherwise noted, the blog posts on this site are licensed under the Creative Commons Attribution 4.0 International (CC BY 4.0) License by the author.">Some rights reserved.</span></p></div><div class="footer-right"><p class="mb-0">Using the <a href="https://jekyllrb.com" target="_blank" rel="noopener">Jekyll</a> theme <a href="https://github.com/cotes2020/jekyll-theme-chirpy" target="_blank" rel="noopener">Chirpy</a>.</p></div></div></div></footer><div id="mask"></div><a id="back-to-top" href="#" aria-label="back-to-top" class="btn btn-lg btn-box-shadow" role="button"> <i class="fas fa-angle-up"></i> </a><div id="notification" class="toast" role="alert" aria-live="assertive" aria-atomic="true" data-animation="true" data-autohide="false"><div class="toast-header"> <button type="button" class="ml-2 ml-auto close" data-dismiss="toast" aria-label="Close"> <span aria-hidden="true">&times;</span> </button></div><div class="toast-body text-center pt-0"><p class="pl-2 pr-2 mb-3">A new version of content is available.</p><button type="button" class="btn btn-primary" aria-label="Update"> Update </button></div></div><script src="https://cdn.jsdelivr.net/npm/simple-jekyll-search@1.10.0/dest/simple-jekyll-search.min.js"></script> <script> SimpleJekyllSearch({ searchInput: document.getElementById('search-input'), resultsContainer: document.getElementById('search-results'), json: '/assets/js/data/search.json', searchResultTemplate: '<div class="pl-1 pr-1 pl-sm-2 pr-sm-2 pl-lg-4 pr-lg-4 pl-xl-0 pr-xl-0"> <a href="{url}">{title}</a><div class="post-meta d-flex flex-column flex-sm-row text-muted mt-1 mb-1"> {categories} {tags}</div><p>{snippet}</p></div>', noResultsText: '<p class="mt-5">Oops! No results found.</p>', templateMiddleware: function(prop, value, template) { if (prop === 'categories') { if (value === '') { return `${value}`; } else { return `<div class="mr-sm-4"><i class="far fa-folder fa-fw"></i>${value}</div>`; } } if (prop === 'tags') { if (value === '') { return `${value}`; } else { return `<div><i class="fa fa-tag fa-fw"></i>${value}</div>`; } } } }); </script> <script src="https://cdn.jsdelivr.net/combine/npm/magnific-popup@1.1.0/dist/jquery.magnific-popup.min.js,npm/lazysizes@5.3.2/lazysizes.min.js,npm/clipboard@2.0.11/dist/clipboard.min.js"></script> <script src="https://cdn.jsdelivr.net/combine/npm/dayjs@1.11.6/dayjs.min.js,npm/dayjs@1.11.6/locale/en.min.js,npm/dayjs@1.11.6/plugin/relativeTime.min.js,npm/dayjs@1.11.6/plugin/localizedFormat.min.js"></script> <script defer src="/assets/js/dist/post.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/js/bootstrap.bundle.min.js"></script> <script defer src="/app.js"></script> <script defer src="https://www.googletagmanager.com/gtag/js?id=G-L66SLQK23K"></script> <script> document.addEventListener("DOMContentLoaded", function(event) { window.dataLayer = window.dataLayer || []; function gtag(){dataLayer.push(arguments);} gtag('js', new Date()); gtag('config', 'G-L66SLQK23K'); }); </script>
