<!DOCTYPE html><html lang="en" ><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="theme-color" media="(prefers-color-scheme: light)" content="#f7f7f7"><meta name="theme-color" media="(prefers-color-scheme: dark)" content="#1b1b1e"><meta name="apple-mobile-web-app-capable" content="yes"><meta name="apple-mobile-web-app-status-bar-style" content="black-translucent"><meta name="viewport" content="width=device-width, user-scalable=no initial-scale=1, shrink-to-fit=no, viewport-fit=cover" ><meta name="generator" content="Jekyll v4.3.2" /><meta property="og:title" content="Parenchymal Texture Analysis in Digital Breast Tomosynthesis for Breast Cancer Risk Estimation" /><meta property="og:locale" content="en" /><meta name="description" content="Rationale and Objectives" /><meta property="og:description" content="Rationale and Objectives" /><link rel="canonical" href="https://clinicaltree.github.io/posts/parenchymal-texture-analysis-in-digital-breast-tomosynthesis-for-breast-cancer-risk-estimation/" /><meta property="og:url" content="https://clinicaltree.github.io/posts/parenchymal-texture-analysis-in-digital-breast-tomosynthesis-for-breast-cancer-risk-estimation/" /><meta property="og:site_name" content="Radiology Tree" /><meta property="og:type" content="article" /><meta property="article:published_time" content="2009-02-28T17:00:00+00:00" /><meta name="twitter:card" content="summary" /><meta property="twitter:title" content="Parenchymal Texture Analysis in Digital Breast Tomosynthesis for Breast Cancer Risk Estimation" /><meta name="twitter:site" content="@twitter_username" /><meta name="google-site-verification" content="RFHVRgQqK0eGjftEMCTDhsDrR8cJ_ZYcfCX52gXW8KM" /> <script type="application/ld+json"> {"@context":"https://schema.org","@type":"BlogPosting","dateModified":"2023-04-07T07:15:32+00:00","datePublished":"2009-02-28T17:00:00+00:00","description":"Rationale and Objectives","headline":"Parenchymal Texture Analysis in Digital Breast Tomosynthesis for Breast Cancer Risk Estimation","mainEntityOfPage":{"@type":"WebPage","@id":"https://clinicaltree.github.io/posts/parenchymal-texture-analysis-in-digital-breast-tomosynthesis-for-breast-cancer-risk-estimation/"},"url":"https://clinicaltree.github.io/posts/parenchymal-texture-analysis-in-digital-breast-tomosynthesis-for-breast-cancer-risk-estimation/"}</script><title>Parenchymal Texture Analysis in Digital Breast Tomosynthesis for Breast Cancer Risk Estimation | Radiology Tree</title><link rel="apple-touch-icon" sizes="180x180" href="/assets/img/favicons/apple-touch-icon.png"><link rel="icon" type="image/png" sizes="32x32" href="/assets/img/favicons/favicon-32x32.png"><link rel="icon" type="image/png" sizes="16x16" href="/assets/img/favicons/favicon-16x16.png"><link rel="manifest" href="/assets/img/favicons/site.webmanifest"><link rel="shortcut icon" href="/assets/img/favicons/favicon.ico"><meta name="apple-mobile-web-app-title" content="Radiology Tree"><meta name="application-name" content="Radiology Tree"><meta name="msapplication-TileColor" content="#da532c"><meta name="msapplication-config" content="/assets/img/favicons/browserconfig.xml"><meta name="theme-color" content="#ffffff"><link rel="preconnect" href="https://fonts.googleapis.com" ><link rel="dns-prefetch" href="https://fonts.googleapis.com" ><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link rel="dns-prefetch" href="https://fonts.gstatic.com" crossorigin><link rel="preconnect" href="https://fonts.googleapis.com" ><link rel="dns-prefetch" href="https://fonts.googleapis.com" ><link rel="preconnect" href="https://cdn.jsdelivr.net" ><link rel="dns-prefetch" href="https://cdn.jsdelivr.net" ><link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Lato&family=Source+Sans+Pro:wght@400;600;700;900&display=swap"><link rel="preconnect" href="https://www.google-analytics.com" crossorigin="use-credentials"><link rel="dns-prefetch" href="https://www.google-analytics.com"><link rel="preconnect" href="https://www.googletagmanager.com" crossorigin="anonymous"><link rel="dns-prefetch" href="https://www.googletagmanager.com"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/css/bootstrap.min.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.2.1/css/all.min.css"><link rel="stylesheet" href="/assets/css/style.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/tocbot@4.20.1/dist/tocbot.min.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/magnific-popup@1.1.0/dist/magnific-popup.min.css"> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.1/dist/jquery.min.js"></script> <script type="text/javascript"> class ModeToggle { static get MODE_KEY() { return "mode"; } static get MODE_ATTR() { return "data-mode"; } static get DARK_MODE() { return "dark"; } static get LIGHT_MODE() { return "light"; } static get ID() { return "mode-toggle"; } constructor() { if (this.hasMode) { if (this.isDarkMode) { if (!this.isSysDarkPrefer) { this.setDark(); } } else { if (this.isSysDarkPrefer) { this.setLight(); } } } let self = this; /* always follow the system prefers */ this.sysDarkPrefers.addEventListener('change', () => { if (self.hasMode) { if (self.isDarkMode) { if (!self.isSysDarkPrefer) { self.setDark(); } } else { if (self.isSysDarkPrefer) { self.setLight(); } } self.clearMode(); } self.notify(); }); } /* constructor() */ get sysDarkPrefers() { return window.matchMedia("(prefers-color-scheme: dark)"); } get isSysDarkPrefer() { return this.sysDarkPrefers.matches; } get isDarkMode() { return this.mode === ModeToggle.DARK_MODE; } get isLightMode() { return this.mode === ModeToggle.LIGHT_MODE; } get hasMode() { return this.mode != null; } get mode() { return sessionStorage.getItem(ModeToggle.MODE_KEY); } /* get the current mode on screen */ get modeStatus() { if (this.isDarkMode || (!this.hasMode && this.isSysDarkPrefer)) { return ModeToggle.DARK_MODE; } else { return ModeToggle.LIGHT_MODE; } } setDark() { $('html').attr(ModeToggle.MODE_ATTR, ModeToggle.DARK_MODE); sessionStorage.setItem(ModeToggle.MODE_KEY, ModeToggle.DARK_MODE); } setLight() { $('html').attr(ModeToggle.MODE_ATTR, ModeToggle.LIGHT_MODE); sessionStorage.setItem(ModeToggle.MODE_KEY, ModeToggle.LIGHT_MODE); } clearMode() { $('html').removeAttr(ModeToggle.MODE_ATTR); sessionStorage.removeItem(ModeToggle.MODE_KEY); } /* Notify another plugins that the theme mode has changed */ notify() { window.postMessage({ direction: ModeToggle.ID, message: this.modeStatus }, "*"); } flipMode() { if (this.hasMode) { if (this.isSysDarkPrefer) { if (this.isLightMode) { this.clearMode(); } else { this.setLight(); } } else { if (this.isDarkMode) { this.clearMode(); } else { this.setDark(); } } } else { if (this.isSysDarkPrefer) { this.setLight(); } else { this.setDark(); } } this.notify(); } /* flipMode() */ } /* ModeToggle */ const modeToggle = new ModeToggle(); </script><body data-topbar-visible="true"><div id="sidebar" class="d-flex flex-column align-items-end"><div class="profile-wrapper text-center"><div id="avatar"> <a href="/" class="mx-auto"> <img src="https://storage.googleapis.com/clinicalpub.com/images/favicon.png" alt="avatar" onerror="this.style.display='none'"> </a></div><div class="site-title"> <a href="/">Radiology Tree</a></div><div class="site-subtitle font-italic">Update every day the best and the lastest articles, books, journals, clinical cases, videos, images... for radiologist</div></div><ul class="w-100"><li class="nav-item"> <a href="/" class="nav-link"> <i class="fa-fw fas fa-home ml-xl-3 mr-xl-3 unloaded"></i> <span>HOME</span> </a><li class="nav-item"> <a href="/categories/" class="nav-link"> <i class="fa-fw fas fa-stream ml-xl-3 mr-xl-3 unloaded"></i> <span>CATEGORIES</span> </a><li class="nav-item"> <a href="/tags/" class="nav-link"> <i class="fa-fw fas fa-tag ml-xl-3 mr-xl-3 unloaded"></i> <span>TAGS</span> </a><li class="nav-item"> <a href="/archives/" class="nav-link"> <i class="fa-fw fas fa-archive ml-xl-3 mr-xl-3 unloaded"></i> <span>ARCHIVES</span> </a><li class="nav-item"> <a href="/about/" class="nav-link"> <i class="fa-fw fas fa-info-circle ml-xl-3 mr-xl-3 unloaded"></i> <span>ABOUT</span> </a></ul><div class="sidebar-bottom mt-auto d-flex flex-wrap justify-content-center align-items-center"> <button class="mode-toggle btn" aria-label="Switch Mode"> <i class="fas fa-adjust"></i> </button> <span class="icon-border"></span> <a href="https://github.com/clinicaltree" aria-label="github" target="_blank" rel="noopener noreferrer"> <i class="fab fa-github"></i> </a> <a href="https://twitter.com/twitter_username" aria-label="twitter" target="_blank" rel="noopener noreferrer"> <i class="fab fa-twitter"></i> </a> <a href="javascript:location.href = 'mailto:' + ['clinicalpub.team','gmail.com'].join('@')" aria-label="email" > <i class="fas fa-envelope"></i> </a> <a href="/feed.xml" aria-label="rss" > <i class="fas fa-rss"></i> </a></div></div><div id="topbar-wrapper"><div id="topbar" class="container d-flex align-items-center justify-content-between h-100 pl-3 pr-3 pl-md-4 pr-md-4"> <span id="breadcrumb"> <span> <a href="/"> Home </a> </span> <span>Parenchymal Texture Analysis in Digital Breast Tomosynthesis for Breast Cancer Risk Estimation</span> </span> <i id="sidebar-trigger" class="fas fa-bars fa-fw"></i><div id="topbar-title"> Post</div><i id="search-trigger" class="fas fa-search fa-fw"></i> <span id="search-wrapper" class="align-items-center"> <i class="fas fa-search fa-fw"></i> <input class="form-control" id="search-input" type="search" aria-label="search" autocomplete="off" placeholder="Search..."> </span> <span id="search-cancel" >Cancel</span></div></div><div id="main-wrapper" class="d-flex justify-content-center"><div id="main" class="container pl-xl-4 pr-xl-4"><div class="row"><div id="core-wrapper" class="col-12 col-lg-11 col-xl-9 pr-xl-4"><div class="post pl-1 pr-1 pl-md-2 pr-md-2"><h1 data-toc-skip>Parenchymal Texture Analysis in Digital Breast Tomosynthesis for Breast Cancer Risk Estimation</h1><div class="post-meta text-muted"> <span> Posted <em class="" data-ts="1235840400" data-df="ll" data-toggle="tooltip" data-placement="bottom"> Feb 28, 2009 </em> </span> <span> Updated <em class="" data-ts="1680851732" data-df="ll" data-toggle="tooltip" data-placement="bottom"> Apr 7, 2023 </em> </span><div class="d-flex justify-content-between"> <span> By <em> <a href="">Despina Kontos PhD</a> </em>, <em> <a href="">Predrag R. Bakic PhD</a> </em>, <em> <a href="">Ann-Katherine Carton PhD</a> </em>, <em> <a href="">Andrea B. Troxel ScD</a> </em>, <em> <a href="">Emily F. Conant MD</a> </em>, <em> <a href="">Andrew D.A. Maidment PhD</a> </em> </span><div> <span class="readtime" data-toggle="tooltip" data-placement="bottom" title="4666 words"> <em>25 min</em> read</span></div></div></div><div class="post-content"><h2 id="rationale-and-objectives"><span class="mr-2">Rationale and Objectives</span><a href="#rationale-and-objectives" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p>Studies have demonstrated a relationship between mammographic parenchymal texture and breast cancer risk. Although promising, texture analysis in mammograms is limited by tissue superposition. Digital breast tomosynthesis (DBT) is a novel tomographic x-ray breast imaging modality that alleviates the effect of tissue superposition, offering superior parenchymal texture visualization compared to mammography. The aim of this study was to investigate the potential advantages of DBT parenchymal texture analysis for breast cancer risk estimation.</p><h2 id="materials-and-methods"><span class="mr-2">Materials and Methods</span><a href="#materials-and-methods" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p>DBT and digital mammographic (DM) images of 39 women were analyzed. Texture features, shown in previous studies with mammograms to correlate with cancer risk, were computed from the retroareolar breast region. The relative performances of the DBT and DM texture features were compared in correlating with two measures of breast cancer risk: (1) the Gail and Claus risk estimates and (2) mammographic breast density. Linear regression was performed to model the association between texture features and increasing levels of risk.</p><h2 id="results"><span class="mr-2">Results</span><a href="#results" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p>No significant correlation was detected between parenchymal texture and the Gail and Claus risk estimates. Significant correlations were observed between texture features and breast density. Overall, the DBT texture features demonstrated stronger correlations with breast percent density than DM features ( <em>P</em> ≤ .05). When dividing the study population into groups of increasing breast percent density, the DBT texture features appeared to be more discriminative, having regression lines with overall lower <em>P</em> values, steeper slopes, and higher <em>R</em> 2 estimates.</p><h2 id="conclusion"><span class="mr-2">Conclusion</span><a href="#conclusion" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p>Although preliminary, the results of this study suggest that DBT parenchymal texture analysis could provide more accurate characterization of breast density patterns, which could ultimately improve breast cancer risk estimation.</p><p>The ability to estimate a woman’s risk of developing breast cancer risk is becoming increasingly important in clinical practice. Breast cancer risk assessment is used as a criterion to form guidelines for offering customized screening recommendations , to tailor individual breast cancer treatments , and to form preventive strategies , especially for women associated with higher risk. Currently, breast cancer risk assessment is limited both by the existing epidemiologic risk estimation models and by the breast imaging methods that have been considered to date.</p><p>The current gold standards for breast cancer risk estimation, the Gail and Claus models , are multivariate statistical models based primarily on nonmodifiable demographic, clinical, and hereditary risk factors. With the exception of childbirth as a modifiable risk factor, the Gail model estimates the risk for breast cancer on the basis of factors such as age at menarche, first-degree relatives with breast cancer, and number of prior biopsies . The Claus model relies on the assumption that susceptibility to breast cancer is regulated primarily by a rare autosomal dominant allele and therefore estimates the risk for breast cancer only on the basis of familial history of breast cancer, including ages at onset of relatives affected by breast cancer . Evaluation of the Gail model has shown that despite its good calibration for population-based risk assessment, it has modest discriminatory accuracy at the individual level . Studies evaluating the accuracy of models that predict genetic susceptibility to breast cancer, including the Claus model, have shown that there is a potential to overestimate the expected number of women at high risk for genetic mutations . Considering also that individual risk can be reduced by interventions such as chemoprevention , the Gail and Claus models lack the desired flexibility to estimate adjustments to risk levels.</p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="APKAICLNFGBCWWYGVIZQ" class="popup img-link "><img data-src="https://d1niluoi1dd30v.cloudfront.net/10766332/S1076633209X00024/S107663320800531X/gr1.jpg?Signature=ch2kA73occOGcbewTKAkq5JJWUOJieSi7mqdjqBXosVZFRSNdQuH8mjlngzbm7bcX-Q0Bt2DwDd%7E3jl9iptaAsVwh4M39BKHnj7ffBi3L7YMptuOuQbwzTijt-WOec3%7EBGGEXY4LLdoWszom%7EyvCcAQ5VR%7EGVb5NcpRAGlYrap4_&amp;Expires=1669550162&amp;Key-Pair-Id=APKAICLNFGBCWWYGVIZQ" alt="" class="lazyload" data-proofer-ignore></a>Open full size image</p><p>Figure 1</p><p>Illustrative example of <strong>(a)</strong> digital breast tomosynthesis acquisition geometry with <strong>(b)</strong> the reconstructed tomographic breast image. <em>3D</em> , three-dimensional.</p><p><a href="APKAICLNFGBCWWYGVIZQ" class="popup img-link "><img data-src="https://d1niluoi1dd30v.cloudfront.net/10766332/S1076633209X00024/S107663320800531X/gr2.jpg?Signature=N66CoNCRT7KHcKrw5NlBbyTzvDdKi4wDwaoHE9fTE6P4Nyf-qtROZmfE4yBu7d82pvlkFfAwMfzNMez2ikqEwh9cH8wk-mStExonx9oVOT%7Et48A84spckDHFnBu-z4bClbGOGtvY-edLJ2%7EulDhkWMFIbGiFd%7EA9KKZKhWHwypA_&amp;Expires=1669550162&amp;Key-Pair-Id=APKAICLNFGBCWWYGVIZQ" alt="" class="lazyload" data-proofer-ignore></a>Open full size image</p><p>Figure 2</p><p>Differences in parenchymal texture in <strong>(a)</strong> a digital mammogram (DM) and <strong>(b,c)</strong> the digital breast tomosynthesis (DBT) tomographic slices for the same breast, where <strong>(b)</strong> the superficial skin layer is separated from <strong>(c)</strong> the deeper fibroglandular tissue layers.</p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="materials-and-methods-1"><span class="mr-2">Materials and methods</span><a href="#materials-and-methods-1" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><h2 id="patient-recruitment"><span class="mr-2">Patient Recruitment</span><a href="#patient-recruitment" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p>∗ The goal of this clinical trial was to develop an understanding of the relative performance of new-generation breast imaging modalities. Eligible participants included women at high risk (&gt;25% Gail and Claus lifetime risk), women with recently detected abnormalities, and previous patients with breast cancer undergoing follow-up. All women were volunteers who provided written informed consent. From March 2002 to August 2007, a total of 886 women enrolled in the trial. Within the same day, the women were imaged with digital mammography, whole-breast ultrasound, magnetic resonance imaging, positron emission tomography, and optical imaging. The individual imaging results were reviewed in a consensus meeting of expert radiologists to determine the relative performance of the breast imaging modalities; the associated clinical information for each woman, such as pathology, likelihood of malignancy, and Breast Imaging Reporting and Data System lesion characterization, was also recorded as part of the study. From August 2004 to August 2005, a prototype DBT system was operating under research investigation, and DBT was offered as an option to the women participating in the clinical trial. During this period, a total of 52 women agreed to also undergo DBT imaging.</p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="breast-imaging"><span class="mr-2">Breast Imaging</span><a href="#breast-imaging" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="study-population-and-risk-evaluation"><span class="mr-2">Study Population and Risk Evaluation</span><a href="#study-population-and-risk-evaluation" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://storage.googleapis.com/dl.dentistrykey.com/clinical/ParenchymalTextureAnalysisinDigitalBreastTomosynthesisforBreastCancerRiskEstimation/0_1s20S107663320800531X.jpg" class="popup img-link "><img data-src="https://storage.googleapis.com/dl.dentistrykey.com/clinical/ParenchymalTextureAnalysisinDigitalBreastTomosynthesisforBreastCancerRiskEstimation/0_1s20S107663320800531X.jpg" alt="Figure 3, Illustration of the Cumulus (version 4.0) software thresholding technique used for mammographic breast percent density (PD) estimation: the image background and the pectoral muscle are excluded (red) , and the dense tissue is segmented by gray-level thresholding (green) . PD is then estimated as the percentage of dense tissue within the delineated breast region." class="lazyload" data-proofer-ignore></a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="image-analysis"><span class="mr-2">Image Analysis</span><a href="#image-analysis" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://storage.googleapis.com/dl.dentistrykey.com/clinical/ParenchymalTextureAnalysisinDigitalBreastTomosynthesisforBreastCancerRiskEstimation/1_1s20S107663320800531X.jpg" class="popup img-link "><img data-src="https://storage.googleapis.com/dl.dentistrykey.com/clinical/ParenchymalTextureAnalysisinDigitalBreastTomosynthesisforBreastCancerRiskEstimation/1_1s20S107663320800531X.jpg" alt="Figure 4, An illustrative example of (a) a three-dimensional region of interest segmented from a reconstructed digital breast tomosynthesis (DBT) image and (b) the corresponding two-dimensional region of interest from the digital mammogram (DM) of the same breast." class="lazyload" data-proofer-ignore></a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://storage.googleapis.com/dl.dentistrykey.com/clinical/ParenchymalTextureAnalysisinDigitalBreastTomosynthesisforBreastCancerRiskEstimation/2_1s20S107663320800531X.jpg" class="popup img-link "><img data-src="https://storage.googleapis.com/dl.dentistrykey.com/clinical/ParenchymalTextureAnalysisinDigitalBreastTomosynthesisforBreastCancerRiskEstimation/2_1s20S107663320800531X.jpg" alt="Figure 5, Examples of various mammographic texture patterns: (a) skewness, (b) coarseness, (c) fractal dimension, and (d) contrast." class="lazyload" data-proofer-ignore></a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="tomographic-2d-texture-analysis"><span class="mr-2">Tomographic (2D) Texture Analysis</span><a href="#tomographic-2d-texture-analysis" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p>skewness=w3w3/22,wk=∑gmaxi=0ni(i−i¯)k/N,N=∑gmaxi=0ni,i¯=∑gmaxi=0(ini/N), skewness</p><p>=</p><p>w</p><p>3</p><p>w</p><p>2</p><p>3</p><p>/</p><p>2</p><p>,</p><p>w</p><p>k</p><p>=</p><p>∑</p><p>i</p><p>=</p><p>0</p><p>g</p><p>max</p><p>n</p><p>i</p><p>(</p><p>i</p><p>−</p><p>i</p><p>¯</p><p>)</p><p>k</p><p>/</p><p>N</p><p>,</p><p>N</p><p>=</p><p>∑</p><p>i</p><p>=</p><p>0</p><p>g</p><p>max</p><p>n</p><p>i</p><p>,</p><p>i</p><p>¯</p><p>=</p><p>∑</p><p>i</p><p>=</p><p>0</p><p>g</p><p>max</p><p>(</p><p>i</p><p>n</p><p>i</p><p>/</p><p>N</p><p>)</p><p>,</p><p>where <em>n__i</em> represents the number of times that gray-level value <em>i</em> occurs in the image region, <em>g__max</em> is the maximum gray-level value, and <em>N</em> is the total number of image pixels.</p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p>coarseness=(∑gmaxi=0piv(i))−1andv(i)={∑∣∣i−L¯¯¯i∣∣fori∈{ni}ifni≠00otherwise}, coarseness</p><p>=</p><p>(</p><p>∑</p><p>i</p><p>=</p><p>0</p><p>g</p><p>max</p><p>p</p><p>i</p><p>v</p><p>(</p><p>i</p><p>)</p><p>)</p><p>−</p><p>1</p><p>and</p><p>v</p><p>(</p><p>i</p><p>)</p><p>=</p><p>{</p><p>∑</p><p>|</p><p>i</p><p>−</p><p>L</p><p>¯</p><p>i</p><p>|</p><p>for</p><p>i</p><p>∈</p><p>{</p><p>n</p><p>i</p><p>}</p><p>if</p><p>n</p><p>i</p><p>≠</p><p>0</p><p>0</p><p>otherwise</p><p>}</p><p>,</p><p>where <em>v</em> ( <em>i</em> ) is the NGTDM. In the above formulas, <em>g__max</em> is the maximum gray-level value, <em>p__i</em> is the probability that gray level <em>i</em> occurs, and { <em>n__i</em> } is the set of pixels having gray-level values equal to <em>i</em> , and L¯¯¯i L</p><p>¯</p><p>i is given by</p><p>L¯¯¯i=1S−1∑tk=−t∑tl=−tj(x+k,y+l), L</p><p>¯</p><p>i</p><p>=</p><p>1</p><p>S</p><p>−</p><p>1</p><p>∑</p><p>k</p><p>=</p><p>−</p><p>t</p><p>t</p><p>∑</p><p>l</p><p>=</p><p>−</p><p>t</p><p>t</p><p>j</p><p>(</p><p>x</p><p>+</p><p>k</p><p>,</p><p>y</p><p>+</p><p>l</p><p>)</p><p>,</p><p>where <em>j</em> ( <em>x</em> , <em>y</em> ) is the pixel located at ( <em>x</em> , <em>y</em> ) with gray-level value <em>i</em> , ( <em>k</em> , <em>l</em> ) ≠ (0,0), and <em>S</em> = (2 <em>t</em> + 1) 2 , with <em>t</em> = 1 specifying the neighborhood size around the pixel located at ( <em>x</em> , <em>y</em> ).</p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p>contrast=∑gmaxi=0∑gmaxj=0|i−j|2C(i,j), contrast</p><p>=</p><p>∑</p><p>i</p><p>=</p><p>0</p><p>g</p><p>max</p><p>∑</p><p>j</p><p>=</p><p>0</p><p>g</p><p>max</p><p>|</p><p>i</p><p>−</p><p>j</p><p>|</p><p>2</p><p>C</p><p>(</p><p>i</p><p>,</p><p>j</p><p>)</p><p>,</p><p>energy=∑gmaxi=0∑gmaxj=0C(i,j)2, energy</p><p>=</p><p>∑</p><p>i</p><p>=</p><p>0</p><p>g</p><p>max</p><p>∑</p><p>j</p><p>=</p><p>0</p><p>g</p><p>max</p><p>C</p><p>(</p><p>i</p><p>,</p><p>j</p><p>)</p><p>2</p><p>,</p><p>and</p><p>homogeneity=∑gmaxi=0∑gmaxj=0C(i,j)1+|i−j|, homogeneity</p><p>=</p><p>∑</p><p>i</p><p>=</p><p>0</p><p>g</p><p>max</p><p>∑</p><p>j</p><p>=</p><p>0</p><p>g</p><p>max</p><p>C</p><p>(</p><p>i</p><p>,</p><p>j</p><p>)</p><p>1</p><p>+</p><p>|</p><p>i</p><p>−</p><p>j</p><p>|</p><p>,</p><p>where <em>g__max</em> is the maximum gray-level value, and <em>C</em> is the normalized co-occurrence matrix . To optimize the computation of the gray-level co-occurrence statistics, gray-level quantization was implemented . The co-occurrence frequencies were calculated symmetrically in the four directions around each pixel using a displacement vector, <em>d</em> = ( <em>dx</em> , <em>dy</em> ), along <em>x</em> and <em>y</em> dimensions, where <em>dx</em> = <em>dy</em> = 1 pixel offset. The texture features calculated in each of these four directions were averaged to create a single measure that was used in our experiments.</p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p>F(u,v)=∑M−1m=0∑N−1n=0I(m,n)e−j(2π/M)ume−j(2π/N)vn,u=0,1,…M−1,v=0,1,…N−1, F</p><p>(</p><p>u</p><p>,</p><p>v</p><p>)</p><p>=</p><p>∑</p><p>m</p><p>=</p><p>0</p><p>M</p><p>−</p><p>1</p><p>∑</p><p>n</p><p>=</p><p>0</p><p>N</p><p>−</p><p>1</p><p>I</p><p>(</p><p>m</p><p>,</p><p>n</p><p>)</p><p>e</p><p>−</p><p>j</p><p>(</p><p>2</p><p>π</p><p>/</p><p>M</p><p>)</p><p>u</p><p>m</p><p>e</p><p>−</p><p>j</p><p>(</p><p>2</p><p>π</p><p>/</p><p>N</p><p>)</p><p>v</p><p>n</p><p>,</p><p>u</p><p>=</p><p>0</p><p>,</p><p>1</p><p>,</p><p>…</p><p>M</p><p>−</p><p>1</p><p>,</p><p>v</p><p>=</p><p>0</p><p>,</p><p>1</p><p>,</p><p>…</p><p>N</p><p>−</p><p>1</p><p>,</p><p>where <em>I</em> is the 2D image region of size ( <em>M</em> , <em>N</em> ), and <em>u</em> and <em>v</em> are the spatial frequencies in the <em>x</em> and <em>y</em> directions. The power spectral density, <em>P</em> , was estimated from <em>F</em> ( <em>u</em> , <em>v</em> ) as</p><p>P(u,v)=|F(u,v)|2. P</p><p>(</p><p>u</p><p>,</p><p>v</p><p>)</p><p>=</p><p>|</p><p>F</p><p>(</p><p>u</p><p>,</p><p>v</p><p>)</p><p>|</p><p>2</p><p>.</p><p>To compute the FD, <em>P</em> was averaged over radial slices spanning the FFT frequency domain. The frequency space was uniformly divided in 24 directions, with each direction uniformly sampled at 30 points along the radial component. To calculate the FD, the least-squares fit of log( <em>P__f</em> ) versus log( <em>f</em> ) was estimated, where f=u2+v2−−−−−−√ f</p><p>=</p><p>u</p><p>2</p><p>+</p><p>v</p><p>2 denotes the radial frequency . The FD is related to the slope, β, of this log-log plot by</p><p>FD=3DT+2−β2=8−β2, FD</p><p>=</p><p>3</p><p>D</p><p>T</p><p>+</p><p>2</p><p>−</p><p>β</p><p>2</p><p>=</p><p>8</p><p>−</p><p>β</p><p>2</p><p>,</p><p>where <em>D__T</em> is the topologic dimension, equal to 2 for a 2D image.</p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="volumetric-3d-texture-analysis"><span class="mr-2">Volumetric (3D) Texture Analysis</span><a href="#volumetric-3d-texture-analysis" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p>L¯¯¯i=1S−1∑tk=−t∑tl=−t∑tq=−tj(x+k,y+l,z+q), L</p><p>¯</p><p>i</p><p>=</p><p>1</p><p>S</p><p>−</p><p>1</p><p>∑</p><p>k</p><p>=</p><p>−</p><p>t</p><p>t</p><p>∑</p><p>l</p><p>=</p><p>−</p><p>t</p><p>t</p><p>∑</p><p>q</p><p>=</p><p>−</p><p>t</p><p>t</p><p>j</p><p>(</p><p>x</p><p>+</p><p>k</p><p>,</p><p>y</p><p>+</p><p>l</p><p>,</p><p>z</p><p>+</p><p>q</p><p>)</p><p>,</p><p>where <em>j</em> ( <em>x</em> , <em>y</em> , <em>z</em> ) is the voxel located at ( <em>x</em> , <em>y</em> , <em>z</em> ) with gray-level value <em>i</em> , ( <em>k</em> , <em>l</em> , <em>z</em> ) ≠ (0,0,0), and <em>S</em> = (2 <em>t</em> + 1) 3 , with <em>t</em> = 1 specifying the 3D voxel window around ( <em>x</em> , <em>y</em> , <em>z</em> ).</p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p>F(u,v,w)=∑M−1m=0∑N−1n=0∑K−1k=0I(m,n,k)e−j(2π/M)ume−j(2π/N)vne−j(2π/K)wk, F</p><p>(</p><p>u</p><p>,</p><p>v</p><p>,</p><p>w</p><p>)</p><p>=</p><p>∑</p><p>m</p><p>=</p><p>0</p><p>M</p><p>−</p><p>1</p><p>∑</p><p>n</p><p>=</p><p>0</p><p>N</p><p>−</p><p>1</p><p>∑</p><p>k</p><p>=</p><p>0</p><p>K</p><p>−</p><p>1</p><p>I</p><p>(</p><p>m</p><p>,</p><p>n</p><p>,</p><p>k</p><p>)</p><p>e</p><p>−</p><p>j</p><p>(</p><p>2</p><p>π</p><p>/</p><p>M</p><p>)</p><p>u</p><p>m</p><p>e</p><p>−</p><p>j</p><p>(</p><p>2</p><p>π</p><p>/</p><p>N</p><p>)</p><p>v</p><p>n</p><p>e</p><p>−</p><p>j</p><p>(</p><p>2</p><p>π</p><p>/</p><p>K</p><p>)</p><p>w</p><p>k</p><p>,</p><p>where <em>I</em> is the 3D image region of size ( <em>M</em> , <em>N</em> , <em>K</em> ), and <em>u</em> , <em>v</em> , and <em>w</em> are the spatial frequencies in the <em>x</em> , <em>y</em> , and <em>z</em> directions, respectively. The power spectral density was estimated as</p><p>P(u,v,w)=|F(u,v,w)|2. P</p><p>(</p><p>u</p><p>,</p><p>v</p><p>,</p><p>w</p><p>)</p><p>=</p><p>|</p><p>F</p><p>(</p><p>u</p><p>,</p><p>v</p><p>,</p><p>w</p><p>)</p><p>|</p><p>2</p><p>.</p><p>To compute the FD, <em>P</em> was averaged over radial sectors spanning the 3D FFT frequency domain. The frequency space was evenly divided in 24 azimuth and 12 zenith directions, and each direction was uniformly sampled at 30 points along the radial component. To calculate the FD, the least-squares fit of log( <em>P__f</em> ) versus log( <em>f</em> ) was estimated, where f=u2+v2+w2−−−−−−−−−−√ f</p><p>=</p><p>u</p><p>2</p><p>+</p><p>v</p><p>2</p><p>+</p><p>w</p><p>2 denotes the radial frequency in spherical coordinates . The FD is related to the slope, β, of this log-log plot as defined in Equation 7 , and for <em>D__T</em> = 3 in three dimensions:</p><p>FD=11−β2. FD</p><p>=</p><p>11</p><p>−</p><p>β</p><p>2</p><p>.</p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="texture-association-with-gail-and-claus-risk"><span class="mr-2">Texture Association with Gail and Claus Risk</span><a href="#texture-association-with-gail-and-claus-risk" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="texture-association-with-breast-density"><span class="mr-2">Texture Association with Breast Density</span><a href="#texture-association-with-breast-density" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="results-1"><span class="mr-2">Results</span><a href="#results-1" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><h2 id="descriptive-statistics"><span class="mr-2">Descriptive Statistics</span><a href="#descriptive-statistics" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="correlation-between-dbt-and-dm-texture-features"><span class="mr-2">Correlation Between DBT and DM Texture Features</span><a href="#correlation-between-dbt-and-dm-texture-features" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p>Table 1</p><p>Pearson’s Correlation Coefficients ( <em>P</em> Values) Between DBT and DM Parenchymal Texture Features</p><p>Variable 2D DBT vs DM 3D DBT vs DM 2D DBT vs 3D DBT Skewness −0.07 (.65) −0.08 (.59) 0.99 (&lt;.001) Coarseness 0.37 (.02) 0.25 (.12) 0.97 (&lt;.001) Contrast 0.46 (.003) 0.41 (.008) 0.98 (&lt;.001) Energy 0.16 (.31) 0.37 (.02) 0.79 (&lt;.001) Homogeneity 0.50 (&lt;.001) 0.51 (&lt;.001) 0.98 (&lt;.001) Fractal dimension 0.73 (&lt;.001) 0.27 (.09) 0.01 (.91)</p><p>DBT, digital breast tomosynthesis; DM, digital mammographic; 3D, three-dimensional; 2D, two-dimensional.</p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="texture-correlation-with-gail-and-claus-risk-estimates"><span class="mr-2">Texture Correlation with Gail and Claus Risk Estimates</span><a href="#texture-correlation-with-gail-and-claus-risk-estimates" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p>Table 2</p><p>Pearson’s Correlation Coefficients ( <em>P</em> Values) Between DM and DBT Parenchymal Texture Features and the Gail and Claus Lifetime Breast Cancer Risk Estimates</p><p>Gail Risk Claus Risk Variable DM 2D DBT 3D DBT DM 2D DBT 3D DBT Skewness 0.01 (.97) 0.04 (.81) 0.05 (.74) −0.07 (.69) 0.32 (.05) ∗ 0.33 (.04) ∗ Coarseness 0.06 (.72) 0.03 (.83) 0.03 (.86) −0.08 (.61) 0.14 (.39) 0.15 (.35) Contrast 0.00 (.99) −0.03 (.86) −0.04 (.80) −0.11 (.50) −0.09 (.57) −0.10 (.55) Energy −0.24 (.14) −0.03 (.85) −0.11 (.50) −0.24 (.13) −0.20 (.13) −0.16 (.34) Homogeneity 0.01 (.95) 0.02 (.92) 0.02 (.88) 0.20 (.22) 0.09 (.60) 0.10 (.53) Fractal dimension 0.02 (.92) 0.19 (.24) −0.01 (.95) −0.02 (.88) −0.01 (.94) 0.03 (.85)</p><p>DBT, digital breast tomosynthesis; DM, digital mammography; 3D, three-dimensional; 2D, two-dimensional.</p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="texture-correlation-with-breast-density"><span class="mr-2">Texture Correlation with Breast Density</span><a href="#texture-correlation-with-breast-density" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p>Table 3</p><p>Pearson’s Correlation Coefficients ( <em>P</em> Values) Between DM and DBT Parenchymal Texture Features and Breast PD</p><p>Breast PD Variable DM 2D DBT 3D DBT Skewness −0.18 (.26) 0.18 (.26) 0.18 (.26) Coarseness −0.15 (.34) 0.40 (.01) 0.46 (.003) Contrast −0.25 (.13) −0.23 (.15) −0.31 (.05) Energy −0.29 (.07) −0.20 (.21) −0.36 (.03) Homogeneity 0.39 (.01) 0.16 (.32) 0.26 (.11) Fractal dimension 0.50 (.001) 0.23 (.16) 0.45 (.004)</p><p>DBT, digital breast tomosynthesis; DM, digital mammographic; PD, percent density; 3D, three-dimensional; 2D, two-dimensional.</p><p><a href="https://storage.googleapis.com/dl.dentistrykey.com/clinical/ParenchymalTextureAnalysisinDigitalBreastTomosynthesisforBreastCancerRiskEstimation/3_1s20S107663320800531X.jpg" class="popup img-link "><img data-src="https://storage.googleapis.com/dl.dentistrykey.com/clinical/ParenchymalTextureAnalysisinDigitalBreastTomosynthesisforBreastCancerRiskEstimation/3_1s20S107663320800531X.jpg" alt="Figure 6, Scatterplots of the texture features versus breast percent density (PD) (left) and the Gail lifetime risk estimates (right) for digital mammography (DM) and digital breast tomosynthesis (DBT)." class="lazyload" data-proofer-ignore></a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://storage.googleapis.com/dl.dentistrykey.com/clinical/ParenchymalTextureAnalysisinDigitalBreastTomosynthesisforBreastCancerRiskEstimation/4_1s20S107663320800531X.jpg" class="popup img-link "><img data-src="https://storage.googleapis.com/dl.dentistrykey.com/clinical/ParenchymalTextureAnalysisinDigitalBreastTomosynthesisforBreastCancerRiskEstimation/4_1s20S107663320800531X.jpg" alt="Figure 7, Box plots with fitted regression lines and associated P values for digital mammographic (DM) and digital breast tomosynthesis (DBT) coarseness, contrast, and fractal dimension texture features versus the five groups of increasing breast percent density (PD): PD &lt; 10%, 10% ≤ PD &lt; 25%, 25% ≤ PD &lt; 50%, 50% ≤ PD &lt; 75%, and 75% ≤ PD &lt; 100%." class="lazyload" data-proofer-ignore></a></p><p><a href="https://storage.googleapis.com/dl.dentistrykey.com/clinical/ParenchymalTextureAnalysisinDigitalBreastTomosynthesisforBreastCancerRiskEstimation/5_1s20S107663320800531X.jpg" class="popup img-link "><img data-src="https://storage.googleapis.com/dl.dentistrykey.com/clinical/ParenchymalTextureAnalysisinDigitalBreastTomosynthesisforBreastCancerRiskEstimation/5_1s20S107663320800531X.jpg" alt="Figure 8, Box plots with fitted regression lines and associated P values for digital mammographic (DM) and digital breast tomosynthesis (DBT) principal-component analysis (PCA) features versus the five groups of increasing breast percent density (PD): PD &lt; 10%, 10% ≤ PD &lt; 25%, 25% ≤ PD &lt; 50%, 50% ≤ PD &lt; 75%, and 75% ≤ PD &lt; 100%." class="lazyload" data-proofer-ignore></a></p><p>Table 4</p><p>Beta <em>b</em> Coefficients, <em>R</em> 2 Values, and <em>P</em> Values for the Fitted Regression Models of Each DM and DBT Texture Descriptor</p><p>DM 3D DBT Variable_b__R_ 2 <em>P__b__R</em> 2 <em>P</em> Skewness −0.08 0.01 .50 0.06 0.03 .33 Coarseness −0.2 × 10 −4 0.01 .55 0.7×10 −5 0.17 .008 Contrast −0.91 0.05 .15 −588 0.10 .05 Energy −0.006 0.06 .14 −0.005 0.07 .09 Homogeneity 0.005 0.10 .04 0.009 0.08 .09 Fractal dimension 0.04 0.18 .006 0.04 0.16 .01 PCA 0.19 0.01 .46 0.84 0.21 .003</p><p>DBT, digital breast tomosynthesis; DM, digital mammography; PCA, principal-component analysis; 3D, three-dimensional.</p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="discussion"><span class="mr-2">Discussion</span><a href="#discussion" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="acknowledgments"><span class="mr-2">Acknowledgments</span><a href="#acknowledgments" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><p><a href="https://clinicalpub.com/app">Get Radiology Tree app to read full this article&lt;</a></p><h2 id="references"><span class="mr-2">References</span><a href="#references" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2><ul><li><p>1. Lehman C.D., Blume J.D., Weatherall P., et. al.: Screening women at high risk for breast cancer with mammography and magnetic resonance imaging. Cancer 2005; 103: pp. 1898-1905.</p><li><p>2. Lehman C.D., Isaacs C., Schnall M.D., et. al.: Cancer yield of mammography, MR, and US in high-risk women: prospective multi-institution breast cancer screening study. Radiology 2007; 224: pp. 381-388.</p><li><p>3. Smith K.L., Isaacs C.: Management of women at increased risk for hereditary breast cancer. Breast Dis 2006–2007; 27: pp. 51-67.</p><li><p>4. Gail M.H., Brinton L.A., Byar D.P., et. al.: Projecting individualized probabilities of developing breast cancer for white females who are being examined annually. J Natl Cancer Inst 1989; 81: pp. 1879-1886.</p><li><p>5. Claus E.B., Risch N., Thompson W.D., Carter D.: Relationship between breast histopathology and family history of breast cancer. Cancer 1993; 71: pp. 147-153.</p><li><p>6. Rockhill B., Spiegelman D., Byrne C., Hunter D.J., Colditz G.A.: Validation of the Gail et al. model of breast cancer risk prediction and implications for chemoprevention. J Natl Cancer Inst 2001; 93: pp. 358-366.</p><li><p>7. Jones J.L., Hughes K.S., Kopans D.B., et. al.: Evaluation of hereditary risk in a mammography population. Clin Breast Cancer 2005; 6: pp. 38-44.</p><li><p>8. Fisher B., Costantino J.P., Wickerham D.L., et. al.: Tamoxifen for the prevention of breast cancer: current status of the National Surgical Adjuvant Breast and Bowel Project P-1 study. J Natl Cancer Inst 2005; 97: pp. 1652-1662.</p><li><p>9. Chlebowski R.T., McTiernan A.: Biological significance of interventions that change breast density. J Natl Cancer Inst 2003; 95: pp. 4-5.</p><li><p>10. Greendale G.A., Reboussin B.A., Slone S., Wasilauskas C., Pike M.C., Ursin G.: Postmenopausal hormone therapy and change in mammographic density. J Natl Cancer Inst 2003; 95: pp. 30-37.</p><li><p>11. Byrne C., Schairer C., Wolfe J., et. al.: Mammographic features and breast cancer risk: effects with time, age, and menopause status. J Natl Cancer Inst 1995; 87: pp. 1622-1629.</p><li><p>12. Fabian C.J., Kimler B.F.: Mammographic density: use in risk assessment and as a biomarker in prevention trials. J Nutr 2006; 136: pp. 2705S-2708S.</p><li><p>13. Wolfe J.N.: Risk for breast cancer development determined by mammographic parenchymal pattern. Cancer 1976; 37: pp. 2486-2492.</p><li><p>14. McCormack V.A., dos Santos Silva I.: Breast density and parenchymal patterns as markers of breast cancer risk: a meta-analysis. Cancer Epidemiol Biomarker Prev 2006; 15: pp. 1159-1169.</p><li><p>15. Kaufhold J., Thomas J.A., Eberhard J.W., Galbo C.E., Trotter D.E.: A calibration approach to glandular tissue composition estimation in digital mammography. Med Phys 2002; 28: pp. 1867-1880.</p><li><p>16. Heine J.J., Malhotra P.: Mammographic tissue, breast cancer risk, serial image analysis, and digital mammography. Part 1. Tissue and related risk factors. Acad Radiol 2002; 9: pp. 298-316.</p><li><p>17. Heine J.J., Malhotra P.: Mammographic tissue, breast cancer risk, serial image analysis, and digital mammography. Part 2. Serial breast tissue change and related temporal influences. Acad Radiol 2002; 9: pp. 317-335.</p><li><p>18. Martin L.J., Boyd N.F.: Mammographic density. Potential mechanisms of breast cancer risk associated with mammographic density: hypotheses based on epidemiological evidence. Breast Cancer Res 2008; 10: pp. 201.</p><li><p>19. Boyd N.F., Guo H., Martin L.J., et. al.: Mammographic density and the risk and detection of breast cancer. N Engl J Med 2007; 356: pp. 227-236.</p><li><p>20. Vachon C.M., van Gils C.H., Sellers T.A., et. al.: Mammographic density, breast cancer risk and risk prediction. Breast Cancer Res 2007; 9: pp. 217.</p><li><p>21. Harvey J.A., Bovbjerg V.E.: Quantitative assessment of mammographic breast density: relationship with breast cancer risk. Radiology 2004; 230: pp. 29-41.</p><li><p>22. Tice J.A., Cummings S.R., Ziv E., Kerlikowske K.: Mammographic breast density and the Gail model for breast cancer risk prediction in a screening population. Breast Cancer Res Treat 2005; 94: pp. 115-122.</p><li><p>23. Chen J., Pee D., Ayyagari R., et. al.: Projecting absolute invasive breast cancer risk in white women with a model that includes mammographic density. J Natl Cancer Inst 2006; 98: pp. 1215-1226.</p><li><p>24. Tice J.A., Cummings S.R., Smith-Bindman R., Ichikawa L., Barlow W.E., Kerlikowske K.: Using clinical factors and mammographic breast density to estimate breast cancer risk: development and validation of a new predictive model. Ann Intern Med 2008; 148: pp. 337-347.</p><li><p>25. Caldwell C.B., Stapleton S.J., Holdsworth D.W., et. al.: Characterization of mammographic parenchymal pattern by fractal dimension. Phys Med Biol 1990; 35: pp. 235-247.</p><li><p>26. Tahoces P.G., Correa J., Souto M., Gomez L., Vidal J.J.: Computer-assisted diagnosis: the classification of mammographic breast parenchymal patterns. Phys Med Biol 1995; 40: pp. 103-117.</p><li><p>27. Huo Z., Giger M.L., Wolverton D.E., Zhong W., Cumming S., Olopade O.I.: Computerized analysis of mammographic parenchymal patterns for breast cancer risk assessment: feature selection. Med Phys 2000; 27: pp. 4-12.</p><li><p>28. Huo Z., Giger M.L., Olopade O.I., et. al.: Computerized analysis of digitized mammograms of BRCA1 and BRCA2 gene mutation carriers. Radiology 2002; 225: pp. 519-526.</p><li><p>29. Li H., Giger M.L., Olopade O.I., Margolis A., Lan L., Chinander M.R.: Computerized texture analysis of mammographic parenchymal patterns of digitized mammograms. Acad Radiol 2005; 12: pp. 863-873.</p><li><p>30. Li H., Giger M.L., Huo Z., et. al.: Computerized analysis of mammographic parenchymal patterns for assessing breast cancer risk: effect of ROI size and location. Med Phys 2004; 31: pp. 549-555.</p><li><p>31. Li H., Giger M.L., Olopade O.I., Lan L.: Fractal analysis of mammographic parenchymal patterns in breast cancer risk assessment. Acad Radiol 2007; 14: pp. 513-521.</p><li><p>32. Kopans D.B.: Basic physics and doubts about relationship between mammographically determined tissue density and breast cancer risk. Radiology 2008; 246: pp. 348-353.</p><li><p>33. Niklason L.T., Christian B.T., Niklason L.E., et. al.: Digital tomosynthesis in breast imaging. Radiology 1997; 205: pp. 399-406.</p><li><p>34. Park J.M., Franken E.A., Garg M., Fajardo L.L., Niklason L.T.: Breast tomosynthesis: present considerations and future applications. RadioGraphics 2007; 27: pp. S231-S240.</p><li><p>35. Rafferty E.A.: Digital mammography: novel applications. Radiol Clin North Am 2007; 45: pp. 831-843.</p><li><p>36. Poplack S.P., Tosteson T.D., Kogel C.A., Nagy H.M.: Digital breast tomosynthesis: initial experience in 98 women with abnormal digital screening mammography. AJR Am J Roentgenol 2007; 189: pp. 616-623.</p><li><p>37. American College of Radiology.2003.American College of Radiology</p><li><p>38. Kontos D., Bakic P.R., Maidment A.D.A.: Analysis of parenchymal texture properties in breast tomosynthesis images.Giger M.L.Karssemeijer N.SPIE medical imaging: computer aided diagnosis.2007.SPIESan Diego, CA:pp. 6514.</p><li><p>39. Bakic P.R., Kontos D., Zhang C., Yaffe M.J., Maidment A.D.A.: Analysis of percent density estimates from digital breast tomosynthesis projection images.Giger M.L.Karssemeijer N.SPIE medical imaging: computer aided diagnosis.2007.SPIESan Diego, CA:pp. 6514.</p><li><p>40. Wu T., Moore R.H., Rafferty E.A., Kopans D.B.: A comparison of reconstruction algorithms for breast tomosynthesis. Med Phys 2004; 31: pp. 2636-2647.</p><li><p>41. Radiological Society of North America. The RSNA Medical Imaging Resource Center (MIRC). Available at: http://www.rsna.org/mirc/ Accessed May 13, 2007</p><li><p>42. Kontos D., Bakic P.R., Maidment A.D.A.: Texture in digital breast tomosynthesis: a comparison between mammographic and tomographic characterization of parenchymal properties.Giger M.L.Karssemeijer N.SPIE medical imaging: computer aided diagnosis.2007.SPIESan Diego, CA:pp. 6915.</p><li><p>43. National Cancer Institute : Breast Cancer Risk Assessment Tool. Available at: http://www.cancer.gov/bcrisktool/ Accessed October 10, 2008</p><li><p>44. Byng J.W., Yaffe M.J., Jong R.A., et. al.: Analysis of mammographic density and breast cancer risk from digitized mammograms. RadioGraphics 1998; 18: pp. 1587-1598.</p><li><p>45. Byng J.W., Boyd N.F., Fishell E., Jong R.A., Yaffe M.J.: The quantitative analysis of mammographic densities. Phys Med Biol 1994; 39: pp. 1629-1638.</p><li><p>46. Bakic P.R., Kontos D., Carton A.K., Maidment A.D.A.: Breast percent density estimation from 3D reconstructed digital breast tomosynthesis images.Hsieh J.Samei E.SPIE medical imaging: physics of medical imaging.2008.SPIESan Diego, CA:pp. 6913.</p><li><p>47. Bakic PR, Carton AK, Zhang C, Yaffe M, Maidment ADA. Inter- and intra-observer agreement in breast percent density estimation from mammograms and central DBT projections. Presented at: Annual meeting of the Radiological Society of North America; Chicago, IL; 2007.</p><li><p>48. Li H., Giger M.L., Olopade O.I., Chinander M.R.: Power spectral analysis of mammographic parenchymal patterns for breast cancer risk assessment. J Digit Imaging 2008; 21: pp. 145-152.</p><li><p>49. Amadasum M., King R.: Textural features corresponding to textural properties. IEEE Trans Syst Man Cybernet 1989; 19: pp. 1264-1274.</p><li><p>50. Castellano G., Bonilha L., Li L.M., Cendes F.: Texture analysis of medical images. Clin Radiol 2004; 59: pp. 1061-1069.</p><li><p>51. Kurani A, Xu D-H, Furst J, Raicu D. Co-occurence matrices for volumetric data. Presented at: 7th IASTED International Conference on Computer Graphics and Imaging; 2004.</p><li><p>52. Mahmoud-Ghoneim D., Toussaint G., Constans J.M., de Certaines J.D.: Three dimensional texture analysis in MRI: a preliminary evaluation in gliomas. Magn Res Imaging 2003; 21: pp. 983-987.</p><li><p>53. Showalter C., Clymer B.D., Richmond B., Powell K.: Three-dimensional texture analysis of cancellous bone cores evaluated at clinical CT resolutions. Osteoporos Int 2006; 17: pp. 259-266.</p><li><p>54. Chen W., Giger M.L., Li H., Bick U., Newstead G.M.: Volumetric texture analysis of breast lesions on contrast-enhanced magnetic resonance images. Magn Reson Med 2007; 58: pp. 562-571.</p><li><p>55. Haralick R.M., Shanmugam K., Dinstein I.: Textural features for image classification. IEEE Trans Syst Man Cybernet 1973; 3: pp. 610-621.</p><li><p>56. Russ J.C.: Fractal surfaces.1994.PlenumNew York</p><li><p>57. Petrie A., Sabin C.: Medical statistics at a glance.2000.BlackwellOxford, UK</p><li><p>58. Glantz S.: Primer of biostatistics.2005.McGraw-HillNew York</p><li><p>59. Duda R., Hart P., Stork D.: Pattern classification.2000.John WileyNew York</p><li><p>60. Rousseau R.: Jaccard similarity leads to the Marczewski-Steinhaus topology for information retrieval. Inform Process Manage 1998; 34: pp. 87-94.</p><li><p>61. Martin K.E., Helvie M.A., Zhou C., et. al.: Mammographic density measured with quantitative computer-aided method: comparison with radiologists’ estimates and BI-RADS categories. Radiology 2006; 240: pp. 656-665.</p><li><p>62. Zhou C., Chan H.P., Petrick N., et. al.: Computerized image analysis: estimation of breast density on mammograms. Med Phys 2001; 28: pp. 1056-1069.</p><li><p>63. Kovalev V., Petrou M.: Texture anisotropy in 3-D images. IEEE Trans Image Process 1999; 8: pp. 346-360.</p><li><p>64. Kontos D., Zhang C., Ruiter N.V., Bakic P.R., Maidment A.D.A.: Evaluating the effect of tomosynthesis acquisition parameters on image texture: a study based on an anthropomorphic breast tissue software model.Krupinski E.A.International Workshop on Digital Mammography (IWDM) 2008.2008.Springer-VerlagBerlin, Germany::pp. 491-498.</p><li><p>65. Palomares M.R., Machia J.R., Lehman C.D., Daling J.R., McTiernan A.: Mammographic density correlation with Gail model breast cancer risk estimates and component risk factors. Cancer Epidemiol Biomarker Prevent 2006; 15: pp. 1324-1330.</p><li><p>66. Santen R.J., Boyd N.F., Chlebowski R.T., et. al.: Critical assessment of new risk factors for breast cancer: considerations for development of an improved risk prediction model. Endocrine Relat Cancer 2007; 14: pp. 169-187.</p><li><p>67. Karellas A.: Mammographic pattern analysis: an emerging risk assessment tool. Acad Radiol 2007; 14: pp. 511-512.</p><li><p>68. Kerlikowske K.: The mammogram that cried Wolfe. N Engl J Med 2007; 356: pp. 297-300.</p><li><p>69. Barlow W.E., White E., Ballard-Barbash R., et. al.: Prospective breast cancer risk prediction model for women undergoing screening mammography. J Natl Cancer Inst 2006; 98: pp. 1204-1214.</p><li><p>70. Zhang Y., Chan H.P., Sahiner B., et. al.: A comparative study of limited-angle cone-beam reconstruction methods for breast tomosynthesis. Med Phys 2006; 33: pp. 3781-3795.</p><li><p>71. Rafferty E.A.: Tomosynthesis: new weapon in breast cancer fight. Decis Imaging Econ 2004; pp. 17.</p></ul></div><div class="post-tail-wrapper text-muted"><div class="post-meta mb-3"> <i class="far fa-folder-open fa-fw mr-1"></i> <a href='/categories/academic-radiology/'>Academic Radiology</a>, <a href='/categories/volume-16/'>Volume 16</a>, <a href='/categories/issue-3/'>Issue 3</a></div><div class="post-tags"> <i class="fa fa-tags fa-fw mr-1"></i> <a href="/tags/journals/" class="post-tag no-text-decoration" >Journals</a> <a href="/tags/general-radiology/" class="post-tag no-text-decoration" >General Radiology</a></div><div class="post-tail-bottom d-flex justify-content-between align-items-center mt-3 pt-5 pb-2"><div class="license-wrapper"> This post is licensed under <a href="https://creativecommons.org/licenses/by/4.0/"> CC BY 4.0 </a> by the author.</div><div class="share-wrapper"> <span class="share-label text-muted mr-1">Share</span> <span class="share-icons"> <a href="https://twitter.com/intent/tweet?text=Parenchymal%20Texture%20Analysis%20in%20Digital%20Breast%20Tomosynthesis%20for%20Breast%20Cancer%20Risk%20Estimation%20-%20Radiology%20Tree&url=https%3A%2F%2Fclinicaltree.github.io%2Fposts%2Fparenchymal-texture-analysis-in-digital-breast-tomosynthesis-for-breast-cancer-risk-estimation%2F" data-toggle="tooltip" data-placement="top" title="Twitter" target="_blank" rel="noopener" aria-label="Twitter"> <i class="fa-fw fab fa-twitter"></i> </a> <a href="https://www.facebook.com/sharer/sharer.php?title=Parenchymal%20Texture%20Analysis%20in%20Digital%20Breast%20Tomosynthesis%20for%20Breast%20Cancer%20Risk%20Estimation%20-%20Radiology%20Tree&u=https%3A%2F%2Fclinicaltree.github.io%2Fposts%2Fparenchymal-texture-analysis-in-digital-breast-tomosynthesis-for-breast-cancer-risk-estimation%2F" data-toggle="tooltip" data-placement="top" title="Facebook" target="_blank" rel="noopener" aria-label="Facebook"> <i class="fa-fw fab fa-facebook-square"></i> </a> <a href="https://t.me/share/url?url=https%3A%2F%2Fclinicaltree.github.io%2Fposts%2Fparenchymal-texture-analysis-in-digital-breast-tomosynthesis-for-breast-cancer-risk-estimation%2F&text=Parenchymal%20Texture%20Analysis%20in%20Digital%20Breast%20Tomosynthesis%20for%20Breast%20Cancer%20Risk%20Estimation%20-%20Radiology%20Tree" data-toggle="tooltip" data-placement="top" title="Telegram" target="_blank" rel="noopener" aria-label="Telegram"> <i class="fa-fw fab fa-telegram"></i> </a> <i id="copy-link" class="fa-fw fas fa-link small" data-toggle="tooltip" data-placement="top" title="Copy link" data-title-succeed="Link copied successfully!"> </i> </span></div></div></div></div></div><div id="panel-wrapper" class="col-xl-3 pl-2 text-muted"><div class="access"><div id="access-lastmod" class="post"><div class="panel-heading">Recently Updated</div><ul class="post-content pl-0 pb-1 ml-1 mt-2"><li><a href="/posts/neurometabolites-alteration-in-the-acute-phase-of-mild-traumatic-brain-injury-mtbi/">Neurometabolites Alteration in the Acute Phase of Mild Traumatic Brain Injury (mTBI)</a><li><a href="/posts/reinforcing-the-importance-and-feasibility-of-implementing-a-low-dose-protocol-for-ct-guided-biopsie/">Reinforcing the Importance and Feasibility of Implementing a Low-dose Protocol for CT-guided Biopsies</a><li><a href="/posts/rethinking-the-pgy-1-basic-clinical-year/">Rethinking the PGY-1 Basic Clinical Year</a><li><a href="/posts/single-injection-dual-phase-cone-beam-ct-dp-cbct-vascular-anatomy-assessment-and-occult-nodule-det/">Single Injection Dual-Phase Cone Beam CT (DP-CBCT) Vascular Anatomy Assessment and Occult Nodule Detection; Have We Reached the Focus?</a><li><a href="/posts/the-yellow-scale-is-superior-to-the-gray-scale-for-detecting-acute-ischemic-stroke-on-a-monitor-disp/">The Yellow Scale Is Superior to the Gray Scale for Detecting Acute Ischemic Stroke on a Monitor Display in Computed Tomography</a></ul></div><div id="access-tags"><div class="panel-heading">Trending Tags</div><div class="d-flex flex-wrap mt-3 mb-1 mr-3"> <a class="post-tag" href="/tags/general-radiology/">General Radiology</a> <a class="post-tag" href="/tags/journals/">Journals</a></div></div></div><div id="toc-wrapper" class="pl-0 pr-4 mb-5"><div class="panel-heading pl-3 pt-2 mb-2">Contents</div><nav id="toc"></nav></div><script src="https://cdn.jsdelivr.net/npm/tocbot@4.20.1/dist/tocbot.min.js"></script></div></div><div class="row"><div id="tail-wrapper" class="col-12 col-lg-11 col-xl-9 pl-3 pr-3 pr-xl-4 mt-5"><div id="related-posts" class="mb-2 mb-sm-4"><h3 class="pt-2 mb-4 ml-1" data-toc-skip>Further Reading</h3><div class="card-deck mb-4"><div class="card"> <a href="/posts/3d-ct-volumetry-of-the-lung-using-multidetector-row-ct/"><div class="card-body"> <em class="small" data-ts="1235840400" data-df="ll" > Feb 28, 2009 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>3D-CT Volumetry of the Lung Using Multidetector Row CT</h3><div class="text-muted small"><p> Rationale and Objectives The aim of this study was to evaluate the accuracy of measurements of lung volumes reconstructed using three-dimensional computed tomographic (CT) imaging from thin-sectio...</p></div></div></a></div><div class="card"> <a href="/posts/a-few-words-about-lobbying/"><div class="card-body"> <em class="small" data-ts="1235840400" data-df="ll" > Feb 28, 2009 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>A few words about lobbying</h3><div class="text-muted small"><p> During the 2008 presidential campaign, both the winner and the loser spent a considerable amount of rhetoric denouncing the activities of lobbyists, scorning (but not always spurning) donations fro...</p></div></div></a></div><div class="card"> <a href="/posts/abstracts-of-funded-national-institutes-of-health-grants/"><div class="card-body"> <em class="small" data-ts="1235840400" data-df="ll" > Feb 28, 2009 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>Abstracts of Funded National Institutes of Health Grants</h3><div class="text-muted small"><p> A novel device to allow zoom-in imaging for PET scanners Grant Number:5R33CA110011-04PI Name:Tai, Yuan-Chuan Abstract: Description (provided by applicant): Positron emission tomography (PET) has ...</p></div></div></a></div></div></div><div class="post-navigation d-flex justify-content-between"> <a href="/posts/mri-interpretation-proficiency-of-musculoskeletal-fellows-in-training/" class="btn btn-outline-primary" prompt="Older"><p>MRI Interpretation Proficiency of Musculoskeletal Fellows in Training</p></a> <a href="/posts/quantitative-estimation-of-dynamic-contrast-enhanced-mri-parameters-in-rat-brain-gliomas-using-a-dua/" class="btn btn-outline-primary" prompt="Newer"><p>Quantitative Estimation of Dynamic Contrast Enhanced MRI Parameters in Rat Brain Gliomas Using a Dual Surface Coil System</p></a></div></div></div></div><div id="search-result-wrapper" class="d-flex justify-content-center unloaded"><div class="col-12 col-sm-11 post-content"><div id="search-hints"><div id="access-tags"><div class="panel-heading">Trending Tags</div><div class="d-flex flex-wrap mt-3 mb-1 mr-3"> <a class="post-tag" href="/tags/general-radiology/">General Radiology</a> <a class="post-tag" href="/tags/journals/">Journals</a></div></div></div><div id="search-results" class="d-flex flex-wrap justify-content-center text-muted mt-3"></div></div></div></div><footer><div class="container pl-lg-4 pr-lg-4"><div class="d-flex justify-content-between align-items-center text-muted ml-md-3 mr-md-3"><div class="footer-left"><p class="mb-0"> © 2023 <a href="https://twitter.com/username">Clinical Team</a>. <span data-toggle="tooltip" data-placement="top" title="Except where otherwise noted, the blog posts on this site are licensed under the Creative Commons Attribution 4.0 International (CC BY 4.0) License by the author.">Some rights reserved.</span></p></div><div class="footer-right"><p class="mb-0">Using the <a href="https://jekyllrb.com" target="_blank" rel="noopener">Jekyll</a> theme <a href="https://github.com/cotes2020/jekyll-theme-chirpy" target="_blank" rel="noopener">Chirpy</a>.</p></div></div></div></footer><div id="mask"></div><a id="back-to-top" href="#" aria-label="back-to-top" class="btn btn-lg btn-box-shadow" role="button"> <i class="fas fa-angle-up"></i> </a><div id="notification" class="toast" role="alert" aria-live="assertive" aria-atomic="true" data-animation="true" data-autohide="false"><div class="toast-header"> <button type="button" class="ml-2 ml-auto close" data-dismiss="toast" aria-label="Close"> <span aria-hidden="true">&times;</span> </button></div><div class="toast-body text-center pt-0"><p class="pl-2 pr-2 mb-3">A new version of content is available.</p><button type="button" class="btn btn-primary" aria-label="Update"> Update </button></div></div><script src="https://cdn.jsdelivr.net/npm/simple-jekyll-search@1.10.0/dest/simple-jekyll-search.min.js"></script> <script> SimpleJekyllSearch({ searchInput: document.getElementById('search-input'), resultsContainer: document.getElementById('search-results'), json: '/assets/js/data/search.json', searchResultTemplate: '<div class="pl-1 pr-1 pl-sm-2 pr-sm-2 pl-lg-4 pr-lg-4 pl-xl-0 pr-xl-0"> <a href="{url}">{title}</a><div class="post-meta d-flex flex-column flex-sm-row text-muted mt-1 mb-1"> {categories} {tags}</div><p>{snippet}</p></div>', noResultsText: '<p class="mt-5">Oops! No results found.</p>', templateMiddleware: function(prop, value, template) { if (prop === 'categories') { if (value === '') { return `${value}`; } else { return `<div class="mr-sm-4"><i class="far fa-folder fa-fw"></i>${value}</div>`; } } if (prop === 'tags') { if (value === '') { return `${value}`; } else { return `<div><i class="fa fa-tag fa-fw"></i>${value}</div>`; } } } }); </script> <script src="https://cdn.jsdelivr.net/combine/npm/magnific-popup@1.1.0/dist/jquery.magnific-popup.min.js,npm/lazysizes@5.3.2/lazysizes.min.js,npm/clipboard@2.0.11/dist/clipboard.min.js"></script> <script src="https://cdn.jsdelivr.net/combine/npm/dayjs@1.11.6/dayjs.min.js,npm/dayjs@1.11.6/locale/en.min.js,npm/dayjs@1.11.6/plugin/relativeTime.min.js,npm/dayjs@1.11.6/plugin/localizedFormat.min.js"></script> <script defer src="/assets/js/dist/post.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/js/bootstrap.bundle.min.js"></script> <script defer src="/app.js"></script> <script defer src="https://www.googletagmanager.com/gtag/js?id=G-L66SLQK23K"></script> <script> document.addEventListener("DOMContentLoaded", function(event) { window.dataLayer = window.dataLayer || []; function gtag(){dataLayer.push(arguments);} gtag('js', new Date()); gtag('config', 'G-L66SLQK23K'); }); </script>
